<!DOCTYPE HTML>
<!--
	Landed by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>
	    
	Preprints

		</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/jquery-ui.min.css" />
		<link rel="stylesheet" href="assets/css/jquery-ui.structure.min.css" />
		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/loyal_mods.css" />
	</head>


	
<body class="is-preload">
<div id="page-wrapper">
	<!-- Header -->
  <header id="header">
    <h1 id="logo">
      <a href="index.html"> <img id="shield" src="images/medicine.shield.small.white.png" height="40" alt=""/>Stein-O'Brien Lab</a>
    </h1>
    <nav id="nav">
      <ul>
        <li><a href="index.html">Home</a></li>
        <!-- <li><a href="news.html">News</a></li> -->
        <li><a href="people.html">People</a></li>
        <li><a href="#">Research</a>
            <ul>
              <li><a href="preprints.html">Preprints</a></li>
              <li><a href="publications.html">Publications</a></li>
              <li><a href="software.html">Software</a></li>
              <li><a href="datasets.html">Datasets</a></li>
            </ul>
        </li>
        <!-- <li><a href="teaching.html">Teaching</a></li> -->
        <!--
        <li><a href="#">Resources</a>
            <ul>
              <li><a href="members.html">Lab Members</a></li>
              <li><a href="links.html">Links</a></li>
            </ul>
        </li>
      -->
        <!-- <li><a href="blog.html">Blog</a></li> -->
        <li><a href="lab_resources.html">Resources</a></li>
        <li><a href="join.html">Opportunites</a></li>
        <li><a href="contact.html">Contact Us</a></li>
        <!--<li><a href="#" class="button primary">Sign Up</a></li>-->
      </ul>
    </nav>
  </header>
	<!-- Main -->
		<div id="main" class="wrapper style1">
			<div class="container">
				<header class="major">
					<h2>Preprints</h2>
				</header>

				<!-- Content -->
					<section id="content">
						<!-- Accordion -->
						<div class="accordion alt" id="accordionPreprints">
							
							<!--  Card 1  -->
							<div class="card text-left">
								<div class="card-header" id="heading1">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse1" aria-expanded="true" aria-controls="collapse1">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">1. Transcriptomic forecasting with neural ODEs (2022)</p>
													 <p class="note font-weight-light"><span class="font-italic">Erbe, R., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Fertig, E. J.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse1" class="collapse" aria-labelledby="heading1" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2022.08.04.502825" class="altmetric-embed pull-left"> </div>
										<p>Single-cell RNA-seq (scRNA-seq) technologies can uncover changes in the molecular states that underlie cellular phenotypes. However, to understand dynamic cellular processes, computational tools are needed to extract temporal information from the snapshots of cellular gene expression that scRNA-seq provides. To address this challenge, we have developed a neural ordinary differential equation based method, RNAForecaster, for predicting gene expression states in single cells for multiple future time steps. We demonstrate that RNAForecaster can accurately predict future expression states in simulated scRNA-seq data. We then show that using metabolic labeling scRNA-seq data from constitutively dividing cells, RNAForecaster accurately recapitulates many of the expected changes in gene expression during progression through the cell cycle over a three day period. Finally, we extend RNAForecaster to use unspliced and spliced counts from scRNA-seq to predict the impact of knock down experiments in pancreatic development. Thus, RNAForecaster enables estimation of future expression states in biological systems.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2022.08.04.502825">10.1101/2022.08.04.502825</a>
									</div>
								</div>
							</div>
							
							<!--  Card 2  -->
							<div class="card text-left">
								<div class="card-header" id="heading2">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse2" aria-expanded="true" aria-controls="collapse2">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">2. Inflammatory Signaling and Fibroblast-Cancer Cell Interactions Transfer from a Harmonized Human Single-cell RNA Sequencing Atlas of Pancreatic Ductal Adenocarcinoma to Organoid Co-Culture (2022)</p>
													 <p class="note font-weight-light"><span class="font-italic">Kinny-Koster, B., Guinn, S., Tandurella, J. A., Mitchell, J. T., Sidiropoulos, D. N., Loth, M., Lyman, M. R., Pucsek, A. B., Seppala, T. T., Cherry, C., Suri, R., Zlomke, H., He, J., Wolfgang, C. L., Yu, J., Zheng, L., Ryan, D. P., Ting, D. T., Kimmelman, A., Gupta, A., Danilova, L., Elisseef, J., Wood, L. D., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kagohara, L. T., Burkhart, R. A., Jaffee, E. M., Fertig, E. J., Zimmerman, J. W.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse2" class="collapse" aria-labelledby="heading2" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2022.07.14.500096" class="altmetric-embed pull-left"> </div>
										<p>Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy driven by a heterogeneous tumor microenvironment enriched with cancer associated fibroblasts (CAFs) that influence its overall immunosuppressive composition. We examined inflammation in PDAC by leveraging our existing patient-derived organoid (PDO) model and a novel PDO-CAF co-culture. We first identified induction of major histocompatibility complex class II expression following treatment with interferon gamma. In parallel, we collated an atlas of 6 published single-cell RNA-sequencing datasets (174,394 cells) combining 61 PDAC (142,807 cells) and 16 non-malignant samples. By combining in silico modeling and in vitro PDO co-culture, we define a gene expression pattern of inflammatory processes in epithelial tumor cells. Following computational inferences, we examined interactions between epithelial tumor cells and CAFs, focusing on VEGF-A and ITGB1 pathways. This work, integrating computational and biological approaches, highlights the value of convergence to accelerate our understanding of key drivers of PDAC.

Statement of SignificanceWe established PDO-CAF co-cultures to model tumor cell interactions and validated discoveries using transfer learning into a single-cell RNA-seq atlas of PDAC tumors. This bidirectional approach from human to experimental systems facilitates interrogation of PDAC biology, including the role of inflammation associated with VEGF-A crosstalk between CAFs and tumor cells.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2022.07.14.500096">10.1101/2022.07.14.500096</a>
									</div>
								</div>
							</div>
							
							<!--  Card 3  -->
							<div class="card text-left">
								<div class="card-header" id="heading3">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse3" aria-expanded="true" aria-controls="collapse3">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">3. Inferring cellular and molecular processes in single-cell data with non-negative matrix factorization using Python, R, and GenePattern Notebook implementations of CoGAPS (2022)</p>
													 <p class="note font-weight-light"><span class="font-italic">Johnson, J. A. I., Tsang, A., Mitchell, J. T., Davis-Marcisak, E. F., Sherman, T., Liefeld, T., Loth, M., Goff, L., Zimmerman, J., Kinny-KoÌˆster, B., Jaffee, E., Tamayo, P., Mesirov, J., Reich, M., Fertig, E. J., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>,</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse3" class="collapse" aria-labelledby="heading3" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2022.07.09.499398" class="altmetric-embed pull-left"> </div>
										<p>Non-negative matrix factorization (NMF) is an unsupervised learning method well suited to high-throughput biology. Still, inferring biological processes requires additional post hoc statistics and annotation for interpretation of features learned from software packages developed for NMF implementation. Here, we aim to introduce a suite of computational tools that implement NMF and provide methods for accurate, clear biological interpretation and analysis. A generalized discussion of NMF covering its benefits, limitations, and open questions in the field is followed by three vignettes for the Bayesian NMF algorithm CoGAPS (Coordinated Gene Activity across Pattern Subsets). Each vignette will demonstrate NMF analysis to quantify cell state transitions in public domain single-cell RNA-sequencing (scRNA-seq) data of malignant epithelial cells in 25 pancreatic ductal adenocarcinoma (PDAC) tumors and 11 control samples. The first uses PyCoGAPS, our new Python interface for CoGAPS that we developed to enhance runtime of Bayesian NMF for large datasets. The second vignette steps through the same analysis using our R CoGAPS interface, and the third introduces two new cloud-based, plug-and-play options for running CoGAPS using GenePattern Notebook and Docker. By providing Python support, cloud-based computing options, and relevant example workflows, we facilitate user-friendly interpretation and implementation of NMF for single-cell analyses.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2022.07.09.499398">10.1101/2022.07.09.499398</a>
									</div>
								</div>
							</div>
							
							<!--  Card 4  -->
							<div class="card text-left">
								<div class="card-header" id="heading4">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse4" aria-expanded="true" aria-controls="collapse4">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">4. Pumping the brakes on RNA velocity -- understanding and interpreting RNA velocity estimates (2022)</p>
													 <p class="note font-weight-light"><span class="font-italic">Zheng, S. C., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Boukas, L., Goff, L. A., Hansen, K. D.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse4" class="collapse" aria-labelledby="heading4" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2022.06.19.494717" class="altmetric-embed pull-left"> </div>
										<p>RNA velocity analysis of single cells promises to predict temporal dynamics from gene expression. Indeed, in many systems, it has been observed that RNA velocity produces a vector field that qualitatively reflects known features of the system. Despite this observation, the limitations of RNA velocity estimates are poorly understood. Using real data and simulations, we dissect the impact of different steps in the RNA velocity workflow on the estimated vector field. We find that the process of mapping RNA velocity estimates into a low-dimensional representation, such as those produced by UMAP, has a large impact on the result. The RNA velocity vector field strongly depends on the k-NN graph of the data. This dependence leads to significant estimator errors when the k-NN graph is not a faithful representation of the true data structure, a feature that cannot be known for most real datasets. Finally, we establish that RNA velocity estimates expression speed neither at the gene nor cellular level. We propose that RNA velocity is best considered a smoothed interpolation of the observed k-NN structure, as opposed to an extrapolation of future cellular states, and that the use of RNA velocity as a validation of latent space embedding structures is circular.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2022.06.19.494717">10.1101/2022.06.19.494717</a>
									</div>
								</div>
							</div>
							
							<!--  Card 5  -->
							<div class="card text-left">
								<div class="card-header" id="heading5">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse5" aria-expanded="true" aria-controls="collapse5">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">5. Insights for disease modeling from single cell transcriptomics of iPSC-derived neurons and astrocytes across differentiation time and co-culture (2022)</p>
													 <p class="note font-weight-light"><span class="font-italic">Das, D., Sonthalia, S., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>,G., Wahbeh, M., Feuer, K., Colantuoni, C., Machairaki, V., Avramopoulos, D.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse5" class="collapse" aria-labelledby="heading5" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2022.06.15.495952" class="altmetric-embed pull-left"> </div>
										<p>Trans-differentiation of human induced pluripotent stem cells into neurons (hiPSC-N) via Ngn2-induction has become an efficient system to quickly generate neurons for disease modeling and in vitro assay development, a significant step up from previously used neoplastic and other cell lines. Recent single-cell interrogation of Ngn2-induced neurons however, has revealed some similarities to unexpected neuronal lineages. Similarly, a straightforward method to generate hiPSC derived astrocytes (hiPSC-A) for the study of neuropsychiatric disorders has also been described. Here we examine the homogeneity and similarity of hiPSC-N and hiPSC-A to their in vivo counterparts, the impact of different lengths of time post Ngn2 induction on hiPSC-N (15 or 21 days) and of hiPSC-N / hiPSC-A co-culture. We explore how often genes differentially expressed between conditions relate to genetic risk for neuropsychiatric disease. Leveraging the wealth of existing public single-cell RNA-seq (scRNA-seq) data in Ngn2-induced neurons and in vivo data from the developing brain, we provide perspectives on the lineage origins and maturation of hiPSC-N and hiPSC-A. Both show heterogeneity and share similarity with multiple in vivo cell fates, and both cell types more precisely approximate their in vivo counterparts when co-cultured. hiPSC-A show more heterogeneity and similarities to other non-neural cell types, especially when cultured in isolation. Gene expression data from the hiPSC-N show excess of genes linked to schizophrenia (SZ) and autism spectrum disorders (ASD) as has been previously shown for neural stem cells and neurons. These overrepresentations of disease genes are strongest in our system at early times (day 15) in Ngn2-induction/maturation of neurons, which together with our observation of similarities with in vivo neurons earlier in development suggest they may be a better model for neurodevelopmental disorders. We have assembled this new scRNA-seq data along with the public data explored here as an integrated biologist-friendly web-resource for researchers seeking to understand this system more deeply: https://nemoanalytics.org/p?l=DasEtAl2022.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2022.06.15.495952">10.1101/2022.06.15.495952</a>
									</div>
								</div>
							</div>
							
							<!--  Card 6  -->
							<div class="card text-left">
								<div class="card-header" id="heading6">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse6" aria-expanded="true" aria-controls="collapse6">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">6. Uncovering the spatial landscape of molecular interactions within the tumor microenvironment through latent spaces (2022)</p>
													 <p class="note font-weight-light"><span class="font-italic">Deshpande, A., Loth, M., Sidiropoulos, D. N., Zhang, S., Yuan, L., Bell, A., Zhu, Q., Ho, W. J., Santa-Maria, C. A., Gilkes, D. M., Williams, S., Uytingco, C., Chew, J., Hartnett, A., Bent, Z., Favorov, A. V., Popel, A. S., Yarchoan, M., Zheng, L., Jaffee, E. M., Anders, R., Danilova, L., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kagohara, L. T., Fertig, E. J.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse6" class="collapse" aria-labelledby="heading6" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2022.06.02.490672" class="altmetric-embed pull-left"> </div>
										<p>Recent advances in spatial transcriptomics (ST) enable gene expression measurements from a tissue sample while retaining its spatial context. This technology enables unprecedented in situ resolution of the regulatory pathways that underlie the heterogeneity in the tumor and its microenvironment (TME). The direct characterization of cellular co-localization with spatial technologies facilities quantification of the molecular changes resulting from direct cell-cell interaction, as occurs in tumor-immune interactions. We present SpaceMarkers, a novel bioinformatics algorithm to infer molecular changes from cell-cell interaction from latent space analysis of ST data. We apply this approach to infer molecular changes from tumor-immune interactions in Visium spatial transcriptomics data of metastasis, invasive and precursor lesions, and immunotherapy treatment. Further transfer learning in matched scRNA-seq data enabled further quantification of the specific cell types in which SpaceMarkers are enriched. Altogether, SpaceMarkers can identify the location and context-specific molecular interactions within the TME from ST data.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2022.06.02.490672">10.1101/2022.06.02.490672</a>
									</div>
								</div>
							</div>
							
							<!--  Card 7  -->
							<div class="card text-left">
								<div class="card-header" id="heading7">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse7" aria-expanded="true" aria-controls="collapse7">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">7. Two-stage Linked Component Analysis for Joint Decomposition of Multiple Biologically Related Data Sets (2021)</p>
													 <p class="note font-weight-light"><span class="font-italic">Chen, H., Caffo, B., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Liu, J., Langmead, B., Colantuoni, C., Xiao, L.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse7" class="collapse" aria-labelledby="heading7" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2021.03.22.435728" class="altmetric-embed pull-left"> </div>
										<p>SO_SCPLOWUMMARYC_SCPLOWIntegrative analysis of multiple data sets has the potential of fully leveraging the vast amount of high throughput biological data being generated. In particular such analysis will be powerful in making inference from publicly available collections of genetic, transcriptomic and epigenetic data sets which are designed to study shared biological processes, but which vary in their target measurements, biological variation, unwanted noise, and batch variation. Thus, methods that enable the joint analysis of multiple data sets are needed to gain insights into shared biological processes that would otherwise be hidden by unwanted intra-data set variation. Here, we propose a method called two-stage linked component analysis (2s-LCA) to jointly decompose multiple biologically related experimental data sets with biological and technological relationships that can be structured into the decomposition. The consistency of the proposed method is established and its empirical performance is evaluated via simulation studies. We apply 2s-LCA to jointly analyze four data sets focused on human brain development and identify meaningful patterns of gene expression in human neurogenesis that have shared structure across these data sets.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2021.03.22.435728">10.1101/2021.03.22.435728</a>
									</div>
								</div>
							</div>
							
							<!--  Card 8  -->
							<div class="card text-left">
								<div class="card-header" id="heading8">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse8" aria-expanded="true" aria-controls="collapse8">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">8. Universal prediction of cell cycle position using transfer learning (2021)</p>
													 <p class="note font-weight-light"><span class="font-italic">Zheng, S. C., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Augustin, J. J., Slosberg, J., Carosso, G. A., Winer, B., Shin, G., Bjornsson, H. T., Goff, L. A., Hansen, K. D.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse8" class="collapse" aria-labelledby="heading8" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2021.04.06.438463" class="altmetric-embed pull-left"> </div>
										<p>BackgroundThe cell cycle is a highly conserved, continuous process which controls faithful replication and division of cells. Single-cell technologies have enabled increasingly precise measurements of the cell cycle both as a biological process of interest and as a possible confounding factor. Despite its importance and conservation, there is no universally applicable approach to infer position in the cell cycle with high-resolution from single-cell RNA-seq data.

ResultsHere, we present tricycle, an R/Bioconductor package, to address this challenge by leveraging key features of the biology of the cell cycle, the mathematical properties of principal component analysis of periodic functions, and the use of transfer learning. We estimate a cell cycle embedding using a fixed reference dataset and project new data into this reference embedding; an approach that overcomes key limitations of learning a dataset dependent embedding. Tricycle then predicts a cell-specific position in the cell cycle based on the data projection. The accuracy of tricycle compares favorably to gold-standard experimental assays, which generally require specialized measurements in specifically constructed in vitro systems. Using internal controls which are available for any dataset, we show that tricycle predictions generalize to datasets with multiple cell types, across tissues, species and even sequencing assays.

ConclusionsTricycle generalizes across datasets, is highly scalable and applicable to atlas-level single-cell RNA-seq data.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2021.04.06.438463">10.1101/2021.04.06.438463</a>
									</div>
								</div>
							</div>
							
							<!--  Card 9  -->
							<div class="card text-left">
								<div class="card-header" id="heading9">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse9" aria-expanded="true" aria-controls="collapse9">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">9. Identifying Gene-wise Differences in Latent Space Projections Across Cell Types and Species in Single Cell Data using scProject (2021)</p>
													 <p class="note font-weight-light"><span class="font-italic">Baraban, A., Clark, B. S., Slosberg, J., Fertig, E. J., Goff, L. A., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>,</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse9" class="collapse" aria-labelledby="heading9" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2021.08.25.457650" class="altmetric-embed pull-left"> </div>
										<p>Latent space techniques have emerged as powerful tools to identify genes and gene sets responsible for cell-type and species-specific differences in single-cell data. Transfer learning methods can compare learned latent spaces across biological systems. However, the robustness that comes from leveraging information across multiple genes in transfer learning is often attained at the sacrifice of gene-wise precision. Thus, methods are needed to identify genes, defined as important within a particular latent space, that significantly differ between contexts. To address this challenge, we have developed a new framework, scProject, and a new metric, projectionDrivers, to quantitatively examine latent space usage across single-cell experimental systems while concurrently extracting the genes driving the differential usage of the latent space between defined contrasts. Here, we demonstrate the efficacy, utility, and scalability of scProject with projectionDrivers and provide experimental validation for predicted species-specific differences between the developing mouse and human retina.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2021.08.25.457650">10.1101/2021.08.25.457650</a>
									</div>
								</div>
							</div>
							
							<!--  Card 10  -->
							<div class="card text-left">
								<div class="card-header" id="heading10">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse10" aria-expanded="true" aria-controls="collapse10">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">10. Mapping origins of variation in neural trajectories of human pluripotent stem cells (2021)</p>
													 <p class="note font-weight-light"><span class="font-italic">Kim, S. K., Seo, S., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Jaishankar, A., Ogawa, K., Micali, N., Wang, Y., Hyde, T., Kleinman, J., Voss, T., Fertig, E., Shin, J. H., Burli, R., Cross, A. J., Brandon, N. J., Weinberger, D. R., Chenoweth, J. G., Hoeppner, D. J., Sestan, N., Colantuoni, C., McKay, R. D.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse10" class="collapse" aria-labelledby="heading10" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2021.03.17.435870" class="altmetric-embed pull-left"> </div>
										<p>Variability between human pluripotent stem cell (hPSC) lines remains a challenge and opportunity in biomedicine. We identify differences in the spontaneous self-organization of individual hPSC lines during self-renewal that lie along a fundamental axis of in vivo development. Distinct stable and dynamic transcriptional elements were revealed by decomposition of RNA-seq data in pluripotency and early lineage emergence. Stable differences within pluripotency predicted regional bias in the dynamics of neural differentiation that were also observed in large collections of hPSC lines. Using replicate human induced PSC (hiPSC) lines and paired adult tissue, we demonstrate that cells from individual humans expressed unique transcriptional signatures that were maintained throughout life. In addition, replicate hiPSC lines from one donor showed divergent expression phenotypes driven by distinct chromatin states. These stable transcriptional states are under both genetic and epigenetic control and predict bias in subsequent forebrain and hindbrain neural trajectories. These results define mechanisms controlling transcription in pluripotency to first establish human neural diversity. Data availability: GSE164055; https://nemoanalytics.org/p?l=KimEtAL2021&g=GBX2.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC="FIGDIR/small/435870v1_ufig1.gif" ALT="Figure 1">
View larger version (45K):
org.highwire.dtl.DTLVardef@1c9434borg.highwire.dtl.DTLVardef@60f3e4org.highwire.dtl.DTLVardef@12c2f2aorg.highwire.dtl.DTLVardef@1ccae99_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2021.03.17.435870">10.1101/2021.03.17.435870</a>
									</div>
								</div>
							</div>
							
							<!--  Card 11  -->
							<div class="card text-left">
								<div class="card-header" id="heading11">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse11" aria-expanded="true" aria-controls="collapse11">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">11. Parallel social information processing circuits are differentially impacted in autism (2020)</p>
													 <p class="note font-weight-light"><span class="font-italic">Lewis, E. M., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Patino, A., Nardou, R., Grossman, C. D., Brown, M., Bangamwabo, B., Ndiaye, N., Giovinazzo, D., Dardani, I., Jiang, C., Goff, L., Dolen, G.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse11" class="collapse" aria-labelledby="heading11" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/2020.03.13.990549" class="altmetric-embed pull-left"> </div>
										<p>Parallel processing circuits are thought to dramatically expand the network capabilities of the nervous system. Magnocellular and parvocellular oxytocin neurons have been proposed to subserve two parallel streams of social information processing, which allow a single molecule to encode a diverse array of ethologically distinct behaviors, although to date direct evidence to support this hypothesis is lacking. Here we provide the first comprehensive characterization of magnocellular and parvocellular oxytocin neurons, validated across anatomical, projection target, electrophysiological, and transcriptional criteria. We next used novel multiple feature selection tools in Fmr1 KO mice to provide direct evidence that normal functioning of the parvocellular but not magnocellular oxytocin pathway is required for autism-relevant social reward behavior. Finally, we demonstrate that autism risk genes are uniquely enriched in parvocellular oxytocin neurons. Taken together these results provide the first evidence that oxytocin pathway specific pathogenic mechanisms account for social impairments across a broad range of autism etiologies.

One Sentence SummaryPathoclisis of parvocellular oxytocin neurons plays an important role in the pathogenesis of social impairments in autism.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/2020.03.13.990549">10.1101/2020.03.13.990549</a>
									</div>
								</div>
							</div>
							
							<!--  Card 12  -->
							<div class="card text-left">
								<div class="card-header" id="heading12">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse12" aria-expanded="true" aria-controls="collapse12">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">12. Human cortical neural stem cells generate regional organizer states in vitro before committing to excitatory neuronal fates (2019)</p>
													 <p class="note font-weight-light"><span class="font-italic">Micali, N., Kim, S.-K., Diaz-Bustamante, M., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Seo, S., Shin, J.-H., Rash, B. G., Ma, S., Olivares, N. A., Arellano, J., Maynard, K. R., Fertig, E. J., Cross, A. J., Burli, R., Brandon, N. J., Weinberger, D. R., Chenoweth, J. G., Hoeppner, D. J., Sestan, N., Rakic, P., Colantuoni, C., McKay, R. D.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse12" class="collapse" aria-labelledby="heading12" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/577544" class="altmetric-embed pull-left"> </div>
										<p>Better understanding the progression of neural stem cells (NSCs) in the developing cerebral cortex is important for modeling neurogenesis and defining the pathogenesis of neuropsychiatric disorders. Here we used RNA-sequencing, cell imaging and lineage tracing of mouse and human in vitro NSCs to model the generation of cortical neuronal fates. We show that conserved signaling mechanisms regulate the acute transition from proliferative NSCs to committed glutamatergic excitatory neurons. As human telencephalic NSCs developed from pluripotency in vitro, they first transitioned through organizer states that spatially pattern the cortex before generating glutamatergic precursor fates. NSCs derived from multiple human pluripotent lines varied in these early patterning states leading differentially to dorsal or ventral telencephalic fates. This work furthers systematic analysis of the earliest patterning events that generate the major neuronal trajectories of the human telencephalon.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/577544">10.1101/577544</a>
									</div>
								</div>
							</div>
							
							<!--  Card 13  -->
							<div class="card text-left">
								<div class="card-header" id="heading13">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse13" aria-expanded="true" aria-controls="collapse13">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">13. Single-cell analysis of human retina identifies evolutionarily conserved and species-specific mechanisms controlling development. (2019)</p>
													 <p class="note font-weight-light"><span class="font-italic">Lu, Y., Shiau, F., Yi, W., Lu, S., Wu, Q., Pearson, J., Kallman, A., Hoang, T., Zhong, S., Zuo, Z., Zhao, F., Zhang, M., Tsai, N., Zhou, Y., He, S., Zhang, J., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Sherman, T. D., Duan, X., Fertig, E. J., Goff, L., Zack, D., Handa, J. T., Xu, T., Bremner, R., Blackshaw, S., Wang, X., Clark, B. S.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse13" class="collapse" aria-labelledby="heading13" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/779694" class="altmetric-embed pull-left"> </div>
										<p>The development of single-cell RNA-Sequencing (scRNA-Seq) has allowed high resolution analysis of cell type diversity and transcriptional networks controlling cell fate specification. To identify the transcriptional networks governing human retinal development, we performed scRNA-Seq over retinal organoid and in vivo retinal development, across 20 timepoints. Using both pseudotemporal and cross-species analyses, we examined the conservation of gene expression across retinal progenitor maturation and specification of all seven major retinal cell types. Furthermore, we examined gene expression differences between developing macula and periphery and between two distinct populations of horizontal cells. We also identify both shared and species-specific patterns of gene expression during human and mouse retinal development. Finally, we identify an unexpected role for ATOH7 expression in regulation of photoreceptor specification during late retinogenesis. These results provide a roadmap to future studies of human retinal development, and may help guide the design of cell-based therapies for treating retinal dystrophies.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/779694">10.1101/779694</a>
									</div>
								</div>
							</div>
							
							<!--  Card 14  -->
							<div class="card text-left">
								<div class="card-header" id="heading14">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse14" aria-expanded="true" aria-controls="collapse14">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">14. projectR: An R/Bioconductor package for transfer learning via PCA, NMF, correlation, and clustering (2019)</p>
													 <p class="note font-weight-light"><span class="font-italic">Sharma, G., Colantuoni, C., Goff, L. A., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Fertig, E.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse14" class="collapse" aria-labelledby="heading14" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/726547" class="altmetric-embed pull-left"> </div>
										<p>MotivationDimension reduction techniques are widely used to interpret high-dimensional biological data. Features learned from these methods are used to discover both technical artifacts and novel biological phenomena. Such feature discovery is critically import to large single-cell datasets, where lack of a ground truth limits validation and interpretation. Transfer learning (TL) can be used to relate the features learned from one source dataset to a new target dataset to perform biologically-driven validation by evaluating their use in or association with additional sample annotations in that independent target dataset.\n\nResultsWe developed an R/Bioconductor package, projectR, to perform TL for analyses of genomics data via TL of clustering, correlation, and factorization methods. We then demonstrate the utility TL for integrated data analysis with an example for spatial single-cell analysis.\n\nAvailabilityprojectR is available on Bioconductor and at https://github.com/genesofeve/projectR.\n\nContactgsteinobrien@jhmi.edu; ejfertig@jhmi.edu</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/726547">10.1101/726547</a>
									</div>
								</div>
							</div>
							
							<!--  Card 15  -->
							<div class="card text-left">
								<div class="card-header" id="heading15">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse15" aria-expanded="true" aria-controls="collapse15">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">15. Combining mathematical and statistical modeling to simulate time course bulk and single cell gene expression data in cancer with CancerInSilico (2018)</p>
													 <p class="note font-weight-light"><span class="font-italic">Sherman, T. D., Kagohara, L. T., Cao, R., Cheng, R., Satriano, M., Considine, M., Krigsfeld, G., Ranaweera, R., Tang, Y., Jablonski, S., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Gaykalova, D., Weiner, L. M., Chung, C., Fertig, E.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse15" class="collapse" aria-labelledby="heading15" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/328807" class="altmetric-embed pull-left"> </div>
										<p>Bioinformatics techniques to analyze time course bulk and single cell omics data are advancing. The absence of a known ground truth of the dynamics of molecular changes challenges benchmarking their performance on real data. Realistic simulated time-course datasets are essential to assess the performance of time course bioinformatics algorithms. We develop an R/Bioconductor package, CancerInSilico, to simulate bulk and single cell transcriptional data from a known ground truth obtained from mathematical models of cellular systems. This package contains a general R infrastructure for running cell-based models and simulating gene expression data based on the model states. We show how to use this package to simulate a gene expression data set and consequently benchmark analysis methods on this data set with a known ground truth. The package is freely available via Bioconductor: http://bioconductor.org/packages/CancerInSilico/</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/328807">10.1101/328807</a>
									</div>
								</div>
							</div>
							
							<!--  Card 16  -->
							<div class="card text-left">
								<div class="card-header" id="heading16">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse16" aria-expanded="true" aria-controls="collapse16">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">16. Expression variation analysis for tumor heterogeneity in single-cell RNA-sequencing data (2018)</p>
													 <p class="note font-weight-light"><span class="font-italic">Davis-Marcisak, E. F., Orugunta, P., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Puram, S. V., Roussos Torres, E., Hopkins, A., Jaffee, E. M., Favorov, A. V., Afsari, B., Goff, L. A., Fertig, E. J.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse16" class="collapse" aria-labelledby="heading16" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/479287" class="altmetric-embed pull-left"> </div>
										<p>Tumor heterogeneity provides a complex challenge to cancer treatment and is a critical component of therapeutic response, disease recurrence, and patient survival. Single-cell RNA-sequencing (scRNA-seq) technologies reveal the prevalence of intra-and inter-tumor heterogeneity. Computational techniques are essential to quantify the differences in variation of these profiles between distinct cell types, tumor subtypes, and patients to fully characterize intra-and inter-tumor molecular heterogeneity. We devised a new algorithm, Expression Variation Analysis in Single Cells (EVAsc), to perform multivariate statistical analyses of differential variation of expression in gene sets for scRNA-seq. EVAsc has high sensitivity and specificity to detect pathways with true differential heterogeneity in simulated data. We then apply EVAsc to several public domain scRNA-seq tumor datasets to quantify the landscape of tumor heterogeneity in several key applications in cancer genomics, i.e. immunogenicity, cancer subtypes, and metastasis. Immune pathway heterogeneity in hematopoietic cell populations in breast tumors corresponded to the amount diversity present in the T-cell repertoire of each individual. In head and neck squamous cell carcinoma (HNSCC) patients, we found dramatic differences in pathway dysregulation across basal primary tumors. Within the basal primary tumors we also identified increased immune dysregulation in individuals with a high proportion of fibroblasts present in the tumor microenvironment. Moreover, cells in HNSCC primary tumors had significantly more heterogeneity across pathways than cells in metastases, consistent with a model of clonal outgrowth. These results demonstrate the broad utility of EVAsc to quantify inter-and intra-tumor heterogeneity from scRNA-seq data without reliance on low dimensional visualization.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/479287">10.1101/479287</a>
									</div>
								</div>
							</div>
							
							<!--  Card 17  -->
							<div class="card text-left">
								<div class="card-header" id="heading17">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse17" aria-expanded="true" aria-controls="collapse17">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">17. Decomposing cell identity for transfer learning across cellular measurements, platforms, tissues, and species. (2018)</p>
													 <p class="note font-weight-light"><span class="font-italic"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Clark, B. S., Sherman, T., Zibetti, C., Hu, Q., Sealfon, R., Liu, S., Qian, J., Colantuoni, C., Blackshaw, S., Goff, L. A., Fertig, E. J.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse17" class="collapse" aria-labelledby="heading17" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/395004" class="altmetric-embed pull-left"> </div>
										<p>New approaches are urgently needed to glean biological insights from the vast amounts of single cell RNA sequencing (scRNA-Seq) data now being generated. To this end, we propose that cell identity should map to a reduced set of factors which will describe both exclusive and shared biology of individual cells, and that the dimensions which contain these factors reflect biologically meaningful relationships across different platforms, tissues and species. To find a robust set of dependent factors in large-scale scRNA- Seq data, we developed a Bayesian non-negative matrix factorization (NMF) algorithm, scCoGAPS. Application of scCoGAPS to scRNA-Seq data obtained over the course of mouse retinal development identified gene expression signatures for factors associated with specific cell types and continuous biological processes. To test whether these signatures are shared across diverse cellular contexts, we developed projectR to map biologically disparate datasets into the factors learned by scCoGAPS. Because projecting these dimensions preserve relative distances between samples, biologically meaningful relationships/factors will stratify new data consistent with their underlying processes, allowing labels or information from one dataset to be used for annotation of the other--a machine learning concept called transfer learning. Using projectR, data from multiple datasets was used to annotate latent spaces and reveal novel parallels between developmental programs in other tissues, species and cellular assays. Using this approach we are able to transfer cell type and state designations across datasets to rapidly annotate cellular features in a new dataset without a priori knowledge of their type, identify a species-specific signature of microglial cells, and identify a previously undescribed subpopulation of neurosecretory cells within the lung. Together, these algorithms define biologically meaningful dimensions of cellular identity, state, and trajectories that persist across technologies, molecular features, and species.\n\nGRAPHICAL ABSTRACT\n\nO_FIG O_LINKSMALLFIG WIDTH=174 HEIGHT=200 SRC=\"FIGDIR/small/395004_ufig1.gif\" ALT=\"Figure 1\">\nView larger version (81K):\norg.highwire.dtl.DTLVardef@dd1c07org.highwire.dtl.DTLVardef@5b1109org.highwire.dtl.DTLVardef@bb6714org.highwire.dtl.DTLVardef@16c66f0_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/395004">10.1101/395004</a>
									</div>
								</div>
							</div>
							
							<!--  Card 18  -->
							<div class="card text-left">
								<div class="card-header" id="heading18">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse18" aria-expanded="true" aria-controls="collapse18">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">18. Comprehensive analysis of retinal development at single cell resolution identifies NFI factors as essential for mitotic exit and specification of late-born cells. (2018)</p>
													 <p class="note font-weight-light"><span class="font-italic">Clark, B., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Shiau, F., Cannon, G., Davis, E., Sherman, T., Rajaii, F., James-Esposito, R., Gronostajski, R., Fertig, E., Goff, L., Blackshaw, S.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse18" class="collapse" aria-labelledby="heading18" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/378950" class="altmetric-embed pull-left"> </div>
										<p>Precise temporal control of gene expression in neuronal progenitors is necessary for correct regulation of neurogenesis and cell fate specification. However, the extensive cellular heterogeneity of the developing CNS has posed a major obstacle to identifying the gene regulatory networks that control these processes. To address this, we used single cell RNA-sequencing to profile ten developmental stages encompassing the full course of retinal neurogenesis. This allowed us to comprehensively characterize changes in gene expression that occur during initiation of neurogenesis, changes in developmental competence, and specification and differentiation of each of the major retinal cell types. These data identify transitions in gene expression between early and late-stage retinal progenitors, as well as a classification of neurogenic progenitors. We identify here the NFI family of transcription factors (Nfia, Nfib, and Nfix) as genes with enriched expression within late RPCs, and show they are regulators of bipolar interneuron and Muller glia specification and the control of proliferative quiescence.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/378950">10.1101/378950</a>
									</div>
								</div>
							</div>
							
							<!--  Card 19  -->
							<div class="card text-left">
								<div class="card-header" id="heading19">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse19" aria-expanded="true" aria-controls="collapse19">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">19. Enter the matrix: Interpreting unsupervised feature learning with matrix decomposition to discover hidden knowledge in high-throughput omics data (2017)</p>
													 <p class="note font-weight-light"><span class="font-italic"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Arora, R., Culhane, A. C., Favorov, A., Greene, C., Goff, L. A., Li, Y., Ngom, A., Ochs, M. F., Xu, Y., Fertig, E.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse19" class="collapse" aria-labelledby="heading19" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/196915" class="altmetric-embed pull-left"> </div>
										<p>Omics data contains signal from the molecular, physical, and kinetic inter- and intra-cellular interactions that control biological systems. Matrix factorization techniques can reveal low-dimensional structure from high-dimensional data that reflect these interactions. These techniques can uncover new biological knowledge from diverse high-throughput omics data in topics ranging from pathway discovery to time course analysis. We review exemplary applications of matrix factorization for systems-level analyses. We discuss appropriate application of these methods, their limitations, and focus on analysis of results to facilitate optimal biological interpretation. The inference of biologically relevant features with matrix factorization enables discovery from high-throughput data beyond the limits of current biological knowledge--answering questions from high-dimensional data that we have not yet thought to ask.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/196915">10.1101/196915</a>
									</div>
								</div>
							</div>
							
							<!--  Card 20  -->
							<div class="card text-left">
								<div class="card-header" id="heading20">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse20" aria-expanded="true" aria-controls="collapse20">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">20. Untangling The Gene-Epigenome Networks: Timing Of Epigenetic Regulation Of Gene Expression In Acquired Cetuximab Resistance (2017)</p>
													 <p class="note font-weight-light"><span class="font-italic"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kagohara, L. T., Li, S., Thakar, M., Ranaweera, R., Ozawa, H., Cheng, H., Considine, M., Favorov, A., Danilova, L., Califano, J., Izumchenko, E., Gaykalova, D. A., Chung, C. H., Fertig, E. J.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse20" class="collapse" aria-labelledby="heading20" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/136564" class="altmetric-embed pull-left"> </div>
										<p>BACKGROUNDTargeted therapies specifically act by blocking the activity of proteins that are encoded by genes critical for tumorigenesis. However, most cancers acquire resistance and long-term disease remission is rarely observed. Understanding the time course of molecular changes responsible for the development of acquired resistance could enable optimization of patients treatment options. Clinically, acquired therapeutic resistance can only be studied at a single time point in resistant tumors. To determine the dynamics of these molecular changes, we obtained high throughput omics data weekly during the development of cetuximab resistance in a head and neck cancer in vitro model.\n\nRESULTSAn unsupervised algorithm, CoGAPS, was used to quantify the evolving transcriptional and epigenetic changes. Applying a PatternMarker statistic to the results from CoGAPS enabled novel heatmap-based visualization of the dynamics in these time course omics data. We demonstrate that transcriptional changes result from immediate therapeutic response or resistance, whereas epigenetic alterations only occur with resistance. Integrated analysis demonstrates delayed onset of changes in DNA methylation relative to transcription, suggesting that resistance is stabilized epigenetically.\n\nCONCLUSIONSGenes with epigenetic alterations associated with resistance that have concordant expression changes are hypothesized to stabilize resistance. These genes include FGFR1, which was associated with EGFR inhibitor resistance previously. Thus, integrated omics analysis distinguishes the timing of molecular drivers of resistance. Our findings provide a relevant towards better understanding of the time course progression of changes resulting in acquired resistance to targeted therapies. This is an important contribution to the development of alternative treatment strategies that would introduce new drugs before the resistant phenotype develops.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/136564">10.1101/136564</a>
									</div>
								</div>
							</div>
							
							<!--  Card 21  -->
							<div class="card text-left">
								<div class="card-header" id="heading21">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse21" aria-expanded="true" aria-controls="collapse21">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">21. Epigenetic Regulation of Gene Expression in Cancer: Techniques, Resources, and Analysis (2017)</p>
													 <p class="note font-weight-light"><span class="font-italic">Kagohara, L., <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kelley, D., Flam, E., Wick, H., Danilova, L., Easwaran, H., Favorov, A., Qian, J., Gaykalova, D., Fertig, E.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse21" class="collapse" aria-labelledby="heading21" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/114025" class="altmetric-embed pull-left"> </div>
										<p>Cancer is a complex disease, driven by aberrant activity in numerous signaling pathways in even individual malignant cells. Epigenetic changes are critical mediators of these functional changes that drive and maintain the malignant phenotype. Changes in DNA methylation, histone acetylation and methylation, non-coding RNAs, post-translational modifications are all epigenetic drivers in cancer, independent of changes in the DNA sequence. These epigenetic alterations, once thought to be crucial only for the malignant phenotype maintenance, are now recognized as critical also for disrupting essential pathways that protect the cells from uncontrolled growth, longer survival and establishment in distant sites from the original tissue. In this review, we focus on DNA methylation and chromatin structure in cancer. While associated with cancer, the precise functional role of these alterations is an area of active research using emerging high-throughput approaches and bioinformatics analysis tools. Therefore, this review describes these high-throughput measurement technologies, public domain databases for high-throughput epigenetic data in tumors and model systems, and bioinformatics algorithms for their analysis. Advances in bioinformatics data integration techniques that combine these epigenetic data with genomics data are essential to infer the function of specific epigenetic alterations in cancer, and are therefore also a focus of this review. Future studies using these emerging technologies will elucidate how alterations in the cancer epigenome cooperate with genetic aberrations to cause tumorigenesis initiation and progression. This deeper understanding is essential to future studies that will precisely infer patients prognosis and select patients who will be responsive to emerging epigenetic therapies.</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/114025">10.1101/114025</a>
									</div>
								</div>
							</div>
							
							<!--  Card 22  -->
							<div class="card text-left">
								<div class="card-header" id="heading22">
									<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse22" aria-expanded="true" aria-controls="collapse22">
											<span class="pull-left text-left">
													 <p class="font-weight-bold">22. PatternMarkers and Genome-Wide CoGAPS Analysis in Parallel Sets (GWCoGAPS) for data-driven detection of novel biomarkers via whole transcriptome Non-negative matrix factorization (NMF) (2016)</p>
													 <p class="note font-weight-light"><span class="font-italic"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Carey, J., Lee, W.-s., Considine, M., Favorov, A., Flam, E., Guo, T., Li, L., Marchionni, L., Sherman, T., Gaykalova, D., McKay, R., Ochs, M., Colantuoni, C., Fertig, E.</span>, bio<span style="color: red">R</span><sub><em>&Chi;</em></sub>iv</p>
											</span>
									</a>
								</div>

								<div id="collapse22" class="collapse" aria-labelledby="heading22" data-parent="#accordionPreprints">
									<div class="card-body" >
										<div data-badge-type="1" data-hide-no-mentions="true" data-doi="10.1101/083717" class="altmetric-embed pull-left"> </div>
										<p>SummaryNon-negative Matrix Factorization (NMF) algorithms associate gene expression with biological processes (e.g., time-course dynamics or disease subtypes). Compared with univariate associations, the relative weights of NMF solutions can obscure biomarkers. Therefore, we developed a novel PatternMarkers statistic to extract genes for biological validation and enhanced visualization of NMF results. Finding novel and unbiased gene markers with PatternMarkers requires whole-genome data. However, NMF algorithms typically do not converge for the tens of thousands of genes in genome-wide profiling. Therefore, we also developed Genome-Wide CoGAPS Analysis in Parallel Sets (GWCoGAPS), the first robust whole genome Bayesian NMF using the sparse, MCMC algorithm, CoGAPS. This software contains analytic and visualization tools including a Shiny web application, patternMatcher, which are generalized for any NMF. Using these tools, we find granular brain-region and cell-type specific signatures with corresponding biomarkers in GTex data, illustrating GWCoGAPS and patternMarkers ascertainment of data-driven biomarkers from whole-genome data.\n\nAvailabilityPatternMarkers & GWCoGAPS are in the CoGAPS Bioconductor package (3.5) under the GPL license.\n\nContactgsteinobrien@jhmi.edu; ccolantu@jhmi.edu; ejfertig@jhmi.edu</p>
										<a target="_blank" href="http://dx.doi.org/10.1101/083717">10.1101/083717</a>
									</div>
								</div>
							</div>
							
						</div>
						<!-- End Accordion -->


					</section>
			</div>
		</div>
</div>

		<!-- Footer -->
  <footer id="footer">
    <ul class="icons">
      <li><a href="https://twitter.com/intent/tweet?original_referer=https%3A%2F%2Fwww.sharelinkgenerator.com%2F&ref_src=twsrc%5Etfw&text=Share%20Link%20Generator%3A%20Facebook%2C%20Twitter%2C%20LinkedIn%2C%20Pinterest%2C%20and%20Email%20mailto%20forms&tw_p=tweetbutton&url=https%3A%2F%2Fwww.sharelinkgenerator.com%2F&via=twitterapi" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
      <!-- <li><a href="#" class="icon brands alt fa-facebook-f"><span class="label">Facebook</span></a></li> -->
      <!-- <li><a href="#" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li> -->
      <!-- <li><a href="#" class="icon brands alt fa-instagram"><span class="label">Instagram</span></a></li> -->
      <li><a href="http://github.com/genesofeve/" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
      <li><a href="mailto:gsteinobrien@jhmi.edu" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
    </ul>
    <ul class="copyright">
      <li>&copy; Stein-O'Brien Lab - JHU. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
    </ul>
  </footer>
		<!-- Scripts -->
<script src="assets/js/jquery.min.js"></script>
<script src="assets/js/jquery.scrolly.min.js"></script>
<script src="assets/js/jquery.dropotron.min.js"></script>
<script src="assets/js/jquery.scrollex.min.js"></script>
<script src="assets/js/browser.min.js"></script>
<script src="assets/js/breakpoints.min.js"></script>
<script src="assets/js/util.js"></script>
<script src="assets/js/main.js"></script>
<script src="assets/js/jquery-ui.min.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script>
  $( function() {
    $( "#accordion" ).accordion();
  } );
  </script>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-318359-8', 'auto');
ga('send', 'pageview');
</script>
	</body>
</html>
