<!DOCTYPE html>
 <html>
 <head>
		<meta charset="utf-8">
    	<meta http-equiv="X-UA-Compatible" content="IE=edge">
    	<meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Loyal Goff</title>
         <link rel="stylesheet" href="css/bootstrap.css">
         <link rel="stylesheet" href="css/gofflab_custom.css">
		 <link rel="stylesheet" href="css/bootstrap-responsive.css">

		 <script src="http://code.jquery.com/jquery-latest.js"></script>
		 <script src="js/bootstrap.min.js"></script>
		 <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

		 <title>
		 
  Publications

		 </title>
		<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
	    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
	    <!--[if lt IE 9]>
	      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
	      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
	    <![endif]-->
 </head>
 <body>
 <!--<div class="container">-->

	 <div class="navbar navbar-default navbar-static-top" role="navigation">
		 <div class="container-fluid">
			 <!-- Brand and toggle get grouped for better mobile display -->
			 <div class="navbar-header">
			 	<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
			 		<span class="sr-only">Toggle navigation</span>
        			<span class="glyphicon glyphicon-bar"></span>
        			<span class="glyphicon glyphicon-bar"></span>
        			<span class="glyphicon glyphicon-bar"></span>
			 	</button>
			 	<a class="navbar-brand" href="index.html">Goff Lab</a>
			 </div>

			 <!-- Collect the nav links, forms, and other content for toggling -->
			 <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			 	<ul class="nav navbar-nav">
				 	
				   		<li>
				   		<a href="index.html">Home</a></li>
		 			
				   		<li>
				   		<a href="research.html">Research</a></li>
		 			
				   		<li>
				   		<a href="people.html">People</a></li>
		 			
				   		<li class="active">
				   		<a href="publications.html">Publications</a></li>
		 			
				   		<li>
				   		<a href="software.html">Software</a></li>
		 			
				   		<li>
				   		<a href="contact.html">Contact</a></li>
		 			
				   		<li>
				   		<a href="join.html">Join</a></li>
		 			
				   		<li>
				   		<a href="links.html">Links</a></li>
		 			
				   		<li>
				   		<a href="news.html">News</a></li>
		 			
			 	</ul>
			 	<a class="navbar-brand pull-right" href="http://www.hopkinsmedicine.org/som/">
			 		<img src="images/JHM_GS_R_H.png" class="text-middle" style="max-height:35px;margin-top:-5px;">
			 	</a>
			 	<!-- Search
			 	<form class="navbar-form navbar-right" role="search">
		   			<span class="glyphicon glyphicon-search" style="color:#fff"></i>
		   			<input type="text" class="form-control search-query" placeholder="Search">
		   			<button type="submit" class="btn btn-default">Search</button>
		 		</form>
		 		--><!--end search -->
			 </div>
		 </div>
	 </div>
	 <div class="row-fluid">
		    <div class="col-md-1 hidden-xs hidden-sm">
		    	<img src="images/dna-left.png" style="position:relative;left:-20px;top:-20px;width:180px;max-width:80%;shape-outside: ellipse();">
			    

		        
         	</div>

         <!-- .span2 -->


	         <div class="col-md-11 col-sm-12">
	         	<img class="pull-right" src="images/complex1-right.png" style="position:relative;top:-20px;width:200px;max-width:50%;shape-outside: ellipse();">
		         
<h2>Publications</h2>
  <ul>
    <li><a href="http://europepmc.org/authors/0000-0003-2875-451X" target="_blank">EBI Summary page</a></li>
    <li><a href="https://scholar.google.com/citations?user=2ZVv2v8AAAAJ&hl=en" target="_blank">Google Scholar</a></li>
  </ul>
  <div class="panel-group" id="accordion">
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse1">
                 <em>A ketogenic diet rescues hippocampal memory defects in a mouse model of Kabuki syndrome.</em> (2017)</a>
                 <p class="note">Benjamin JS, Pilarowski GO, Carosso GA, Zhang L, Huso DL, Goff LA, Vernon HJ, Hansen KD, Bjornsson HT<br>
                 Proc Natl Acad Sci U S A. 2017 Jan 3;114(1):125-130. doi: 10.1073/pnas.1611431114. Epub 2016 Dec 20.</p>
        </span>
      </div>
      <div id="collapse1" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="27999180" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Kabuki syndrome is a Mendelian intellectual disability syndrome caused by mutations in either of two genes (KMT2D and KDM6A) involved in chromatin accessibility. We previously showed that an agent that promotes chromatin opening, the histone deacetylase inhibitor (HDACi) AR-42, ameliorates the deficiency of adult neurogenesis in the granule cell layer of the dentate gyrus and rescues hippocampal memory defects in a mouse model of Kabuki syndrome (Kmt2d+/betaGeo). Unlike a drug, a dietary intervention could be quickly transitioned to the clinic. Therefore, we have explored whether treatment with a ketogenic diet could lead to a similar rescue through increased amounts of beta-hydroxybutyrate, an endogenous HDACi. Here, we report that a ketogenic diet in Kmt2d+/betaGeo mice modulates H3ac and H3K4me3 in the granule cell layer, with concomitant rescue of both the neurogenesis defect and hippocampal memory abnormalities seen in Kmt2d+/betaGeo mice; similar effects on neurogenesis were observed on exogenous administration of beta-hydroxybutyrate. These data suggest that dietary modulation of epigenetic modifications through elevation of beta-hydroxybutyrate may provide a feasible strategy to treat the intellectual disability seen in Kabuki syndrome and related disorders.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1611431114 ">10.1073/pnas.1611431114 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse2">
                 <em>The DPYSL2 gene connects mTOR and schizophrenia.</em> (2016)</a>
                 <p class="note">Pham X, Song G, Lao S, Goff L, Zhu H, Valle D, Avramopoulos D<br>
                 Transl Psychiatry. 2016 Nov 1;6(11):e933. doi: 10.1038/tp.2016.204.</p>
        </span>
      </div>
      <div id="collapse2" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="27801893" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>We previously reported a schizophrenia-associated polymorphic CT di-nucleotide repeat (DNR) at the 5'-untranslated repeat (UTR) of DPYSL2, which responds to mammalian target of Rapamycin (mTOR) signaling with allelic differences in reporter assays. Now using microarray analysis, we show that the DNR alleles interact differentially with specific proteins, including the mTOR-related protein HuD/ELAVL4. We confirm the differential binding to HuD and other known mTOR effectors by electrophoretic mobility shift assays. We edit HEK293 cells by CRISPR/Cas9 to carry the schizophrenia risk variant (13DNR) and observe a significant reduction of the corresponding CRMP2 isoform. These edited cells confirm the response to mTOR inhibitors and show a twofold shortening of the cellular projections. Transcriptome analysis of these modified cells by RNA-seq shows changes in 12.7% of expressed transcripts at a false discovery rate of 0.05. These transcripts are enriched in immunity-related genes, overlap significantly with those modified by the schizophrenia-associated gene, ZNF804A, and have a reverse expression signature from that seen with antipsychotic drugs. Our results support the functional importance of the DPYSL2 DNR and a role for mTOR signaling in schizophrenia.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/tp.2016.204 ">10.1038/tp.2016.204 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse3">
                 <em>The long non-coding RNA Morrbid regulates Bim and short-lived myeloid cell lifespan.</em> (2016)</a>
                 <p class="note">Kotzin JJ, Spencer SP, McCright SJ, Kumar DB, Collet MA, Mowel WK, Elliott EN, Uyar A, Makiya MA, Dunagin MC, Harman CC, Virtue AT, Zhu S, Bailis W, Stein J, Hughes C, Raj A, Wherry EJ, Goff LA, Klion AD, Rinn JL, Williams A, Flavell RA, Henao-Mejia J<br>
                 Nature. 2016 Sep 8;537(7619):239-243. doi: 10.1038/nature19346. Epub 2016 Aug 15.</p>
        </span>
      </div>
      <div id="collapse3" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="27525555" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Neutrophils, eosinophils and 'classical' monocytes collectively account for about 70% of human blood leukocytes and are among the shortest-lived cells in the body. Precise regulation of the lifespan of these myeloid cells is critical to maintain protective immune responses and minimize the deleterious consequences of prolonged inflammation. However, how the lifespan of these cells is strictly controlled remains largely unknown. Here we identify a long non-coding RNA that we termed Morrbid, which tightly controls the survival of neutrophils, eosinophils and classical monocytes in response to pro-survival cytokines in mice. To control the lifespan of these cells, Morrbid regulates the transcription of the neighbouring pro-apoptotic gene, Bcl2l11 (also known as Bim), by promoting the enrichment of the PRC2 complex at the Bcl2l11 promoter to maintain this gene in a poised state. Notably, Morrbid regulates this process in cis, enabling allele-specific control of Bcl2l11 transcription. Thus, in these highly inflammatory cells, changes in Morrbid levels provide a locus-specific regulatory mechanism that allows rapid control of apoptosis in response to extracellular pro-survival signals. As MORRBID is present in humans and dysregulated in individuals with hypereosinophilic syndrome, this long non-coding RNA may represent a potential therapeutic target for inflammatory disorders characterized by aberrant short-lived myeloid cell lifespan.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nature19346 ">10.1038/nature19346 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse4">
                 <em>Investigating long noncoding RNAs using animal models.</em> (2016)</a>
                 <p class="note">Feyder M, Goff LA<br>
                 J Clin Invest. 2016 Aug 1;126(8):2783-91. doi: 10.1172/JCI84422. Epub 2016 Aug 1.</p>
        </span>
      </div>
      <div id="collapse4" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="27479747" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>The number of long noncoding RNAs (lncRNAs) has grown rapidly; however, our understanding of their function remains limited. Although cultured cells have facilitated investigations of lncRNA function at the molecular level, the use of animal models provides a rich context in which to investigate the phenotypic impact of these molecules. Promising initial studies using animal models demonstrated that lncRNAs influence a diverse number of phenotypes, ranging from subtle dysmorphia to viability. Here, we highlight the diversity of animal models and their unique advantages, discuss the use of animal models to profile lncRNA expression, evaluate experimental strategies to manipulate lncRNA function in vivo, and review the phenotypes attributable to lncRNAs. Despite a limited number of studies leveraging animal models, lncRNAs are already recognized as a notable class of molecules with important implications for health and disease.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1172/JCI84422 ">10.1172/JCI84422 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse5">
                 <em>Long noncoding RNAs: Central to nervous system development.</em> (2016)</a>
                 <p class="note">Hart RP, Goff LA<br>
                 Int J Dev Neurosci. 2016 Dec;55:109-116. doi: 10.1016/j.ijdevneu.2016.06.001. Epub 2016 Jun 11.</p>
        </span>
      </div>
      <div id="collapse5" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="27296516" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>The development of the central nervous system (CNS) is a complex orchestration of stem cells, transcription factors, growth/differentiation factors, and epigenetic control. Noncoding RNAs have been identified, classified, and studied for their functional roles in many systems including the CNS. In particular, the class of long noncoding RNAs (lncRNAs) has generated both enthusiasm and skepticism due to the unexpected discovery, the diversity of mechanisms, and the lower level of expression than found in protein-coding RNAs. Here we describe evidence supporting the role of lncRNAs in driving CNS-specific differentiation. It is clear that lncRNAs exhibit a functional diversity that makes their study and compartmentalization more challenging than other classes of noncoding RNAs. We predict, however, that lncRNAs will be essential for the characterization of discrete neuronal cell types in the age of single-cell transcriptomics and that these regulatory RNAs contribute to the multitude of functional mechanisms during CNS differentiation that will rival the diversities of protein-based mechanisms.</p>
          
              <a target="_blank" href="http://dx.doi.org/S0736-5748(16)30109-5 [pii] 10.1016/j.ijdevneu.2016.06.001 ">S0736-5748(16)30109-5 [pii] 10.1016/j.ijdevneu.2016.06.001 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse6">
                 <em>Creation and characterization of an airway epithelial cell line for stable expression of CFTR variants.</em> (2016)</a>
                 <p class="note">Gottschalk LB, Vecchio-Pagan B, Sharma N, Han ST, Franca A, Wohler ES, Batista DA, Goff LA, Cutting GR<br>
                 J Cyst Fibros. 2016 May;15(3):285-94. doi: 10.1016/j.jcf.2015.11.010. Epub 2015 Dec 13.</p>
        </span>
      </div>
      <div id="collapse6" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="26694805" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>BACKGROUND: Analysis of the functional consequences and treatment response of rare CFTR variants is challenging due to the limited availability of primary airways cells. METHODS: A Flp recombination target (FRT) site for stable expression of CFTR was incorporated into an immortalized CF bronchial epithelial cell line (CFBE41o-). CFTR cDNA was integrated into the FRT site. Expression was evaluated by western blotting and confocal microscopy and function measured by short circuit current. RNA sequencing was used to compare the transcriptional profile of the resulting CF8Flp cell line to primary cells and tissues. RESULTS: Functional CFTR was expressed from integrated cDNA at the FRT site of the CF8Flp cell line at levels comparable to that seen in native airway cells. CF8Flp cells expressing WT-CFTR have a stable transcriptome comparable to that of primary cultured airway epithelial cells, including genes that play key roles in CFTR pathways. CONCLUSION: CF8Flp cells provide a viable substitute for primary CF airway cells for the analysis of CFTR variants in a native context.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1016/j.jcf.2015.11.010 ">10.1016/j.jcf.2015.11.010 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse7">
                 <em>Linking RNA biology to lncRNAs.</em> (2015)</a>
                 <p class="note">Goff LA, Rinn JL<br>
                 Genome Res. 2015 Oct;25(10):1456-65. doi: 10.1101/gr.191122.115.</p>
        </span>
      </div>
      <div id="collapse7" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="26430155" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>The regulatory potential of RNA has never ceased to amaze: from RNA catalysis, to RNA-mediated splicing, to RNA-based silencing of an entire chromosome during dosage compensation. More recently, thousands of long noncoding RNA (lncRNA) transcripts have been identified, the majority with unknown function. Thus, it is tempting to think that these lncRNAs represent a cadre of new factors that function through ribonucleic mechanisms. Some evidence points to several lncRNAs with tantalizing physiological contributions and thought-provoking molecular modalities. However, dissecting the RNA biology of lncRNAs has been difficult, and distinguishing the independent contributions of functional RNAs from underlying DNA elements, or the local act of transcription, is challenging. Here, we aim to survey the existing literature and highlight future approaches that will be needed to link the RNA-based biology and mechanisms of lncRNAs in vitro and in vivo.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1101/gr.191122.115 ">10.1101/gr.191122.115 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse8">
                 <em>Spatiotemporal expression and transcriptional perturbations by long noncoding RNAs in the mouse brain.</em> (2015)</a>
                 <p class="note">Goff LA, Groff AF, Sauvageau M, Trayes-Gibson Z, Sanchez-Gomez DB, Morse M, Martin RD, Elcavage LE, Liapis SC, Gonzalez-Celeiro M, Plana O, Li E, Gerhardinger C, Tomassy GS, Arlotta P, Rinn JL<br>
                 Proc Natl Acad Sci U S A. 2015 Jun 2;112(22):6855-62.</p>
        </span>
      </div>
      <div id="collapse8" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="26034286" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Long noncoding RNAs (lncRNAs) have been implicated in numerous cellular processes including brain development. However, the in vivo expression dynamics and molecular pathways regulated by these loci are not well understood. Here, we leveraged a cohort of 13 lncRNAnull mutant mouse models to investigate the spatiotemporal expression of lncRNAs in the developing and adult brain and the transcriptome alterations resulting from the loss of these lncRNA loci. We show that several lncRNAs are differentially expressed both in time and space, with some presenting highly restricted expression in only selected brain regions. We further demonstrate altered regulation of genes for a large variety of cellular pathways and processes upon deletion of the lncRNA loci. Finally, we found that 4 of the 13 lncRNAs significantly affect the expression of several neighboring proteincoding genes in a cis-like manner. By providing insight into the endogenous expression patterns and the transcriptional perturbations caused by deletion of the lncRNA locus in the developing and postnatal mammalian brain, these data provide a resource to facilitate future examination of the specific functional relevance of these genes in neural development, brain function, and disease.</p>
          
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26034286">Article</a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse9">
                 <em>DeCoN: genome-wide analysis of in vivo transcriptional dynamics during pyramidal neuron fate selection in neocortex.</em> (2015)</a>
                 <p class="note">Molyneaux BJ, Goff LA, Brettler AC, Chen HH, Brown JR, Hrvatin S, Rinn JL, Arlotta P<br>
                 Neuron. 2015 Jan 21;85(2):275-88. doi: 10.1016/j.neuron.2014.12.024. Epub 2014 Dec 31.</p>
        </span>
      </div>
      <div id="collapse9" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="25556833" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>UNLABELLED: Neuronal development requires a complex choreography of transcriptional decisions to obtain specific cellular identities. Realizing the ultimate goal of identifying genome-wide signatures that define and drive specific neuronal fates has been hampered by enormous complexity in both time and space during development. Here, we have paired high-throughput purification of pyramidal neuron subclasses with deep profiling of spatiotemporal transcriptional dynamics during corticogenesis to resolve lineage choice decisions. We identified numerous features ranging from spatial and temporal usage of alternative mRNA isoforms and promoters to a host of mRNA genes modulated during fate specification. Notably, we uncovered numerous long noncoding RNAs with restricted temporal and cell-type-specific expression. To facilitate future exploration, we provide an interactive online database to enable multidimensional data mining and dissemination. This multifaceted study generates a powerful resource and informs understanding of the transcriptional regulation underlying pyramidal neuron diversity in the neocortex. VIDEO ABSTRACT:</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1016/j.neuron.2014.12.024 ">10.1016/j.neuron.2014.12.024 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse10">
                 <em>Gene co-regulation by Fezf2 selects neurotransmitter identity and connectivity of corticospinal neurons.</em> (2014)</a>
                 <p class="note">Lodato S, Molyneaux BJ, Zuccaro E, Goff LA, Chen HH, Yuan W, Meleski A, Takahashi E, Mahony S, Rinn JL, Gifford DK, Arlotta P<br>
                 Nat Neurosci. 2014 Aug;17(8):1046-54. doi: 10.1038/nn.3757. Epub 2014 Jul 6.</p>
        </span>
      </div>
      <div id="collapse10" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24997765" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>The neocortex contains an unparalleled diversity of neuronal subtypes, each defined by distinct traits that are developmentally acquired under the control of subtype-specific and pan-neuronal genes. The regulatory logic that orchestrates the expression of these unique combinations of genes is unknown for any class of cortical neuron. Here, we report that Fezf2 is a selector gene able to regulate the expression of gene sets that collectively define mouse corticospinal motor neurons (CSMN). We find that Fezf2 directly induces the glutamatergic identity of CSMN via activation of Vglut1 (Slc17a7) and inhibits a GABAergic fate by repressing transcription of Gad1. In addition, we identify the axon guidance receptor EphB1 as a target of Fezf2 necessary to execute the ipsilateral extension of the corticospinal tract. Our data indicate that co-regulated expression of neuron subtype-specific and pan-neuronal gene batteries by a single transcription factor is one component of the regulatory logic responsible for the establishment of CSMN identity.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nn.3757 ">10.1038/nn.3757 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse11">
                 <em>A positive feedback mechanism that regulates expression of miR-9 during neurogenesis.</em> (2014)</a>
                 <p class="note">Davila JL, Goff LA, Ricupero CL, Camarillo C, Oni EN, Swerdel MR, Toro-Ramos AJ, Li J, Hart RP<br>
                 PLoS One. 2014 Apr 8;9(4):e94348. doi: 10.1371/journal.pone.0094348. eCollection 2014.</p>
        </span>
      </div>
      <div id="collapse11" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24714615" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>MiR-9, a neuron-specific miRNA, is an important regulator of neurogenesis. In this study we identify how miR-9 is regulated during early differentiation from a neural stem-like cell. We utilized two immortalized rat precursor clones, one committed to neurogenesis (L2.2) and another capable of producing both neurons and non-neuronal cells (L2.3), to reproducibly study early neurogenesis. Exogenous miR-9 is capable of increasing neurogenesis from L2.3 cells. Only one of three genomic loci capable of encoding miR-9 was regulated during neurogenesis and the promoter region of this locus contains sufficient functional elements to drive expression of a luciferase reporter in a developmentally regulated pattern. Furthermore, among a large number of potential regulatory sites encoded in this sequence, Mef2 stood out because of its known pro-neuronal role. Of four Mef2 paralogs, we found only Mef2C mRNA was regulated during neurogenesis. Removal of predicted Mef2 binding sites or knockdown of Mef2C expression reduced miR-9-2 promoter activity. Finally, the mRNA encoding the Mef2C binding partner HDAC4 was shown to be targeted by miR-9. Since HDAC4 protein could be co-immunoprecipitated with Mef2C protein or with genomic Mef2 binding sequences, we conclude that miR-9 regulation is mediated, at least in part, by Mef2C binding but that expressed miR-9 has the capacity to reduce inhibitory HDAC4, stabilizing its own expression in a positive feedback mechanism.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0094348 ">10.1371/journal.pone.0094348 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse12">
                 <em>Topological organization of multichromosomal regions by the long intergenic noncoding RNA Firre.</em> (2014)</a>
                 <p class="note">Hacisuleyman E, Goff LA, Trapnell C, Williams A, Henao-Mejia J, Sun L, McClanahan P, Hendrickson DG, Sauvageau M, Kelley DR, Morse M, Engreitz J, Lander ES, Guttman M, Lodish HF, Flavell R, Raj A, Rinn JL<br>
                 Nat Struct Mol Biol. 2014 Feb;21(2):198-206. doi: 10.1038/nsmb.2764. Epub 2014 Jan 26.</p>
        </span>
      </div>
      <div id="collapse12" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24463464" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>RNA, including long noncoding RNA (lncRNA), is known to be an abundant and important structural component of the nuclear matrix. However, the molecular identities, functional roles and localization dynamics of lncRNAs that influence nuclear architecture remain poorly understood. Here, we describe one lncRNA, Firre, that interacts with the nuclear-matrix factor hnRNPU through a 156-bp repeating sequence and localizes across an ~5-Mb domain on the X chromosome. We further observed Firre localization across five distinct trans-chromosomal loci, which reside in spatial proximity to the Firre genomic locus on the X chromosome. Both genetic deletion of the Firre locus and knockdown of hnRNPU resulted in loss of colocalization of these trans-chromosomal interacting loci. Thus, our data suggest a model in which lncRNAs such as Firre can interface with and modulate nuclear architecture across chromosomes.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nsmb.2764 ">10.1038/nsmb.2764 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse13">
                 <em>RNase-mediated protein footprint sequencing reveals protein-binding sites throughout the human transcriptome.</em> (2014)</a>
                 <p class="note">Silverman IM, Li F, Alexander A, Goff L, Trapnell C, Rinn JL, Gregory BD<br>
                 Genome Biol. 2014 Jan 7;15(1):R3. doi: 10.1186/gb-2014-15-1-r3.</p>
        </span>
      </div>
      <div id="collapse13" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24393486" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Although numerous approaches have been developed to map RNA-binding sites of individual RNA-binding proteins (RBPs), few methods exist that allow assessment of global RBP-RNA interactions. Here, we describe PIP-seq, a universal, high-throughput, ribonuclease-mediated protein footprint sequencing approach that reveals RNA-protein interaction sites throughout a transcriptome of interest. We apply PIP-seq to the HeLa transcriptome and compare binding sites found using different cross-linkers and ribonucleases. From this analysis, we identify numerous putative RBP-binding motifs, reveal novel insights into co-binding by RBPs, and uncover a significant enrichment for disease-associated polymorphisms within RBP interaction sites.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1186/gb-2014-15-1-r3 ">10.1186/gb-2014-15-1-r3 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse14">
                 <em>Multiple knockout mouse models reveal lincRNAs are required for life and brain development.</em> (2013)</a>
                 <p class="note">Sauvageau M, Goff LA, Lodato S, Bonev B, Groff AF, Gerhardinger C, Sanchez-Gomez DB, Hacisuleyman E, Li E, Spence M, Liapis SC, Mallard W, Morse M, Swerdel MR, D'Ecclessis MF, Moore JC, Lai V, Gong G, Yancopoulos GD, Frendewey D, Kellis M, Hart RP, Valenzuela DM, Arlotta P, Rinn JL<br>
                 Elife. 2013 Dec 31;2:e01749. doi: 10.7554/eLife.01749.</p>
        </span>
      </div>
      <div id="collapse14" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24381249" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Many studies are uncovering functional roles for long noncoding RNAs (lncRNAs), yet few have been tested for in vivo relevance through genetic ablation in animal models. To investigate the functional relevance of lncRNAs in various physiological conditions, we have developed a collection of 18 lncRNA knockout strains in which the locus is maintained transcriptionally active. Initial characterization revealed peri- and postnatal lethal phenotypes in three mutant strains (Fendrr, Peril, and Mdgt), the latter two exhibiting incomplete penetrance and growth defects in survivors. We also report growth defects for two additional mutant strains (linc-Brn1b and linc-Pint). Further analysis revealed defects in lung, gastrointestinal tract, and heart in Fendrr(-/-) neonates, whereas linc-Brn1b(-/-) mutants displayed distinct abnormalities in the generation of upper layer II-IV neurons in the neocortex. This study demonstrates that lncRNAs play critical roles in vivo and provides a framework and impetus for future larger-scale functional investigation into the roles of lncRNA molecules. DOI: http://dx.doi.org/10.7554/eLife.01749.001.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.7554/eLife.01749 ">10.7554/eLife.01749 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse15">
                 <em>Poly-combing the genome for RNA.</em> (2013)</a>
                 <p class="note">Goff LA, Rinn JL<br>
                 Nat Struct Mol Biol. 2013 Dec;20(12):1344-6. doi: 10.1038/nsmb.2728.</p>
        </span>
      </div>
      <div id="collapse15" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24304912" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>An unresolved question in mammalian epigenetic regulation is how ubiquitously expressed chromatin-modifying complexes such as Polycomb group complex 2 (PRC2) find their specific target sites across an intricate choreography of localization events in time and space. Two recent studies now provide critical new insights into an intriguing genome-wide role for RNA in PRC2 regulation.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nsmb.2728 ">10.1038/nsmb.2728 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse16">
                 <em>DNMT1-interacting RNAs block gene-specific DNA methylation.</em> (2013)</a>
                 <p class="note">Di Ruscio A, Ebralidze AK, Benoukraf T, Amabile G, Goff LA, Terragni J, Figueroa ME, De Figueiredo Pontes LL, Alberich-Jorda M, Zhang P, Wu M, D'Alo F, Melnick A, Leone G, Ebralidze KK, Pradhan S, Rinn JL, Tenen DG<br>
                 Nature. 2013 Nov 21;503(7476):371-6. doi: 10.1038/nature12598. Epub 2013 Oct 9.</p>
        </span>
      </div>
      <div id="collapse16" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="24107992" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>DNA methylation was first described almost a century ago; however, the rules governing its establishment and maintenance remain elusive. Here we present data demonstrating that active transcription regulates levels of genomic methylation. We identify a novel RNA arising from the CEBPA gene locus that is critical in regulating the local DNA methylation profile. This RNA binds to DNMT1 and prevents CEBPA gene locus methylation. Deep sequencing of transcripts associated with DNMT1 combined with genome-scale methylation and expression profiling extend the generality of this finding to numerous gene loci. Collectively, these results delineate the nature of DNMT1-RNA interactions and suggest strategies for gene-selective demethylation of therapeutic targets in human diseases.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nature12598 ">10.1038/nature12598 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse17">
                 <em>The microRNA miR-181 is a critical cellular metabolic rheostat essential for NKT cell ontogenesis and lymphocyte development and homeostasis.</em> (2013)</a>
                 <p class="note">Henao-Mejia J, Williams A, Goff LA, Staron M, Licona-Limon P, Kaech SM, Nakayama M, Rinn JL, Flavell RA<br>
                 Immunity. 2013 May 23;38(5):984-97. doi: 10.1016/j.immuni.2013.02.021. Epub 2013 Apr 25.</p>
        </span>
      </div>
      <div id="collapse17" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="23623381" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Regulation of metabolic pathways in the immune system provides a mechanism to actively control cellular function, growth, proliferation, and survival. Here, we report that miR-181 is a nonredundant determinant of cellular metabolism and is essential for supporting the biosynthetic demands of early NKT cell development. As a result, miR-181-deficient mice showed a complete absence of mature NKT cells in the thymus and periphery. Mechanistically, miR-181 modulated expression of the phosphatase PTEN to control PI3K signaling, which was a primary stimulus for anabolic metabolism in immune cells. Thus miR-181-deficient mice also showed severe defects in lymphoid development and T cell homeostasis associated with impaired PI3K signaling. These results uncover miR-181 as essential for NKT cell development and establish this family of miRNAs as central regulators of PI3K signaling and global metabolic fitness during development and homeostasis.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1016/j.immuni.2013.02.021 ">10.1016/j.immuni.2013.02.021 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse18">
                 <em>Long noncoding RNAs regulate adipogenesis.</em> (2013)</a>
                 <p class="note">Sun L, Goff LA, Trapnell C, Alexander R, Lo KA, Hacisuleyman E, Sauvageau M, Tazon-Vega B, Kelley DR, Hendrickson DG, Yuan B, Kellis M, Lodish HF, Rinn JL<br>
                 Proc Natl Acad Sci U S A. 2013 Feb 26;110(9):3387-92. doi: 10.1073/pnas.1222643110. Epub 2013 Feb 11.</p>
        </span>
      </div>
      <div id="collapse18" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="23401553" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>The prevalence of obesity has led to a surge of interest in understanding the detailed mechanisms underlying adipocyte development. Many protein-coding genes, mRNAs, and microRNAs have been implicated in adipocyte development, but the global expression patterns and functional contributions of long noncoding RNA (lncRNA) during adipogenesis have not been explored. Here we profiled the transcriptome of primary brown and white adipocytes, preadipocytes, and cultured adipocytes and identified 175 lncRNAs that are specifically regulated during adipogenesis. Many lncRNAs are adipose-enriched, strongly induced during adipogenesis, and bound at their promoters by key transcription factors such as peroxisome proliferator-activated receptor gamma (PPARgamma) and CCAAT/enhancer-binding protein alpha (CEBPalpha). RNAi-mediated loss of function screens identified functional lncRNAs with varying impact on adipogenesis. Collectively, we have identified numerous lncRNAs that are functionally required for proper adipogenesis.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1222643110 ">10.1073/pnas.1222643110 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse19">
                 <em>Differential analysis of gene regulation at transcript resolution with RNA-seq.</em> (2013)</a>
                 <p class="note">Trapnell C, Hendrickson DG, Sauvageau M, Goff L, Rinn JL, Pachter L<br>
                 Nat Biotechnol. 2013 Jan;31(1):46-53. doi: 10.1038/nbt.2450. Epub 2012 Dec 9.</p>
        </span>
      </div>
      <div id="collapse19" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="23222703" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Differential analysis of gene and transcript expression using high-throughput RNA sequencing (RNA-seq) is complicated by several sources of measurement variability and poses numerous statistical challenges. We present Cuffdiff 2, an algorithm that estimates expression at transcript-level resolution and controls for variability evident across replicate libraries. Cuffdiff 2 robustly identifies differentially expressed transcripts and genes and reveals differential splicing and promoter-preference changes. We demonstrate the accuracy of our approach through differential analysis of lung fibroblasts in response to loss of the developmental transcription factor HOXA1, which we show is required for lung fibroblast and HeLa cell cycle progression. Loss of HOXA1 results in significant expression level changes in thousands of individual transcripts, along with isoform switching events in key regulators of the cell cycle. Cuffdiff 2 performs robust differential analysis in RNA-seq experiments at transcript resolution, revealing a layer of regulation not readily observable with other high-throughput technologies.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nbt.2450 ">10.1038/nbt.2450 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse20">
                 <em>Computational analysis of noncoding RNAs.</em> (2012)</a>
                 <p class="note">Washietl S, Will S, Hendrix DA, Goff LA, Rinn JL, Berger B, Kellis M<br>
                 Wiley Interdiscip Rev RNA. 2012 Nov-Dec;3(6):759-78. doi: 10.1002/wrna.1134. Epub 2012 Sep 18.</p>
        </span>
      </div>
      <div id="collapse20" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="22991327" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Noncoding RNAs have emerged as important key players in the cell. Understanding their surprisingly diverse range of functions is challenging for experimental and computational biology. Here, we review computational methods to analyze noncoding RNAs. The topics covered include basic and advanced techniques to predict RNA structures, annotation of noncoding RNAs in genomic data, mining RNA-seq data for novel transcripts and prediction of transcript structures, computational aspects of microRNAs, and database resources.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1002/wrna.1134 ">10.1002/wrna.1134 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse21">
                 <em>Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</em> (2012)</a>
                 <p class="note">Trapnell C, Roberts A, Goff L, Pertea G, Kim D, Kelley DR, Pimentel H, Salzberg SL, Rinn JL, Pachter L<br>
                 Nat Protoc. 2012 Mar 1;7(3):562-78. doi: 10.1038/nprot.2012.016.</p>
        </span>
      </div>
      <div id="collapse21" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="22383036" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Recent advances in high-throughput cDNA sequencing (RNA-seq) can reveal new genes and splice variants and quantify expression genome-wide in a single assay. The volume and complexity of data from RNA-seq experiments necessitate scalable, fast and mathematically principled analysis software. TopHat and Cufflinks are free, open-source software tools for gene discovery and comprehensive expression analysis of high-throughput mRNA sequencing (RNA-seq) data. Together, they allow biologists to identify new genes and new splice variants of known ones, as well as compare gene and transcript expression under two or more conditions. This protocol describes in detail how to use TopHat and Cufflinks to perform such analyses. It also covers several accessory tools and utilities that aid in managing data, including CummeRbund, a tool for visualizing RNA-seq analysis results. Although the procedure assumes basic informatics skills, these tools assume little to no background with RNA-seq analysis and are meant for novices and experts alike. The protocol begins with raw sequencing reads and produces a transcriptome assembly, lists of differentially expressed and regulated genes and transcripts, and publication-quality visualizations of analysis results. The protocol's execution time depends on the volume of transcriptome sequencing data and available computing resources but takes less than 1 d of computer time for typical experiments and approximately 1 h of hands-on time.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1038/nprot.2012.016 ">10.1038/nprot.2012.016 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse22">
                 <em>Expression profiling of synaptic microRNAs from the adult rat brain identifies regional differences and seizure-induced dynamic modulation.</em> (2012)</a>
                 <p class="note">Pichardo-Casas I, Goff LA, Swerdel MR, Athie A, Davila J, Ramos-Brossier M, Lapid-Volosin M, Friedman WJ, Hart RP, Vaca L<br>
                 Brain Res. 2012 Feb 3;1436:20-33. doi: 10.1016/j.brainres.2011.12.001. Epub 2011 Dec 9.</p>
        </span>
      </div>
      <div id="collapse22" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="22197703" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>In recent years, microRNAs or miRNAs have been proposed to target neuronal mRNAs localized near the synapse, exerting a pivotal role in modulating local protein synthesis, and presumably affecting adaptive mechanisms such as synaptic plasticity. In the present study we have characterized the distribution of miRNAs in five regions of the adult mammalian brain and compared the relative abundance between total fractions and purified synaptoneurosomes (SN), using three different methodologies. The results show selective enrichment or depletion of some miRNAs when comparing total versus SN fractions. These miRNAs were different for each brain region explored. Changes in distribution could not be attributed to simple diffusion or to a targeting sequence inside the miRNAs. In silico analysis suggest that the differences in distribution may be related to the preferential concentration of synaptically localized mRNA targeted by the miRNAs. These results favor a model of co-transport of the miRNA-mRNA complex to the synapse, although further studies are required to validate this hypothesis. Using an in vivo model for increasing excitatory activity in the cortex and the hippocampus indicates that the distribution of some miRNAs can be modulated by enhanced neuronal (epileptogenic) activity. All these results demonstrate the dynamic modulation in the local distribution of miRNAs from the adult brain, which may play key roles in controlling localized protein synthesis at the synapse.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1016/j.brainres.2011.12.001 ">10.1016/j.brainres.2011.12.001 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse23">
                 <em>Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses.</em> (2011)</a>
                 <p class="note">Cabili MN, Trapnell C, Goff L, Koziol M, Tazon-Vega B, Regev A, Rinn JL<br>
                 Genes Dev. 2011 Sep 15;25(18):1915-27. doi: 10.1101/gad.17446611. Epub 2011 Sep 2.</p>
        </span>
      </div>
      <div id="collapse23" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="21890647" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Large intergenic noncoding RNAs (lincRNAs) are emerging as key regulators of diverse cellular processes. Determining the function of individual lincRNAs remains a challenge. Recent advances in RNA sequencing (RNA-seq) and computational methods allow for an unprecedented analysis of such transcripts. Here, we present an integrative approach to define a reference catalog of >8000 human lincRNAs. Our catalog unifies previously existing annotation sources with transcripts we assembled from RNA-seq data collected from approximately 4 billion RNA-seq reads across 24 tissues and cell types. We characterize each lincRNA by a panorama of >30 properties, including sequence, structural, transcriptional, and orthology features. We found that lincRNA expression is strikingly tissue-specific compared with coding genes, and that lincRNAs are typically coexpressed with their neighboring genes, albeit to an extent similar to that of pairs of neighboring protein-coding genes. We distinguish an additional subset of transcripts that have high evolutionary conservation but may include short ORFs and may serve as either lincRNAs or small peptides. Our integrated, comprehensive, yet conservative reference catalog of human lincRNAs reveals the global properties of lincRNAs and will facilitate experimental studies and further functional classification of these genes.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1101/gad.17446611 ">10.1101/gad.17446611 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse24">
                 <em>Differential regulation of microRNA stability.</em> (2010)</a>
                 <p class="note">Bail S, Swerdel M, Liu H, Jiao X, Goff LA, Hart RP, Kiledjian M<br>
                 RNA. 2010 May;16(5):1032-9. doi: 10.1261/rna.1851510. Epub 2010 Mar 26.</p>
        </span>
      </div>
      <div id="collapse24" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="20348442" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>MicroRNAs (miRNAs) are endogenous single-stranded RNA molecules of about 21 nucleotides in length that are fundamental post-transcriptional regulators of gene expression. Although the transcriptional and processing events involved in the generation of miRNAs have been extensively studied, very little is known pertaining to components that regulate the stability of individual miRNAs. All RNAs have distinct inherent half-lives that dictate their level of accumulation and miRNAs would be expected to follow a similar principle. Here we demonstrate that although most miRNA appear to be stable, like mRNAs, miRNAs possess differential stability in human cells. In particular, we found that miR-382, a miRNA that contributes to HIV-1 provirus latency, is unstable in cells. To determine the region of miR-382 responsible for its rapid decay, we developed a cell-free system that recapitulated the observed cell-based-regulated miR-382 turnover. The system utilizes in vitro-processed mature miRNA derived from pre-miRNA and follows the decay of the processed miRNA. Using this system, we demonstrate that instability of miR-382 is driven by sequences outside its seed region and required the 3' terminal seven nucleotides where mutations in this region increased the stability of the RNA. Moreover, the exosome 3'-5' exoribonuclease complex was identified as the primary nuclease involved in miR-382 decay with a more modest contribution by the Xrn1 and no detectable contribution by Xrn2. These studies provide evidence for an miRNA element essential for rapid miRNA decay and implicate the exosome in this process. The development of a biochemically amendable system to analyze the mechanism of differential miRNA stability provides an important step in efforts to regulate gene expression by modulating miRNA stability.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1261/rna.1851510 ">10.1261/rna.1851510 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse25">
                 <em>Ago2 immunoprecipitation identifies predicted microRNAs in human embryonic stem cells and neural precursors.</em> (2009)</a>
                 <p class="note">Goff LA, Davila J, Swerdel MR, Moore JC, Cohen RI, Wu H, Sun YE, Hart RP<br>
                 PLoS One. 2009 Sep 28;4(9):e7192. doi: 10.1371/journal.pone.0007192.</p>
        </span>
      </div>
      <div id="collapse25" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="19784364" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>BACKGROUND: MicroRNAs are required for maintenance of pluripotency as well as differentiation, but since more microRNAs have been computationally predicted in genome than have been found, there are likely to be undiscovered microRNAs expressed early in stem cell differentiation. METHODOLOGY/PRINCIPAL FINDINGS: SOLiD ultra-deep sequencing identified >10(7) unique small RNAs from human embryonic stem cells (hESC) and neural-restricted precursors that were fit to a model of microRNA biogenesis to computationally predict 818 new microRNA genes. These predicted genomic loci are associated with chromatin patterns of modified histones that are predictive of regulated gene expression. 146 of the predicted microRNAs were enriched in Ago2-containing complexes along with 609 known microRNAs, demonstrating association with a functional RISC complex. This Ago2 IP-selected subset was consistently expressed in four independent hESC lines and exhibited complex patterns of regulation over development similar to previously-known microRNAs, including pluripotency-specific expression in both hESC and iPS cells. More than 30% of the Ago2 IP-enriched predicted microRNAs are new members of existing families since they share seed sequences with known microRNAs. CONCLUSIONS/SIGNIFICANCE: Extending the classic definition of microRNAs, this large number of new microRNA genes, the majority of which are less conserved than their canonical counterparts, likely represent evolutionarily recent regulators of early differentiation. The enrichment in Ago2 containing complexes, the presence of chromatin marks indicative of regulated gene expression, and differential expression over development all support the identification of 146 new microRNAs active during early hESC differentiation.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0007192 ">10.1371/journal.pone.0007192 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse26">
                 <em>Rapid induction of genes associated with tissue protection and neural development in contused adult spinal cord after radial glial cell transplantation.</em> (2009)</a>
                 <p class="note">Chang YW, Goff LA, Li H, Kane-Goldsmith N, Tzatzalos E, Hart RP, Young W, Grumet M<br>
                 J Neurotrauma. 2009 Jul;26(7):979-93. doi: 10.1089/neu.2008-0762.</p>
        </span>
      </div>
      <div id="collapse26" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="19257808" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Cell-based therapy has been widely evaluated in spinal cord injury (SCI) animal models and shown to improve functional recovery. However, host response to cell transplants at gene expression level is rarely discussed. We reported previously that acute transplantation of radial glial cells RG3.6 following SCI promoted early locomotion improvement within 1 week post-injury. To identify rapid molecular changes induced by RG3.6 transplantation in the host tissue, distal spinal cord segments were subjected to microarray analysis. Although RG3.6 transplantation, reduced activity of macrophages as early as 1-2 weeks post-injury, the expression levels of inflammatory genes (e.g., IL-6, MIP-2, MCP-1) were not decreased by RG3.6 treatment as compared to medium or other cell controls at 6-12 h post-injury. However, genes associated with tissue protection (Hsp70 and Hsp32) and neural cell development (Foxg1, Top2a, Sox11, Nkx2.2, Vimentin) were found to be significantly up-regulated by RG3.6 transplants. Foxg1 was the most highly induced gene in the RG3.6-treated spinal cords, and its expression by immunocytochemistry was confirmed in the host tissue. Moreover, RG3.6 treatment boosted the number of Nkx2.2 cells in the spinal cord, and these cells frequently co-expressed NG2, which marks progenitor cells. Taken together, these results demonstrate that radial glial transplants induced rapid and specific gene expression in the injured host tissue, and suggest that these early responses are associated with mechanisms of tissue protection and activation of endogenous neural progenitor cells.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1089/neu.2008-0762 ">10.1089/neu.2008-0762 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse27">
                 <em>Functional differentiation of a clone resembling embryonic cortical interneuron progenitors.</em> (2008)</a>
                 <p class="note">Li H, Han YR, Bi C, Davila J, Goff LA, Thompson K, Swerdel M, Camarillo C, Ricupero CL, Hart RP, Plummer MR, Grumet M<br>
                 Dev Neurobiol. 2008 Dec;68(14):1549-64. doi: 10.1002/dneu.20679.</p>
        </span>
      </div>
      <div id="collapse27" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="18814314" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>We have generated clones (L2.3 and RG3.6) of neural progenitors with radial glial properties from rat E14.5 cortex that differentiate into astrocytes, neurons, and oligodendrocytes. Here, we describe a different clone (L2.2) that gives rise exclusively to neurons, but not to glia. Neuronal differentiation of L2.2 cells was inhibited by bone morphogenic protein 2 (BMP2) and enhanced by Sonic Hedgehog (SHH) similar to cortical interneuron progenitors. Compared with L2.3, differentiating L2.2 cells expressed significantly higher levels of mRNAs for glutamate decarboxylases (GADs), DLX transcription factors, calretinin, calbindin, neuropeptide Y (NPY), and somatostatin. Increased levels of DLX-2, GADs, and calretinin proteins were confirmed upon differentiation. L2.2 cells differentiated into neurons that fired action potentials in vitro, and their electrophysiological differentiation was accelerated and more complete when cocultured with developing astroglial cells but not with conditioned medium from these cells. The combined results suggest that clone L2.2 resembles GABAergic interneuron progenitors in the developing forebrain.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1002/dneu.20679 ">10.1002/dneu.20679 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse28">
                 <em>Differentiating human multipotent mesenchymal stromal cells regulate microRNAs: prediction of microRNA regulation by PDGF during osteogenesis.</em> (2008)</a>
                 <p class="note">Goff LA, Boucher S, Ricupero CL, Fenstermacher S, Swerdel M, Chase LG, Adams CC, Chesnut J, Lakshmipathy U, Hart RP<br>
                 Exp Hematol. 2008 Oct;36(10):1354-1369. doi: 10.1016/j.exphem.2008.05.004. Epub 2008 Jul 26.</p>
        </span>
      </div>
      <div id="collapse28" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="18657893" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>OBJECTIVE: Human multipotent mesenchymal stromal cells (MSC) have the potential to differentiate into multiple cell types, although little is known about factors that control their fate. Differentiation-specific microRNAs may play a key role in stem cell self-renewal and differentiation. We propose that specific intracellular signaling pathways modulate gene expression during differentiation by regulating microRNA expression. MATERIALS AND METHODS: Illumina mRNA and NCode microRNA expression analyses were performed on MSC and their differentiated progeny. A combination of bioinformatic prediction and pathway inhibition was used to identify microRNAs associated with platelet-derived growth factor (PDGF) signaling. RESULTS: The pattern of microRNA expression in MSC is distinct from that in pluripotent stem cells, such as human embryonic stem cells. Specific populations of microRNAs are regulated in MSC during differentiation targeted toward specific cell types. Complementary mRNA expression analysis increases the pool of markers characteristic of MSC or differentiated progeny. To identify microRNA expression patterns affected by signaling pathways, we examined the PDGF pathway found to be regulated during osteogenesis by microarray studies. A set of microRNAs bioinformatically predicted to respond to PDGF signaling was experimentally confirmed by direct PDGF inhibition. CONCLUSION: Our results demonstrate that a subset of microRNAs regulated during osteogenic differentiation of MSCs is responsive to perturbation of the PDGF pathway. This approach not only identifies characteristic classes of differentiation-specific mRNAs and microRNAs, but begins to link regulated molecules with specific cellular pathways.</p>
          
              <a target="_blank" href="http://dx.doi.org/10.1016/j.exphem.2008.05.004 ">10.1016/j.exphem.2008.05.004 </a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse29">
                 <em>MicroRNA expression pattern of undifferentiated and differentiated human embryonic stem cells.</em> (2007)</a>
                 <p class="note">Lakshmipathy U, Love B, Goff LA, Jornsten R, Graichen R, Hart RP, Chesnut JD<br>
                 Stem Cells Dev. 2007 Dec;16(6):1003-16.</p>
        </span>
      </div>
      <div id="collapse29" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="18004940" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Many of the currently established human embryonic stem (hES) cell lines have been characterized extensively in terms of their gene expression profiles and genetic stability in culture. Recent studies have indicated that microRNAs (miRNAs), a class of noncoding small RNAs that participate in the regulation of gene expression, may play a key role in stem cell self-renewal and differentiation. Using both microarrays and quantitative PCR, we report here the differences in miRNA expression between undifferentiated hES cells and their corresponding differentiated cells that underwent differentiation in vitro over a period of 2 weeks. Our results confirm the identity of a signature miRNA profile in pluripotent cells, comprising a small subset of differentially expressed miRNAs in hES cells. Examining both mRNA and miRNA profiles under multiple conditions using cross-correlation, we find clusters of miRNAs grouped with specific, biologically interpretable mRNAs. We identify patterns of expression in the progression from hES cells to differentiated cells that suggest a role for selected miRNAs in maintenance of the undifferentiated, pluripotent state. Profiling of the hES cell "miRNA-ome" provides an insight into molecules that control cellular differentiation and maintenance of the pluripotent state, findings that have broad implications in development, homeostasis, and human disease states.</p>
          
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/18004940">Article</a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse30">
                 <em>Bioinformatic analysis of neural stem cell differentiation.</em> (2007)</a>
                 <p class="note">Goff LA, Davila J, Jornsten R, Keles S, Hart RP<br>
                 J Biomol Tech. 2007 Sep;18(4):205-12.</p>
        </span>
      </div>
      <div id="collapse30" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="17916793" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Regulated mRnAs during differentiation of rat neural stem cells were analyzed using the ABi1700 microarray platform. This microarray, while technically advanced, suffers from the difficulty of integrating hybridization results into public databases for systems-level analysis. This is particularly true for the rat array, since many of the probes were designed for transcripts based on predicted human and mouse homologs. using several strategies, we increased the public annotation of the 27,531 probes from 43% to over 65%. To increase the dynamic range of annotation, probes were mapped to numerous public keys from several data sources. consensus annotation from multiple sources was determined for well-scoring alignments, and a confidence-based ranking system established for probes with less agreement across multiple data sources. previous attempts at genomic interpretation using the celera annotation model resulted in poor overlap with expected genomic sequences. since the public keys are more precisely mapped to the genome, we could now analyze the relationships between predicted transcription-factor binding sites and expression clusters. Results collected from a differentiation time course of two neural stem cell clones were clustered using a model-based algorithm. Transcription-factor binding sites were predicted from upstream regions of mapped transcripts using position weight matrices from either JAspAR or TRAnsFAc, and the resulting scores were used to discriminate between observed expression clusters. A classification and regression tree analysis was conducted using cluster numbers as gene identifiers and TFBs scores as predictors, pruning back to obtain a tree with the lowest gene class prediction error rate. Results identify several transcription factors, the presence or absence of which are sufficient to describe clusters of mRnAs changing over time-those that are static, as well as clusters describing cell line differences. public annotation of the AB1700 rat genome array will be valuable for integrating results into future systems-level analyses.</p>
          
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17916793">Article</a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse31">
                 <em>Rational probe optimization and enhanced detection strategy for microRNAs using microarrays.</em> (2005)</a>
                 <p class="note">Goff LA, Yang M, Bowers J, Getts RC, Padgett RW, Hart RP<br>
                 RNA Biol. 2005 Jul-Sep;2(3):93-100. Epub 2005 Jul 20.</p>
        </span>
      </div>
      <div id="collapse31" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="17114923" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>MicroRNAs (miRNAs) are post-transcriptional regulators participating in biological processes ranging from differentiation to carcinogenesis. We developed a rational probe design algorithm and a sensitive labelling scheme for optimizing miRNA microarrays. Our microarray contains probes for all validated miRNAs from five species, with the potential for drawing on species conservation to identify novel miRNAs with homologous probes. These methods are useful for high-throughput analysis of micro RNAs from various sources, and allow analysis with limiting quantities of RNA. The system design can also be extended for use on Luminex beads or on 96-well plates in an ELISA-style assay. We optimized hybridization temperatures using sequence variations on 20 of the probes and determined that all probes distinguish wild-type from 2 nt mutations, and most probes distinguish a 1 nt mutation, producing good selectivity between closely-related small RNA sequences. Results of tissue comparisons on our microarrays reveal patterns of hybridization that agree with results from Northern blots and other methods.</p>
          
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17114923">Article</a>
          
        </div>
      </div>
    </div>
    
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse32">
                 <em>Evaluation of sense-strand mRNA amplification by comparative quantitative PCR.</em> (2004)</a>
                 <p class="note">Goff LA, Bowers J, Schwalm J, Howerton K, Getts RC, Hart RP<br>
                 BMC Genomics. 2004 Oct 6;5:76.</p>
        </span>
      </div>
      <div id="collapse32" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="15469607" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>BACKGROUND: RNA amplification is required for incorporating laser-capture microdissection techniques into microarray assays. However, standard oligonucleotide microarrays contain sense-strand probes, so traditional T7 amplification schemes producing anti-sense RNA are not appropriate for hybridization when combined with conventional reverse transcription labeling methods. We wished to assess the accuracy of a new sense-strand RNA amplification method by comparing ratios between two samples using quantitative real-time PCR (qPCR), mimicking a two-color microarray assay. RESULTS: We performed our validation using qPCR. Three samples of rat brain RNA and three samples of rat liver RNA were amplified using several kits (Ambion messageAmp, NuGen Ovation, and several versions of Genisphere SenseAmp). Results were assessed by comparing the liver/brain ratio for 192 mRNAs before and after amplification. In general, all kits produced strong correlations with unamplified RNAs. The SenseAmp kit produced the highest correlation, and was also able to amplify a partially degraded sample accurately. CONCLUSION: We have validated an optimized sense-strand RNA amplification method for use in comparative studies such as two-color microarrays.</p>
          
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/15469607">Article</a>
          
        </div>
      </div>
    </div>
    
  </div>


	         </div><!-- .span10 -->

		 </div><!-- .row -->


		 <div class="container" style="padding:5px;">
			 <div style="border-top: 2px solid #ccc;">
					
          
          <p class="pull-right" style="padding:5px;"><img style="height: 20px" src="http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/by-nc-nd.svg"> Loyal Goff, 2015</p>
			 </div>
		 </div>

 </body>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-318359-8', 'auto');
  ga('send', 'pageview');

</script>
</html>
