<!DOCTYPE html>
 <html>
 <head>
		 <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Loyal Goff</title>
         <link rel="stylesheet" href="css/bootstrap.min.css">
		 <link rel="stylesheet" href="css/bootstrap-responsive.css">
		 
		 <script src="http://code.jquery.com/jquery-latest.js"></script>
		 <script src="js/bootstrap.min.js"></script>
		 
 </head>
 <body>	  
 <div class="container">
	 
	 <div class="navbar navbar-inverse">
		 <div class="navbar-inner">
			 <a class="brand" href="index.html">Loyal Goff</a>
		 <ul class="nav">
		   <li class="active"><a href="index.html"><i class="icon-home icon-white"></i> Home</a></li>
		 </ul>
		 
		 <form class="navbar-search pull-right">
		   <i class="icon-search icon-white"></i>
		   <input type="text" class="search-query" placeholder="Search">
		 </form>
	 	</div>
	 </div>

 <div class="row">
     <div class="span2">
		 <ul class="nav nav-list">
		 
		   <li><a href="index.html">Home</a></li>
		 
		   <li><a href="research.html">Research</a></li>
		 
		   <li class="active"><a href="publications.html">Publications</a></li>
		 
		   <li><a href="software.html">Software</a></li>
		 
		   <li><a href="contact.html">Contact</a></li>
		 
		   <li><a href="about.html">About</a></li>
		 
		   <li><a href="links.html">Links</a></li>
		 
		 </ul>

     </div><!-- .span2 -->

         <div class="span10">
	         
<h2>Publications</h2>
	<div class="accordion" id="accordionpubs">
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse1">
     		         Hacisuleyman E, Goff LA, Trapnell C, Williams A, Henao-Mejia J, Sun L, McClanahan P, Hendrickson DG, Sauvageau M, Kelley DR, Morse M, Engreitz J, Lander ES, Guttman M, Lodish HF, Flavell R, Raj A, Rinn JL (2014) <em>Topological organization of multichromosomal regions by the long intergenic noncoding RNA Firre.</em> Nat Struct Mol Biol. 2014 Jan 26. doi: 10.1038/nsmb.2764.
     		       </a>
     		     </div>
     		     <div id="collapse1" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>RNA, including long noncoding RNA (lncRNA), is known to be an abundant and important structural component of the nuclear matrix. However, the molecular identities, functional roles and localization dynamics of lncRNAs that influence nuclear architecture remain poorly understood. Here, we describe one lncRNA, Firre, that interacts with the nuclear-matrix factor hnRNPU through a 156-bp repeating sequence and localizes across an ~5-Mb domain on the X chromosome. We further observed Firre localization across five distinct trans-chromosomal loci, which reside in spatial proximity to the Firre genomic locus on the X chromosome. Both genetic deletion of the Firre locus and knockdown of hnRNPU resulted in loss of colocalization of these trans-chromosomal interacting loci. Thus, our data suggest a model in which lncRNAs such as Firre can interface with and modulate nuclear architecture across chromosomes.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1038/nsmb.2764">10.1038/nsmb.2764 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1038/nsmb.2764 ">10.1038/nsmb.2764 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse2">
     		         Silverman IM, Li F, Alexander A, Goff L, Trapnell C, Rinn JL, Gregory BD (2014) <em>RNase-mediated protein footprint sequencing reveals protein-binding sites throughout the human transcriptome.</em> Genome Biol. 2014 Jan 7;15(1):R3.
     		       </a>
     		     </div>
     		     <div id="collapse2" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Although numerous approaches have been developed to map RNA-binding sites of individual RNA-binding proteins (RBPs), few methods exist that allow assessment of global RBP-RNA interactions. Here, we describe PIP-seq, a universal, high-throughput, ribonuclease-mediated protein footprint sequencing approach that reveals RNA-protein interaction sites throughout a transcriptome of interest. We apply PIP-seq to the HeLa transcriptome and compare binding sites found using different cross-linkers and ribonucleases. From this analysis, we identify numerous putative RBP-binding motifs, reveal novel insights into co-binding by RBPs, and uncover a significant enrichment for disease-associated polymorphisms within RBP interaction sites.</p>
				   
			   <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/24393486">Article</a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse3">
     		         Sauvageau M, Goff LA, Lodato S, Bonev B, Groff AF, Gerhardinger C, Sanchez-Gomez DB, Hacisuleyman E, Li E, Spence M, Liapis SC, Mallard W, Morse M, Swerdel MR, D'Ecclessis MF, Moore JC, Lai V, Gong G, Yancopoulos GD, Frendewey D, Kellis M, Hart RP, Valenzuela DM, Arlotta P, Rinn JL (2013) <em>Multiple knockout mouse models reveal lincRNAs are required for life and brain development.</em> Elife. 2013 Dec 31;2(0):e01749. doi: 10.7554/eLife.01749.
     		       </a>
     		     </div>
     		     <div id="collapse3" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Many studies are uncovering functional roles for long noncoding RNAs (lncRNAs), yet few have been tested for in vivo relevance through genetic ablation in animal models. To investigate the functional relevance of lncRNAs in various physiological conditions, we have developed a collection of 18 lncRNA knockout strains in which the locus is maintained transcriptionally active. Initial characterization revealed peri- and postnatal lethal phenotypes in three mutant strains (Fendrr, Peril, and Mdgt), the latter two exhibiting incomplete penetrance and growth defects in survivors. We also report growth defects for two additional mutant strains (linc-Brn1b and linc-Pint). Further analysis revealed defects in lung, gastrointestinal tract, and heart in Fendrr(-/-) neonates, whereas linc-Brn1b(-/-) mutants displayed distinct abnormalities in the generation of upper layer II-IV neurons in the neocortex. This study demonstrates that lncRNAs play critical roles in vivo and provides a framework and impetus for future larger-scale functional investigation into the roles of lncRNA molecules. DOI: http://dx.doi.org/10.7554/eLife.01749.001.</p>
				   
			   <!--<a href="http://dx.doi.org/10.7554/eLife.01749">10.7554/eLife.01749 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.7554/eLife.01749 ">10.7554/eLife.01749 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse4">
     		         Goff LA, Rinn JL (2013) <em>Poly-combing the genome for RNA.</em> Nat Struct Mol Biol. 2013 Dec;20(12):1344-6. doi: 10.1038/nsmb.2728.
     		       </a>
     		     </div>
     		     <div id="collapse4" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>An unresolved question in mammalian epigenetic regulation is how ubiquitously expressed chromatin-modifying complexes such as Polycomb group complex 2 (PRC2) find their specific target sites across an intricate choreography of localization events in time and space. Two recent studies now provide critical new insights into an intriguing genome-wide role for RNA in PRC2 regulation.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1038/nsmb.2728">10.1038/nsmb.2728 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1038/nsmb.2728 ">10.1038/nsmb.2728 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse5">
     		         Di Ruscio A, Ebralidze AK, Benoukraf T, Amabile G, Goff LA, Terragni J, Figueroa ME, De Figueiredo Pontes LL, Alberich-Jorda M, Zhang P, Wu M, D'Alo F, Melnick A, Leone G, Ebralidze KK, Pradhan S, Rinn JL, Tenen DG (2013) <em>DNMT1-interacting RNAs block gene-specific DNA methylation.</em> Nature. 2013 Nov 21;503(7476):371-6. doi: 10.1038/nature12598. Epub 2013 Oct 9.
     		       </a>
     		     </div>
     		     <div id="collapse5" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>DNA methylation was first described almost a century ago; however, the rules governing its establishment and maintenance remain elusive. Here we present data demonstrating that active transcription regulates levels of genomic methylation. We identify a novel RNA arising from the CEBPA gene locus that is critical in regulating the local DNA methylation profile. This RNA binds to DNMT1 and prevents CEBPA gene locus methylation. Deep sequencing of transcripts associated with DNMT1 combined with genome-scale methylation and expression profiling extend the generality of this finding to numerous gene loci. Collectively, these results delineate the nature of DNMT1-RNA interactions and suggest strategies for gene-selective demethylation of therapeutic targets in human diseases.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1038/nature12598">10.1038/nature12598 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1038/nature12598 ">10.1038/nature12598 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse6">
     		         Henao-Mejia J, Williams A, Goff LA, Staron M, Licona-Limon P, Kaech SM, Nakayama M, Rinn JL, Flavell RA (2013) <em>The microRNA miR-181 is a critical cellular metabolic rheostat essential for NKT cell ontogenesis and lymphocyte development and homeostasis.</em> Immunity. 2013 May 23;38(5):984-97. doi: 10.1016/j.immuni.2013.02.021. Epub 2013 Apr 25.
     		       </a>
     		     </div>
     		     <div id="collapse6" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Regulation of metabolic pathways in the immune system provides a mechanism to actively control cellular function, growth, proliferation, and survival. Here, we report that miR-181 is a nonredundant determinant of cellular metabolism and is essential for supporting the biosynthetic demands of early NKT cell development. As a result, miR-181-deficient mice showed a complete absence of mature NKT cells in the thymus and periphery. Mechanistically, miR-181 modulated expression of the phosphatase PTEN to control PI3K signaling, which was a primary stimulus for anabolic metabolism in immune cells. Thus miR-181-deficient mice also showed severe defects in lymphoid development and T cell homeostasis associated with impaired PI3K signaling. These results uncover miR-181 as essential for NKT cell development and establish this family of miRNAs as central regulators of PI3K signaling and global metabolic fitness during development and homeostasis.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1016/j.immuni.2013.02.021 [doi] S1074-7613(13)00191-X">10.1016/j.immuni.2013.02.021 [doi] S1074-7613(13)00191-X [pii]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1016/j.immuni.2013.02.021 ">10.1016/j.immuni.2013.02.021 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse7">
     		         Sun L, Goff LA, Trapnell C, Alexander R, Lo KA, Hacisuleyman E, Sauvageau M, Tazon-Vega B, Kelley DR, Hendrickson DG, Yuan B, Kellis M, Lodish HF, Rinn JL (2013) <em>Long noncoding RNAs regulate adipogenesis.</em> Proc Natl Acad Sci U S A. 2013 Feb 26;110(9):3387-92. doi: 10.1073/pnas.1222643110. Epub 2013 Feb 11.
     		       </a>
     		     </div>
     		     <div id="collapse7" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>The prevalence of obesity has led to a surge of interest in understanding the detailed mechanisms underlying adipocyte development. Many protein-coding genes, mRNAs, and microRNAs have been implicated in adipocyte development, but the global expression patterns and functional contributions of long noncoding RNA (lncRNA) during adipogenesis have not been explored. Here we profiled the transcriptome of primary brown and white adipocytes, preadipocytes, and cultured adipocytes and identified 175 lncRNAs that are specifically regulated during adipogenesis. Many lncRNAs are adipose-enriched, strongly induced during adipogenesis, and bound at their promoters by key transcription factors such as peroxisome proliferator-activated receptor gamma (PPARgamma) and CCAAT/enhancer-binding protein alpha (CEBPalpha). RNAi-mediated loss of function screens identified functional lncRNAs with varying impact on adipogenesis. Collectively, we have identified numerous lncRNAs that are functionally required for proper adipogenesis.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1073/pnas.1222643110">10.1073/pnas.1222643110 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1222643110 ">10.1073/pnas.1222643110 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse8">
     		         Trapnell C, Hendrickson DG, Sauvageau M, Goff L, Rinn JL, Pachter L (2013) <em>Differential analysis of gene regulation at transcript resolution with RNA-seq.</em> Nat Biotechnol. 2013 Jan;31(1):46-53. doi: 10.1038/nbt.2450. Epub 2012 Dec 9.
     		       </a>
     		     </div>
     		     <div id="collapse8" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Differential analysis of gene and transcript expression using high-throughput RNA sequencing (RNA-seq) is complicated by several sources of measurement variability and poses numerous statistical challenges. We present Cuffdiff 2, an algorithm that estimates expression at transcript-level resolution and controls for variability evident across replicate libraries. Cuffdiff 2 robustly identifies differentially expressed transcripts and genes and reveals differential splicing and promoter-preference changes. We demonstrate the accuracy of our approach through differential analysis of lung fibroblasts in response to loss of the developmental transcription factor HOXA1, which we show is required for lung fibroblast and HeLa cell cycle progression. Loss of HOXA1 results in significant expression level changes in thousands of individual transcripts, along with isoform switching events in key regulators of the cell cycle. Cuffdiff 2 performs robust differential analysis in RNA-seq experiments at transcript resolution, revealing a layer of regulation not readily observable with other high-throughput technologies.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1038/nbt.2450">10.1038/nbt.2450 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1038/nbt.2450 ">10.1038/nbt.2450 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse9">
     		         Washietl S, Will S, Hendrix DA, Goff LA, Rinn JL, Berger B, Kellis M (2012) <em>Computational analysis of noncoding RNAs.</em> Wiley Interdiscip Rev RNA. 2012 Nov-Dec;3(6):759-78. doi: 10.1002/wrna.1134. Epub 2012 Sep 18.
     		       </a>
     		     </div>
     		     <div id="collapse9" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Noncoding RNAs have emerged as important key players in the cell. Understanding their surprisingly diverse range of functions is challenging for experimental and computational biology. Here, we review computational methods to analyze noncoding RNAs. The topics covered include basic and advanced techniques to predict RNA structures, annotation of noncoding RNAs in genomic data, mining RNA-seq data for novel transcripts and prediction of transcript structures, computational aspects of microRNAs, and database resources.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1002/wrna.1134">10.1002/wrna.1134 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1002/wrna.1134 ">10.1002/wrna.1134 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse10">
     		         Trapnell C, Roberts A, Goff L, Pertea G, Kim D, Kelley DR, Pimentel H, Salzberg SL, Rinn JL, Pachter L (2012) <em>Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</em> Nat Protoc. 2012 Mar 1;7(3):562-78. doi: 10.1038/nprot.2012.016.
     		       </a>
     		     </div>
     		     <div id="collapse10" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Recent advances in high-throughput cDNA sequencing (RNA-seq) can reveal new genes and splice variants and quantify expression genome-wide in a single assay. The volume and complexity of data from RNA-seq experiments necessitate scalable, fast and mathematically principled analysis software. TopHat and Cufflinks are free, open-source software tools for gene discovery and comprehensive expression analysis of high-throughput mRNA sequencing (RNA-seq) data. Together, they allow biologists to identify new genes and new splice variants of known ones, as well as compare gene and transcript expression under two or more conditions. This protocol describes in detail how to use TopHat and Cufflinks to perform such analyses. It also covers several accessory tools and utilities that aid in managing data, including CummeRbund, a tool for visualizing RNA-seq analysis results. Although the procedure assumes basic informatics skills, these tools assume little to no background with RNA-seq analysis and are meant for novices and experts alike. The protocol begins with raw sequencing reads and produces a transcriptome assembly, lists of differentially expressed and regulated genes and transcripts, and publication-quality visualizations of analysis results. The protocol's execution time depends on the volume of transcriptome sequencing data and available computing resources but takes less than 1 d of computer time for typical experiments and approximately 1 h of hands-on time.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1038/nprot.2012.016">10.1038/nprot.2012.016 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1038/nprot.2012.016 ">10.1038/nprot.2012.016 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse11">
     		         Pichardo-Casas I, Goff LA, Swerdel MR, Athie A, Davila J, Ramos-Brossier M, Lapid-Volosin M, Friedman WJ, Hart RP, Vaca L (2012) <em>Expression profiling of synaptic microRNAs from the adult rat brain identifies regional differences and seizure-induced dynamic modulation.</em> Brain Res. 2012 Feb 3;1436:20-33. doi: 10.1016/j.brainres.2011.12.001. Epub 2011 Dec 9.
     		       </a>
     		     </div>
     		     <div id="collapse11" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>In recent years, microRNAs or miRNAs have been proposed to target neuronal mRNAs localized near the synapse, exerting a pivotal role in modulating local protein synthesis, and presumably affecting adaptive mechanisms such as synaptic plasticity. In the present study we have characterized the distribution of miRNAs in five regions of the adult mammalian brain and compared the relative abundance between total fractions and purified synaptoneurosomes (SN), using three different methodologies. The results show selective enrichment or depletion of some miRNAs when comparing total versus SN fractions. These miRNAs were different for each brain region explored. Changes in distribution could not be attributed to simple diffusion or to a targeting sequence inside the miRNAs. In silico analysis suggest that the differences in distribution may be related to the preferential concentration of synaptically localized mRNA targeted by the miRNAs. These results favor a model of co-transport of the miRNA-mRNA complex to the synapse, although further studies are required to validate this hypothesis. Using an in vivo model for increasing excitatory activity in the cortex and the hippocampus indicates that the distribution of some miRNAs can be modulated by enhanced neuronal (epileptogenic) activity. All these results demonstrate the dynamic modulation in the local distribution of miRNAs from the adult brain, which may play key roles in controlling localized protein synthesis at the synapse.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1016/j.brainres.2011.12.001">10.1016/j.brainres.2011.12.001 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1016/j.brainres.2011.12.001 ">10.1016/j.brainres.2011.12.001 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse12">
     		         Cabili MN, Trapnell C, Goff L, Koziol M, Tazon-Vega B, Regev A, Rinn JL (2011) <em>Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses.</em> Genes Dev. 2011 Sep 15;25(18):1915-27. doi: 10.1101/gad.17446611. Epub 2011 Sep 2.
     		       </a>
     		     </div>
     		     <div id="collapse12" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Large intergenic noncoding RNAs (lincRNAs) are emerging as key regulators of diverse cellular processes. Determining the function of individual lincRNAs remains a challenge. Recent advances in RNA sequencing (RNA-seq) and computational methods allow for an unprecedented analysis of such transcripts. Here, we present an integrative approach to define a reference catalog of >8000 human lincRNAs. Our catalog unifies previously existing annotation sources with transcripts we assembled from RNA-seq data collected from approximately 4 billion RNA-seq reads across 24 tissues and cell types. We characterize each lincRNA by a panorama of >30 properties, including sequence, structural, transcriptional, and orthology features. We found that lincRNA expression is strikingly tissue-specific compared with coding genes, and that lincRNAs are typically coexpressed with their neighboring genes, albeit to an extent similar to that of pairs of neighboring protein-coding genes. We distinguish an additional subset of transcripts that have high evolutionary conservation but may include short ORFs and may serve as either lincRNAs or small peptides. Our integrated, comprehensive, yet conservative reference catalog of human lincRNAs reveals the global properties of lincRNAs and will facilitate experimental studies and further functional classification of these genes.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1101/gad.17446611">10.1101/gad.17446611 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1101/gad.17446611 ">10.1101/gad.17446611 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse13">
     		         Bail S, Swerdel M, Liu H, Jiao X, Goff LA, Hart RP, Kiledjian M (2010) <em>Differential regulation of microRNA stability.</em> RNA. 2010 May;16(5):1032-9. doi: 10.1261/rna.1851510. Epub 2010 Mar 26.
     		       </a>
     		     </div>
     		     <div id="collapse13" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>MicroRNAs (miRNAs) are endogenous single-stranded RNA molecules of about 21 nucleotides in length that are fundamental post-transcriptional regulators of gene expression. Although the transcriptional and processing events involved in the generation of miRNAs have been extensively studied, very little is known pertaining to components that regulate the stability of individual miRNAs. All RNAs have distinct inherent half-lives that dictate their level of accumulation and miRNAs would be expected to follow a similar principle. Here we demonstrate that although most miRNA appear to be stable, like mRNAs, miRNAs possess differential stability in human cells. In particular, we found that miR-382, a miRNA that contributes to HIV-1 provirus latency, is unstable in cells. To determine the region of miR-382 responsible for its rapid decay, we developed a cell-free system that recapitulated the observed cell-based-regulated miR-382 turnover. The system utilizes in vitro-processed mature miRNA derived from pre-miRNA and follows the decay of the processed miRNA. Using this system, we demonstrate that instability of miR-382 is driven by sequences outside its seed region and required the 3' terminal seven nucleotides where mutations in this region increased the stability of the RNA. Moreover, the exosome 3'-5' exoribonuclease complex was identified as the primary nuclease involved in miR-382 decay with a more modest contribution by the Xrn1 and no detectable contribution by Xrn2. These studies provide evidence for an miRNA element essential for rapid miRNA decay and implicate the exosome in this process. The development of a biochemically amendable system to analyze the mechanism of differential miRNA stability provides an important step in efforts to regulate gene expression by modulating miRNA stability.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1261/rna.1851510">10.1261/rna.1851510 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1261/rna.1851510 ">10.1261/rna.1851510 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse14">
     		         Goff LA, Davila J, Swerdel MR, Moore JC, Cohen RI, Wu H, Sun YE, Hart RP (2009) <em>Ago2 immunoprecipitation identifies predicted microRNAs in human embryonic stem cells and neural precursors.</em> PLoS One. 2009 Sep 28;4(9):e7192. doi: 10.1371/journal.pone.0007192.
     		       </a>
     		     </div>
     		     <div id="collapse14" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>BACKGROUND: MicroRNAs are required for maintenance of pluripotency as well as differentiation, but since more microRNAs have been computationally predicted in genome than have been found, there are likely to be undiscovered microRNAs expressed early in stem cell differentiation. METHODOLOGY/PRINCIPAL FINDINGS: SOLiD ultra-deep sequencing identified >10(7) unique small RNAs from human embryonic stem cells (hESC) and neural-restricted precursors that were fit to a model of microRNA biogenesis to computationally predict 818 new microRNA genes. These predicted genomic loci are associated with chromatin patterns of modified histones that are predictive of regulated gene expression. 146 of the predicted microRNAs were enriched in Ago2-containing complexes along with 609 known microRNAs, demonstrating association with a functional RISC complex. This Ago2 IP-selected subset was consistently expressed in four independent hESC lines and exhibited complex patterns of regulation over development similar to previously-known microRNAs, including pluripotency-specific expression in both hESC and iPS cells. More than 30% of the Ago2 IP-enriched predicted microRNAs are new members of existing families since they share seed sequences with known microRNAs. CONCLUSIONS/SIGNIFICANCE: Extending the classic definition of microRNAs, this large number of new microRNA genes, the majority of which are less conserved than their canonical counterparts, likely represent evolutionarily recent regulators of early differentiation. The enrichment in Ago2 containing complexes, the presence of chromatin marks indicative of regulated gene expression, and differential expression over development all support the identification of 146 new microRNAs active during early hESC differentiation.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1371/journal.pone.0007192">10.1371/journal.pone.0007192 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0007192 ">10.1371/journal.pone.0007192 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse15">
     		         Chang YW, Goff LA, Li H, Kane-Goldsmith N, Tzatzalos E, Hart RP, Young W, Grumet M (2009) <em>Rapid induction of genes associated with tissue protection and neural development in contused adult spinal cord after radial glial cell transplantation.</em> J Neurotrauma. 2009 Jul;26(7):979-93. doi: 10.1089/neu.2008-0762.
     		       </a>
     		     </div>
     		     <div id="collapse15" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Cell-based therapy has been widely evaluated in spinal cord injury (SCI) animal models and shown to improve functional recovery. However, host response to cell transplants at gene expression level is rarely discussed. We reported previously that acute transplantation of radial glial cells RG3.6 following SCI promoted early locomotion improvement within 1 week post-injury. To identify rapid molecular changes induced by RG3.6 transplantation in the host tissue, distal spinal cord segments were subjected to microarray analysis. Although RG3.6 transplantation, reduced activity of macrophages as early as 1-2 weeks post-injury, the expression levels of inflammatory genes (e.g., IL-6, MIP-2, MCP-1) were not decreased by RG3.6 treatment as compared to medium or other cell controls at 6-12 h post-injury. However, genes associated with tissue protection (Hsp70 and Hsp32) and neural cell development (Foxg1, Top2a, Sox11, Nkx2.2, Vimentin) were found to be significantly up-regulated by RG3.6 transplants. Foxg1 was the most highly induced gene in the RG3.6-treated spinal cords, and its expression by immunocytochemistry was confirmed in the host tissue. Moreover, RG3.6 treatment boosted the number of Nkx2.2 cells in the spinal cord, and these cells frequently co-expressed NG2, which marks progenitor cells. Taken together, these results demonstrate that radial glial transplants induced rapid and specific gene expression in the injured host tissue, and suggest that these early responses are associated with mechanisms of tissue protection and activation of endogenous neural progenitor cells.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1089/neu.2008-0762">10.1089/neu.2008-0762 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1089/neu.2008-0762 ">10.1089/neu.2008-0762 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse16">
     		         Li H, Han YR, Bi C, Davila J, Goff LA, Thompson K, Swerdel M, Camarillo C, Ricupero CL, Hart RP, Plummer MR, Grumet M (2008) <em>Functional differentiation of a clone resembling embryonic cortical interneuron progenitors.</em> Dev Neurobiol. 2008 Dec;68(14):1549-64. doi: 10.1002/dneu.20679.
     		       </a>
     		     </div>
     		     <div id="collapse16" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>We have generated clones (L2.3 and RG3.6) of neural progenitors with radial glial properties from rat E14.5 cortex that differentiate into astrocytes, neurons, and oligodendrocytes. Here, we describe a different clone (L2.2) that gives rise exclusively to neurons, but not to glia. Neuronal differentiation of L2.2 cells was inhibited by bone morphogenic protein 2 (BMP2) and enhanced by Sonic Hedgehog (SHH) similar to cortical interneuron progenitors. Compared with L2.3, differentiating L2.2 cells expressed significantly higher levels of mRNAs for glutamate decarboxylases (GADs), DLX transcription factors, calretinin, calbindin, neuropeptide Y (NPY), and somatostatin. Increased levels of DLX-2, GADs, and calretinin proteins were confirmed upon differentiation. L2.2 cells differentiated into neurons that fired action potentials in vitro, and their electrophysiological differentiation was accelerated and more complete when cocultured with developing astroglial cells but not with conditioned medium from these cells. The combined results suggest that clone L2.2 resembles GABAergic interneuron progenitors in the developing forebrain.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1002/dneu.20679">10.1002/dneu.20679 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1002/dneu.20679 ">10.1002/dneu.20679 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse17">
     		         Goff LA, Boucher S, Ricupero CL, Fenstermacher S, Swerdel M, Chase LG, Adams CC, Chesnut J, Lakshmipathy U, Hart RP (2008) <em>Differentiating human multipotent mesenchymal stromal cells regulate microRNAs: prediction of microRNA regulation by PDGF during osteogenesis.</em> Exp Hematol. 2008 Oct;36(10):1354-1369. doi: 10.1016/j.exphem.2008.05.004. Epub 2008 Jul 26.
     		       </a>
     		     </div>
     		     <div id="collapse17" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>OBJECTIVE: Human multipotent mesenchymal stromal cells (MSC) have the potential to differentiate into multiple cell types, although little is known about factors that control their fate. Differentiation-specific microRNAs may play a key role in stem cell self-renewal and differentiation. We propose that specific intracellular signaling pathways modulate gene expression during differentiation by regulating microRNA expression. MATERIALS AND METHODS: Illumina mRNA and NCode microRNA expression analyses were performed on MSC and their differentiated progeny. A combination of bioinformatic prediction and pathway inhibition was used to identify microRNAs associated with platelet-derived growth factor (PDGF) signaling. RESULTS: The pattern of microRNA expression in MSC is distinct from that in pluripotent stem cells, such as human embryonic stem cells. Specific populations of microRNAs are regulated in MSC during differentiation targeted toward specific cell types. Complementary mRNA expression analysis increases the pool of markers characteristic of MSC or differentiated progeny. To identify microRNA expression patterns affected by signaling pathways, we examined the PDGF pathway found to be regulated during osteogenesis by microarray studies. A set of microRNAs bioinformatically predicted to respond to PDGF signaling was experimentally confirmed by direct PDGF inhibition. CONCLUSION: Our results demonstrate that a subset of microRNAs regulated during osteogenic differentiation of MSCs is responsive to perturbation of the PDGF pathway. This approach not only identifies characteristic classes of differentiation-specific mRNAs and microRNAs, but begins to link regulated molecules with specific cellular pathways.</p>
				   
			   <!--<a href="http://dx.doi.org/10.1016/j.exphem.2008.05.004">10.1016/j.exphem.2008.05.004 [doi]</a>  #Deprecated due to invalid parsing of LID records: LAG 9-9-13 -->
         <a target="_blank" href="http://dx.doi.org/10.1016/j.exphem.2008.05.004 ">10.1016/j.exphem.2008.05.004 </a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse18">
     		         Lakshmipathy U, Love B, Goff LA, Jornsten R, Graichen R, Hart RP, Chesnut JD (2007) <em>MicroRNA expression pattern of undifferentiated and differentiated human embryonic stem cells.</em> Stem Cells Dev. 2007 Dec;16(6):1003-16.
     		       </a>
     		     </div>
     		     <div id="collapse18" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Many of the currently established human embryonic stem (hES) cell lines have been characterized extensively in terms of their gene expression profiles and genetic stability in culture. Recent studies have indicated that microRNAs (miRNAs), a class of noncoding small RNAs that participate in the regulation of gene expression, may play a key role in stem cell self-renewal and differentiation. Using both microarrays and quantitative PCR, we report here the differences in miRNA expression between undifferentiated hES cells and their corresponding differentiated cells that underwent differentiation in vitro over a period of 2 weeks. Our results confirm the identity of a signature miRNA profile in pluripotent cells, comprising a small subset of differentially expressed miRNAs in hES cells. Examining both mRNA and miRNA profiles under multiple conditions using cross-correlation, we find clusters of miRNAs grouped with specific, biologically interpretable mRNAs. We identify patterns of expression in the progression from hES cells to differentiated cells that suggest a role for selected miRNAs in maintenance of the undifferentiated, pluripotent state. Profiling of the hES cell "miRNA-ome" provides an insight into molecules that control cellular differentiation and maintenance of the pluripotent state, findings that have broad implications in development, homeostasis, and human disease states.</p>
				   
			   <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/18004940">Article</a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse19">
     		         Goff LA, Davila J, Jornsten R, Keles S, Hart RP (2007) <em>Bioinformatic analysis of neural stem cell differentiation.</em> J Biomol Tech. 2007 Sep;18(4):205-12.
     		       </a>
     		     </div>
     		     <div id="collapse19" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>Regulated mRnAs during differentiation of rat neural stem cells were analyzed using the ABi1700 microarray platform. This microarray, while technically advanced, suffers from the difficulty of integrating hybridization results into public databases for systems-level analysis. This is particularly true for the rat array, since many of the probes were designed for transcripts based on predicted human and mouse homologs. using several strategies, we increased the public annotation of the 27,531 probes from 43% to over 65%. To increase the dynamic range of annotation, probes were mapped to numerous public keys from several data sources. consensus annotation from multiple sources was determined for well-scoring alignments, and a confidence-based ranking system established for probes with less agreement across multiple data sources. previous attempts at genomic interpretation using the celera annotation model resulted in poor overlap with expected genomic sequences. since the public keys are more precisely mapped to the genome, we could now analyze the relationships between predicted transcription-factor binding sites and expression clusters. Results collected from a differentiation time course of two neural stem cell clones were clustered using a model-based algorithm. Transcription-factor binding sites were predicted from upstream regions of mapped transcripts using position weight matrices from either JAspAR or TRAnsFAc, and the resulting scores were used to discriminate between observed expression clusters. A classification and regression tree analysis was conducted using cluster numbers as gene identifiers and TFBs scores as predictors, pruning back to obtain a tree with the lowest gene class prediction error rate. Results identify several transcription factors, the presence or absence of which are sufficient to describe clusters of mRnAs changing over time-those that are static, as well as clusters describing cell line differences. public annotation of the AB1700 rat genome array will be valuable for integrating results into future systems-level analyses.</p>
				   
			   <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17916793">Article</a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse20">
     		         Goff LA, Yang M, Bowers J, Getts RC, Padgett RW, Hart RP (2005) <em>Rational probe optimization and enhanced detection strategy for microRNAs using microarrays.</em> RNA Biol. 2005 Jul-Sep;2(3):93-100. Epub 2005 Jul 20.
     		       </a>
     		     </div>
     		     <div id="collapse20" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>MicroRNAs (miRNAs) are post-transcriptional regulators participating in biological processes ranging from differentiation to carcinogenesis. We developed a rational probe design algorithm and a sensitive labelling scheme for optimizing miRNA microarrays. Our microarray contains probes for all validated miRNAs from five species, with the potential for drawing on species conservation to identify novel miRNAs with homologous probes. These methods are useful for high-throughput analysis of micro RNAs from various sources, and allow analysis with limiting quantities of RNA. The system design can also be extended for use on Luminex beads or on 96-well plates in an ELISA-style assay. We optimized hybridization temperatures using sequence variations on 20 of the probes and determined that all probes distinguish wild-type from 2 nt mutations, and most probes distinguish a 1 nt mutation, producing good selectivity between closely-related small RNA sequences. Results of tissue comparisons on our microarrays reveal patterns of hybridization that agree with results from Northern blots and other methods.</p>
				   
			   <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17114923">Article</a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
  		   <div class="accordion-group">
     		     <div class="accordion-heading">
     		       <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordionpubs" href="#collapse21">
     		         Goff LA, Bowers J, Schwalm J, Howerton K, Getts RC, Hart RP (2004) <em>Evaluation of sense-strand mRNA amplification by comparative quantitative PCR.</em> BMC Genomics. 2004 Oct 6;5:76.
     		       </a>
     		     </div>
     		     <div id="collapse21" class="accordion-body collapse">
     		       <div class="accordion-inner" style="background-color:#fffff5;">
  				   <h5>Abstract</h5>
  				   <p>BACKGROUND: RNA amplification is required for incorporating laser-capture microdissection techniques into microarray assays. However, standard oligonucleotide microarrays contain sense-strand probes, so traditional T7 amplification schemes producing anti-sense RNA are not appropriate for hybridization when combined with conventional reverse transcription labeling methods. We wished to assess the accuracy of a new sense-strand RNA amplification method by comparing ratios between two samples using quantitative real-time PCR (qPCR), mimicking a two-color microarray assay. RESULTS: We performed our validation using qPCR. Three samples of rat brain RNA and three samples of rat liver RNA were amplified using several kits (Ambion messageAmp, NuGen Ovation, and several versions of Genisphere SenseAmp). Results were assessed by comparing the liver/brain ratio for 192 mRNAs before and after amplification. In general, all kits produced strong correlations with unamplified RNAs. The SenseAmp kit produced the highest correlation, and was also able to amplify a partially degraded sample accurately. CONCLUSION: We have validated an optimized sense-strand RNA amplification method for use in comparative studies such as two-color microarrays.</p>
				   
			   <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/15469607">Article</a>
			   
     		       </div>
     		     </div>
     		</div>
	
		
	</div>

         </div><!-- .span4 -->
   
 </div><!-- .row -->

 </div><!-- .container -->
 <div class="container">
	 <div class="navbar navbar-bottom">
			<div class="well well-small pull-right"><a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US"></a>This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.</div>
	 </div>
 </div>
     
 </body>
</html>
