<!DOCTYPE HTML>
<!--
	Landed by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>
	    
	Publications

		</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/jquery-ui.min.css" />
		<link rel="stylesheet" href="assets/css/jquery-ui.structure.min.css" />
		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/loyal_mods.css" />
	</head>


	
<body class="is-preload">
<div id="page-wrapper">
	<!-- Header -->
  <header id="header">
    <h1 id="logo">
      <a href="index.html"> <img id="shield" src="images/medicine.shield.small.white.png" height="40" alt=""/>Stein-O'Brien Lab</a>
    </h1>
    <nav id="nav">
      <ul>
        <li><a href="index.html">Home</a></li>
        <!-- <li><a href="news.html">News</a></li> -->
        <li><a href="people.html">People</a></li>
        <li><a href="#">Research</a>
            <ul>
              <li><a href="preprints.html">Preprints</a></li>
              <li><a href="publications.html">Publications</a></li>
              <li><a href="software.html">Software</a></li>
              <li><a href="datasets.html">Datasets</a></li>
            </ul>
        </li>
        <!-- <li><a href="teaching.html">Teaching</a></li> -->
        <!--
        <li><a href="#">Resources</a>
            <ul>
              <li><a href="members.html">Lab Members</a></li>
              <li><a href="links.html">Links</a></li>
            </ul>
        </li>
      -->
        <!-- <li><a href="blog.html">Blog</a></li> -->
        <li><a href="lab_resources.html">Resources</a></li>
        <li><a href="join.html">Opportunites</a></li>
        <li><a href="contact.html">Contact Us</a></li>
        <!--<li><a href="#" class="button primary">Sign Up</a></li>-->
      </ul>
    </nav>
  </header>
	<!-- Main -->
		<div id="main" class="wrapper style1">
			<div class="container">
				<header class="major">
					<h2>Publications</h2>
						<!-- Publication summary links -->
						<p>
							<a href="http://europepmc.org/authors/0000-0001-8681-9110" target="_blank">EBI Summary page</a> |
							<a href="https://scholar.google.com/citations?user=VHrJXsUAAAAJ&hl=en" target="_blank">Google Scholar</a> |
							<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Stein-O'Brien%2C+G+%5BAuthor%5D" target="_blank">Pubmed</a> |
							<a href="https://orcid.org/0000-0001-8681-9110" target="_blank">ORCHID: 0000-0001-8681-9110</a> |
							<a href="https://profiles.impactstory.org/u/0000-0001-8681-9110" target="_blank">Impactstory</a>
						</p>
				</header>
				<!-- Content -->
					<section id="content">

							<div class="accordion alt" id="accordionpubs">
						    
								<!--  Card 1  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading1">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse1" aria-expanded="true" aria-controls="collapse1">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">1. Two-stage linked component analysis for joint decomposition of multiple biologically related data sets. (2022)</p>
						                 <p class="font-italic font-weight-light">Chen H, Caffo B, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Liu J, Langmead B, Colantuoni C, Xiao L</p>
						                 <p>Biostatistics. 2022 Mar 31. pii: 6561796. doi: 10.1093/biostatistics/kxac005. PMID: 35358296</p>
											</span>
										</a>
						      </div>

						      <div id="collapse1" class="collapse" aria-labelledby="heading1" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="35358296" class="altmetric-embed pull-left"> </div>
						          <p>Integrative analysis of multiple data sets has the potential of fully leveraging the vast amount of high throughput biological data being generated. In particular such analysis will be powerful in making inference from publicly available collections of genetic, transcriptomic and epigenetic data sets which are designed to study shared biological processes, but which vary in their target measurements, biological variation, unwanted noise, and batch variation. Thus, methods that enable the joint analysis of multiple data sets are needed to gain insights into shared biological processes that would otherwise be hidden by unwanted intra-data set variation. Here, we propose a method called two-stage linked component analysis (2s-LCA) to jointly decompose multiple biologically related experimental data sets with biological and technological relationships that can be structured into the decomposition. The consistency of the proposed method is established and its empirical performance is evaluated via simulation studies. We apply 2s-LCA to jointly analyze four data sets focused on human brain development and identify meaningful patterns of gene expression in human neurogenesis that have shared structure across these data sets.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1093/biostatistics/kxac005 ">10.1093/biostatistics/kxac005 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 2  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading2">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse2" aria-expanded="true" aria-controls="collapse2">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">2. Universal prediction of cell-cycle position using transfer learning. (2022)</p>
						                 <p class="font-italic font-weight-light">Zheng SC, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Augustin JJ, Slosberg J, Carosso GA, Winer B, Shin G, Bjornsson HT, Goff LA, Hansen KD</p>
						                 <p>Genome Biol. 2022 Jan 31;23(1):41. doi: 10.1186/s13059-021-02581-y. PMID: 35101061</p>
											</span>
										</a>
						      </div>

						      <div id="collapse2" class="collapse" aria-labelledby="heading2" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="35101061" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: The cell cycle is a highly conserved, continuous process which controls faithful replication and division of cells. Single-cell technologies have enabled increasingly precise measurements of the cell cycle both as a biological process of interest and as a possible confounding factor. Despite its importance and conservation, there is no universally applicable approach to infer position in the cell cycle with high-resolution from single-cell RNA-seq data. RESULTS: Here, we present tricycle, an R/Bioconductor package, to address this challenge by leveraging key features of the biology of the cell cycle, the mathematical properties of principal component analysis of periodic functions, and the use of transfer learning. We estimate a cell-cycle embedding using a fixed reference dataset and project new data into this reference embedding, an approach that overcomes key limitations of learning a dataset-dependent embedding. Tricycle then predicts a cell-specific position in the cell cycle based on the data projection. The accuracy of tricycle compares favorably to gold-standard experimental assays, which generally require specialized measurements in specifically constructed in vitro systems. Using internal controls which are available for any dataset, we show that tricycle predictions generalize to datasets with multiple cell types, across tissues, species, and even sequencing assays. CONCLUSIONS: Tricycle generalizes across datasets and is highly scalable and applicable to atlas-level single-cell RNA-seq data.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/s13059-021-02581-y ">10.1186/s13059-021-02581-y </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 3  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading3">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse3" aria-expanded="true" aria-controls="collapse3">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">3. Neoadjuvant Cabozantinib and Nivolumab Converts Locally Advanced HCC into Resectable Disease with Enhanced Antitumor Immunity. (2021)</p>
						                 <p class="font-italic font-weight-light">Ho WJ, Zhu Q, Durham J, Popovic A, Xavier S, Leatherman J, Mohan A, Mo G, Zhang S, Gross N, Charmsaz S, Lin D, Quong D, Wilt B, Kamel IR, Weiss M, Philosophe B, Burkhart R, Burns WR, Shubert C, Ejaz A, He J, Deshpande A, Danilova L, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Sugar EA, Laheru DA, Anders RA, Fertig EJ, Jaffee EM, Yarchoan M</p>
						                 <p>Nat Cancer. 2021 Sep;2(9):891-903. doi: 10.1038/s43018-021-00234-4. Epub 2021 Jul 29. PMID: 34796337</p>
											</span>
										</a>
						      </div>

						      <div id="collapse3" class="collapse" aria-labelledby="heading3" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="34796337" class="altmetric-embed pull-left"> </div>
						          <p>A potentially curative hepatic resection is the optimal treatment for hepatocellular carcinoma (HCC), but most patients are not candidates for resection and most resected HCCs eventually recur. Until recently, neoadjuvant systemic therapy for HCC has been limited by a lack of effective systemic agents. Here, in a single arm phase 1b study, we evaluated the feasibility of neoadjuvant cabozantinib and nivolumab in patients with HCC including patients outside of traditional resection criteria (NCT03299946). Of 15 patients enrolled, 12 (80%) underwent successful margin negative resection, and 5/12 (42%) patients had major pathologic responses. In-depth biospecimen profiling demonstrated an enrichment in T effector cells, as well as tertiary lymphoid structures, CD138+ plasma cells, and a distinct spatial arrangement of B cells in responders as compared to non-responders, indicating an orchestrated B-cell contribution to antitumor immunity in HCC.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/s43018-021-00234-4 ">10.1038/s43018-021-00234-4 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 4  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading4">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse4" aria-expanded="true" aria-controls="collapse4">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">4. Forecasting cellular states: from descriptive to predictive biology via single-cell multiomics. (2021)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Ainsile MC, Fertig EJ</p>
						                 <p>Curr Opin Syst Biol. 2021 Jun;26:24-32. doi: 10.1016/j.coisb.2021.03.008. Epub 2021 Apr 3. PMID: 34660940</p>
											</span>
										</a>
						      </div>

						      <div id="collapse4" class="collapse" aria-labelledby="heading4" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="34660940" class="altmetric-embed pull-left"> </div>
						          <p>As the single cell field races to characterize each cell type, state, and behavior, the complexity of the computational analysis approaches the complexity of the biological systems. Single cell and imaging technologies now enable unprecedented measurements of state transitions in biological systems, providing high-throughput data that capture tens-of-thousands of measurements on hundreds-of-thousands of samples. Thus, the definition of cell type and state is evolving to encompass the broad range of biological questions now attainable. To answer these questions requires the development of computational tools for integrated multi-omics analysis. Merged with mathematical models, these algorithms will be able to forecast future states of biological systems, going from statistical inferences of phenotypes to time course predictions of the biological systems with dynamic maps analogous to weather systems. Thus, systems biology for forecasting biological system dynamics from multi-omic data represents the future of cell biology empowering a new generation of technology-driven predictive medicine.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.coisb.2021.03.008 ">10.1016/j.coisb.2021.03.008 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 5  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading5">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse5" aria-expanded="true" aria-controls="collapse5">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">5. Author Correction: Community-wide hackathons to identify central themes in single-cell multi-omics. (2021)</p>
						                 <p class="font-italic font-weight-light">Cao KL, Abadi AJ, Davis-Marcisak EF, Hsu L, Arora A, Coullomb A, Deshpande A, Feng Y, Jeganathan P, Loth M, Meng C, Mu W, Pancaldi V, Sankaran K, Righelli D, Singh A, Sodicoff JS, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Subramanian A, Welch JD, You Y, Argelaguet R, Carey VJ, Dries R, Greene CS, Holmes S, Love MI, Ritchie ME, Yuan GC, Culhane AC, Fertig E</p>
						                 <p>Genome Biol. 2021 Aug 25;22(1):246. doi: 10.1186/s13059-021-02468-y. PMID: 34433496</p>
											</span>
										</a>
						      </div>

						      <div id="collapse5" class="collapse" aria-labelledby="heading5" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="34433496" class="altmetric-embed pull-left"> </div>
						          <p></p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/s13059-021-02468-y ">10.1186/s13059-021-02468-y </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 6  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading6">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse6" aria-expanded="true" aria-controls="collapse6">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">6. Community-wide hackathons to identify central themes in single-cell multi-omics. (2021)</p>
						                 <p class="font-italic font-weight-light">Le Cao KA, Abadi AJ, Davis-Marcisak EF, Hsu L, Arora A, Coullomb A, Deshpande A, Feng Y, Jeganathan P, Loth M, Meng C, Mu W, Pancaldi V, Sankaran K, Righelli D, Singh A, Sodicoff JS, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Subramanian A, Welch JD, You Y, Argelaguet R, Carey VJ, Dries R, Greene CS, Holmes S, Love MI, Ritchie ME, Yuan GC, Culhane AC, Fertig E</p>
						                 <p>Genome Biol. 2021 Aug 5;22(1):220. doi: 10.1186/s13059-021-02433-9. PMID: 34353350</p>
											</span>
										</a>
						      </div>

						      <div id="collapse6" class="collapse" aria-labelledby="heading6" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="34353350" class="altmetric-embed pull-left"> </div>
						          <p></p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/s13059-021-02433-9 ">10.1186/s13059-021-02433-9 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 7  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading7">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse7" aria-expanded="true" aria-controls="collapse7">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">7. From bench to bedside: Single-cell analysis for cancer immunotherapy. (2021)</p>
						                 <p class="font-italic font-weight-light">Davis-Marcisak EF, Deshpande A, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Ho WJ, Laheru D, Jaffee EM, Fertig EJ, Kagohara LT</p>
						                 <p>Cancer Cell. 2021 Aug 9;39(8):1062-1080. doi: 10.1016/j.ccell.2021.07.004. Epub 2021 Jul 29. PMID: 34329587</p>
											</span>
										</a>
						      </div>

						      <div id="collapse7" class="collapse" aria-labelledby="heading7" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="34329587" class="altmetric-embed pull-left"> </div>
						          <p>Single-cell technologies are emerging as powerful tools for cancer research. These technologies characterize the molecular state of each cell within a tumor, enabling new exploration of tumor heterogeneity, microenvironment cell-type composition, and cell state transitions that affect therapeutic response, particularly in the context of immunotherapy. Analyzing clinical samples has great promise for precision medicine but is technically challenging. Successfully identifying predictors of response requires well-coordinated, multi-disciplinary teams to ensure adequate sample processing for high-quality data generation and computational analysis for data interpretation. Here, we review current approaches to sample processing and computational analysis regarding their application to translational cancer immunotherapy research.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.ccell.2021.07.004 ">10.1016/j.ccell.2021.07.004 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 8  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading8">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse8" aria-expanded="true" aria-controls="collapse8">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">8. Multi-omic profiling of lung and liver tumor microenvironments of metastatic pancreatic cancer reveals site-specific immune regulatory pathways. (2021)</p>
						                 <p class="font-italic font-weight-light">Ho WJ, Erbe R, Danilova L, Phyo Z, Bigelow E, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Thomas DL 2nd, Charmsaz S, Gross N, Woolman S, Cruz K, Munday RM, Zaidi N, Armstrong TD, Sztein MB, Yarchoan M, Thompson ED, Jaffee EM, Fertig EJ</p>
						                 <p>Genome Biol. 2021 May 13;22(1):154. doi: 10.1186/s13059-021-02363-6. PMID: 33985562</p>
											</span>
										</a>
						      </div>

						      <div id="collapse8" class="collapse" aria-labelledby="heading8" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="33985562" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: The majority of pancreatic ductal adenocarcinomas (PDAC) are diagnosed at the metastatic stage, and standard therapies have limited activity with a dismal 5-year survival rate of only 8%. The liver and lung are the most common sites of PDAC metastasis, and each have been differentially associated with prognoses and responses to systemic therapies. A deeper understanding of the molecular and cellular landscape within the tumor microenvironment (TME) metastasis at these different sites is critical to informing future therapeutic strategies against metastatic PDAC. RESULTS: By leveraging combined mass cytometry, immunohistochemistry, and RNA sequencing, we identify key regulatory pathways that distinguish the liver and lung TMEs in a preclinical mouse model of metastatic PDAC. We demonstrate that the lung TME generally exhibits higher levels of immune infiltration, immune activation, and pro-immune signaling pathways, whereas multiple immune-suppressive pathways are emphasized in the liver TME. We then perform further validation of these preclinical findings in paired human lung and liver metastatic samples using immunohistochemistry from PDAC rapid autopsy specimens. Finally, in silico validation with transfer learning between our mouse model and TCGA datasets further demonstrates that many of the site-associated features are detectable even in the context of different primary tumors. CONCLUSIONS: Determining the distinctive immune-suppressive features in multiple liver and lung TME datasets provides further insight into the tissue specificity of molecular and cellular pathways, suggesting a potential mechanism underlying the discordant clinical responses that are often observed in metastatic diseases.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/s13059-021-02363-6 ">10.1186/s13059-021-02363-6 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 9  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading9">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse9" aria-expanded="true" aria-controls="collapse9">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">9. Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species. (2021)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Clark BS, Sherman T, Zibetti C, Hu Q, Sealfon R, Liu S, Qian J, Colantuoni C, Blackshaw S, Goff LA, Fertig EJ</p>
						                 <p>Cell Syst. 2021 Feb 17;12(2):203. doi: 10.1016/j.cels.2021.01.005. PMID: 33600760</p>
											</span>
										</a>
						      </div>

						      <div id="collapse9" class="collapse" aria-labelledby="heading9" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="33600760" class="altmetric-embed pull-left"> </div>
						          <p></p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.cels.2021.01.005 ">10.1016/j.cels.2021.01.005 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 10  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading10">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse10" aria-expanded="true" aria-controls="collapse10">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">10. Integrated immunological analysis of a successful conversion of locally advanced hepatocellular carcinoma to resectability with neoadjuvant therapy. (2020)</p>
						                 <p class="font-italic font-weight-light">Ho WJ, Sharma G, Zhu Q, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Durham J, Anders R, Popovic A, Mo G, Kamel I, Weiss M, Jaffee E, Fertig EJ, Yarchoan M</p>
						                 <p>J Immunother Cancer. 2020 Nov;8(2). pii: jitc-2020-000932. doi: 10.1136/jitc-2020-000932. PMID: 33219090</p>
											</span>
										</a>
						      </div>

						      <div id="collapse10" class="collapse" aria-labelledby="heading10" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="33219090" class="altmetric-embed pull-left"> </div>
						          <p>Hepatocellular carcinoma (HCC) is the fourth leading cause of cancer death worldwide with a minority of patients being diagnosed early enough for curative-intent interventions. We report the first use of preoperative cabozantinib plus nivolumab to successfully downstage what presented as unresectable HCC as part of an ongoing phase 1b study. Preoperative treatment with cabozantinib and nivolumab led to >99% reduction in alpha-fetoprotein, -37.3% radiographic reduction by RECIST 1.1 and a near complete pathologic response (80% to 100% necrosis). An integrated immunological analysis was performed on the post-treatment surgical tumor sample and matched pre-treatment and post-treatment peripheral blood samples with high-dimensional imaging and cytometry techniques. Bayesian non-negative matrix factorization (CoGAPS, Coordinated Gene Activity in Pattern Sets) and self-organizing map (FlowSOM) algorithms were used to distinguish changes in functional markers across cellular neighborhoods in the single cell data sets. Brisk immunological infiltration into the tumor microenvironment was observed in non-random, organized cellular neighborhoods. Systemically, combination therapy led to marked promotion of effector cytotoxic T cells and effector memory helper T cells. Natural killer cells also increased with therapy. The patient remains without disease recurrence and with a normal alpha-fetoprotein approximately 2 years from presentation. Our study provides proof-of-concept that borderline resectable or locally advanced HCC warrants consideration of downstaging with effective neoadjuvant systemic therapy for subsequent curative resection.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1136/jitc-2020-000932 ">10.1136/jitc-2020-000932 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 11  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading11">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse11" aria-expanded="true" aria-controls="collapse11">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">11. Parallel Social Information Processing Circuits Are Differentially Impacted in Autism. (2020)</p>
						                 <p class="font-italic font-weight-light">Lewis EM, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Patino AV, Nardou R, Grossman CD, Brown M, Bangamwabo B, Ndiaye N, Giovinazzo D, Dardani I, Jiang C, Goff LA, Dolen G</p>
						                 <p>Neuron. 2020 Nov 25;108(4):659-675.e6. doi: 10.1016/j.neuron.2020.10.002. Epub 2020 Oct 27. PMID: 33113347</p>
											</span>
										</a>
						      </div>

						      <div id="collapse11" class="collapse" aria-labelledby="heading11" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="33113347" class="altmetric-embed pull-left"> </div>
						          <p>Parallel processing circuits are thought to dramatically expand the network capabilities of the nervous system. Magnocellular and parvocellular oxytocin neurons have been proposed to subserve two parallel streams of social information processing, which allow a single molecule to encode a diverse array of ethologically distinct behaviors. Here we provide the first comprehensive characterization of magnocellular and parvocellular oxytocin neurons in male mice, validated across anatomical, projection target, electrophysiological, and transcriptional criteria. We next use novel multiple feature selection tools in Fmr1-KO mice to provide direct evidence that normal functioning of the parvocellular but not magnocellular oxytocin pathway is required for autism-relevant social reward behavior. Finally, we demonstrate that autism risk genes are enriched in parvocellular compared with magnocellular oxytocin neurons. Taken together, these results provide the first evidence that oxytocin-pathway-specific pathogenic mechanisms account for social impairments across a broad range of autism etiologies.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.neuron.2020.10.002 ">10.1016/j.neuron.2020.10.002 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 12  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading12">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse12" aria-expanded="true" aria-controls="collapse12">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">12. Single-Cell Analysis of Human Retina Identifies Evolutionarily Conserved and Species-Specific Mechanisms Controlling Development. (2020)</p>
						                 <p class="font-italic font-weight-light">Lu Y, Shiau F, Yi W, Lu S, Wu Q, Pearson JD, Kallman A, Zhong S, Hoang T, Zuo Z, Zhao F, Zhang M, Tsai N, Zhuo Y, He S, Zhang J, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Sherman TD, Duan X, Fertig EJ, Goff LA, Zack DJ, Handa JT, Xue T, Bremner R, Blackshaw S, Wang X, Clark BS</p>
						                 <p>Dev Cell. 2020 May 18;53(4):473-491.e9. doi: 10.1016/j.devcel.2020.04.009. Epub 2020 May 7. PMID: 32386599</p>
											</span>
										</a>
						      </div>

						      <div id="collapse12" class="collapse" aria-labelledby="heading12" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32386599" class="altmetric-embed pull-left"> </div>
						          <p>The development of single-cell RNA sequencing (scRNA-seq) has allowed high-resolution analysis of cell-type diversity and transcriptional networks controlling cell-fate specification. To identify the transcriptional networks governing human retinal development, we performed scRNA-seq analysis on 16 time points from developing retina as well as four early stages of retinal organoid differentiation. We identified evolutionarily conserved patterns of gene expression during retinal progenitor maturation and specification of all seven major retinal cell types. Furthermore, we identified gene-expression differences between developing macula and periphery and between distinct populations of horizontal cells. We also identified species-specific patterns of gene expression during human and mouse retinal development. Finally, we identified an unexpected role for ATOH7 expression in regulation of photoreceptor specification during late retinogenesis. These results provide a roadmap to future studies of human retinal development and may help guide the design of cell-based therapies for treating retinal dystrophies.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.devcel.2020.04.009 ">10.1016/j.devcel.2020.04.009 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 13  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading13">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse13" aria-expanded="true" aria-controls="collapse13">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">13. Variation of Human Neural Stem Cells Generating Organizer States In Vitro before Committing to Cortical Excitatory or Inhibitory Neuronal Fates. (2020)</p>
						                 <p class="font-italic font-weight-light">Micali N, Kim SK, Diaz-Bustamante M, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Seo S, Shin JH, Rash BG, Ma S, Wang Y, Olivares NA, Arellano JI, Maynard KR, Fertig EJ, Cross AJ, Burli RW, Brandon NJ, Weinberger DR, Chenoweth JG, Hoeppner DJ, Sestan N, Rakic P, Colantuoni C, McKay RD</p>
						                 <p>Cell Rep. 2020 May 5;31(5):107599. doi: 10.1016/j.celrep.2020.107599. PMID: 32375049</p>
											</span>
										</a>
						      </div>

						      <div id="collapse13" class="collapse" aria-labelledby="heading13" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32375049" class="altmetric-embed pull-left"> </div>
						          <p>Better understanding of the progression of neural stem cells (NSCs) in the developing cerebral cortex is important for modeling neurogenesis and defining the pathogenesis of neuropsychiatric disorders. Here, we use RNA sequencing, cell imaging, and lineage tracing of mouse and human in vitro NSCs and monkey brain sections to model the generation of cortical neuronal fates. We show that conserved signaling mechanisms regulate the acute transition from proliferative NSCs to committed glutamatergic excitatory neurons. As human telencephalic NSCs develop from pluripotency in vitro, they transition through organizer states that spatially pattern the cortex before generating glutamatergic precursor fates. NSCs derived from multiple human pluripotent lines vary in these early patterning states, leading differentially to dorsal or ventral telencephalic fates. This work furthers systematic analyses of the earliest patterning events that generate the major neuronal trajectories of the human telencephalon.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.celrep.2020.107599 ">10.1016/j.celrep.2020.107599 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 14  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading14">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse14" aria-expanded="true" aria-controls="collapse14">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">14. projectR: an R/Bioconductor package for transfer learning via PCA, NMF, correlation and clustering. (2020)</p>
						                 <p class="font-italic font-weight-light">Sharma G, Colantuoni C, Goff LA, Fertig EJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>,</p>
						                 <p>Bioinformatics. 2020 Jun 1;36(11):3592-3593. doi: 10.1093/bioinformatics/btaa183. PMID: 32167521</p>
											</span>
										</a>
						      </div>

						      <div id="collapse14" class="collapse" aria-labelledby="heading14" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32167521" class="altmetric-embed pull-left"> </div>
						          <p>MOTIVATION: Dimension reduction techniques are widely used to interpret high-dimensional biological data. Features learned from these methods are used to discover both technical artifacts and novel biological phenomena. Such feature discovery is critically importent in analysis of large single-cell datasets, where lack of a ground truth limits validation and interpretation. Transfer learning (TL) can be used to relate the features learned from one source dataset to a new target dataset to perform biologically driven validation by evaluating their use in or association with additional sample annotations in that independent target dataset. RESULTS: We developed an R/Bioconductor package, projectR, to perform TL for analyses of genomics data via TL of clustering, correlation and factorization methods. We then demonstrate the utility TL for integrated data analysis with an example for spatial single-cell analysis. AVAILABILITY AND IMPLEMENTATION: projectR is available on Bioconductor and at https://github.com/genesofeve/projectR. CONTACT: gsteinobrien@jhmi.edu or ejfertig@jhmi.edu. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btaa183 ">10.1093/bioinformatics/btaa183 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 15  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading15">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse15" aria-expanded="true" aria-controls="collapse15">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">15. Comprehensive analysis of a mouse model of spontaneous uveoretinitis using single-cell RNA sequencing. (2019)</p>
						                 <p class="font-italic font-weight-light">Heng JS, Hackett SF, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Winer BL, Williams J, Goff LA, Nathans J</p>
						                 <p>Proc Natl Acad Sci U S A. 2019 Dec 16. pii: 1915571116. doi: 10.1073/pnas.1915571116. PMID: 31843893</p>
											</span>
										</a>
						      </div>

						      <div id="collapse15" class="collapse" aria-labelledby="heading15" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31843893" class="altmetric-embed pull-left"> </div>
						          <p>Autoimmune uveoretinitis is a significant cause of visual loss, and mouse models offer unique opportunities to study its disease mechanisms. Aire (-/-) mice fail to express self-antigens in the thymus, exhibit reduced central tolerance, and develop a spontaneous, chronic, and progressive uveoretinitis. Using single-cell RNA sequencing (scRNA-seq), we characterized wild-type and Aire (-/-) retinas to define, in a comprehensive and unbiased manner, the cell populations and gene expression patterns associated with disease. Based on scRNA-seq, immunostaining, and in situ hybridization, we infer that 1) the dominant effector response in Aire (-/-) retinas is Th1-driven, 2) a subset of monocytes convert to either a macrophage/microglia state or a dendritic cell state, 3) the development of tertiary lymphoid structures constitutes part of the Aire (-/-) retinal phenotype, 4) all major resident retinal cell types respond to interferon gamma (IFNG) by changing their patterns of gene expression, and 5) Muller glia up-regulate specific genes in response to IFN gamma and may act as antigen-presenting cells.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1915571116 ">10.1073/pnas.1915571116 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 16  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading16">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse16" aria-expanded="true" aria-controls="collapse16">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">16. Differential Variation Analysis Enables Detection of Tumor Heterogeneity Using Single-Cell RNA-Sequencing Data. (2019)</p>
						                 <p class="font-italic font-weight-light">Davis-Marcisak EF, Sherman TD, Orugunta P, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Puram SV, Roussos Torres ET, Hopkins AC, Jaffee EM, Favorov AV, Afsari B, Goff LA, Fertig EJ</p>
						                 <p>Cancer Res. 2019 Oct 1;79(19):5102-5112. doi: 10.1158/0008-5472.CAN-18-3882. Epub 2019 Jul 23. PMID: 31337651</p>
											</span>
										</a>
						      </div>

						      <div id="collapse16" class="collapse" aria-labelledby="heading16" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31337651" class="altmetric-embed pull-left"> </div>
						          <p>Tumor heterogeneity provides a complex challenge to cancer treatment and is a critical component of therapeutic response, disease recurrence, and patient survival. Single-cell RNA-sequencing (scRNA-seq) technologies have revealed the prevalence of intratumor and intertumor heterogeneity. Computational techniques are essential to quantify the differences in variation of these profiles between distinct cell types, tumor subtypes, and patients to fully characterize intratumor and intertumor molecular heterogeneity. In this study, we adapted our algorithm for pathway dysregulation, Expression Variation Analysis (EVA), to perform multivariate statistical analyses of differential variation of expression in gene sets for scRNA-seq. EVA has high sensitivity and specificity to detect pathways with true differential heterogeneity in simulated data. EVA was applied to several public domain scRNA-seq tumor datasets to quantify the landscape of tumor heterogeneity in several key applications in cancer genomics such as immunogenicity, metastasis, and cancer subtypes. Immune pathway heterogeneity of hematopoietic cell populations in breast tumors corresponded to the amount of diversity present in the T-cell repertoire of each individual. Cells from head and neck squamous cell carcinoma (HNSCC) primary tumors had significantly more heterogeneity across pathways than cells from metastases, consistent with a model of clonal outgrowth. Moreover, there were dramatic differences in pathway dysregulation across HNSCC basal primary tumors. Within the basal primary tumors, there was increased immune dysregulation in individuals with a high proportion of fibroblasts present in the tumor microenvironment. These results demonstrate the broad utility of EVA to quantify intertumor and intratumor heterogeneity from scRNA-seq data without reliance on low-dimensional visualization. SIGNIFICANCE: This study presents a robust statistical algorithm for evaluating gene expression heterogeneity within pathways or gene sets in single-cell RNA-seq data.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1158/0008-5472.CAN-18-3882 ">10.1158/0008-5472.CAN-18-3882 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 17  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading17">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse17" aria-expanded="true" aria-controls="collapse17">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">17. Single-Cell RNA-Seq Analysis of Retinal Development Identifies NFI Factors as Regulating Mitotic Exit and Late-Born Cell Specification. (2019)</p>
						                 <p class="font-italic font-weight-light">Clark BS, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Shiau F, Cannon GH, Davis-Marcisak E, Sherman T, Santiago CP, Hoang TV, Rajaii F, James-Esposito RE, Gronostajski RM, Fertig EJ, Goff LA, Blackshaw S</p>
						                 <p>Neuron. 2019 Jun 19;102(6):1111-1126.e5. doi: 10.1016/j.neuron.2019.04.010. Epub 2019 May 22. PMID: 31128945</p>
											</span>
										</a>
						      </div>

						      <div id="collapse17" class="collapse" aria-labelledby="heading17" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31128945" class="altmetric-embed pull-left"> </div>
						          <p>Precise temporal control of gene expression in neuronal progenitors is necessary for correct regulation of neurogenesis and cell fate specification. However, the cellular heterogeneity of the developing CNS has posed a major obstacle to identifying the gene regulatory networks that control these processes. To address this, we used single-cell RNA sequencing to profile ten developmental stages encompassing the full course of retinal neurogenesis. This allowed us to comprehensively characterize changes in gene expression that occur during initiation of neurogenesis, changes in developmental competence, and specification and differentiation of each major retinal cell type. We identify the NFI transcription factors (Nfia, Nfib, and Nfix) as selectively expressed in late retinal progenitor cells and show that they control bipolar interneuron and Muller glia cell fate specification and promote proliferative quiescence.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.neuron.2019.04.010 ">10.1016/j.neuron.2019.04.010 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 18  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading18">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse18" aria-expanded="true" aria-controls="collapse18">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">18. Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species. (2019)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Clark BS, Sherman T, Zibetti C, Hu Q, Sealfon R, Liu S, Qian J, Colantuoni C, Blackshaw S, Goff LA, Fertig EJ</p>
						                 <p>Cell Syst. 2019 May 22;8(5):395-411.e8. doi: 10.1016/j.cels.2019.04.004. PMID: 31121116</p>
											</span>
										</a>
						      </div>

						      <div id="collapse18" class="collapse" aria-labelledby="heading18" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31121116" class="altmetric-embed pull-left"> </div>
						          <p>Analysis of gene expression in single cells allows for decomposition of cellular states as low-dimensional latent spaces. However, the interpretation and validation of these spaces remains a challenge. Here, we present scCoGAPS, which defines latent spaces from a source single-cell RNA-sequencing (scRNA-seq) dataset, and projectR, which evaluates these latent spaces in independent target datasets via transfer learning. Application of developing mouse retina to scRNA-Seq reveals intrinsic relationships across biological contexts and assays while avoiding batch effects and other technical features. We compare the dimensions learned in this source dataset to adult mouse retina, a time-course of human retinal development, select scRNA-seq datasets from developing brain, chromatin accessibility data, and a murine-cell type atlas to identify shared biological features. These tools lay the groundwork for exploratory analysis of scRNA-seq data via latent space representations, enabling a shift in how we compare and identify cells beyond reliance on marker genes or ensemble molecular identity.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.cels.2019.04.004 ">10.1016/j.cels.2019.04.004 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 19  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading19">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse19" aria-expanded="true" aria-controls="collapse19">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">19. CancerInSilico: An R/Bioconductor package for combining mathematical and statistical modeling to simulate time course bulk and single cell gene expression data in cancer. (2018)</p>
						                 <p class="font-italic font-weight-light">Sherman TD, Kagohara LT, Cao R, Cheng R, Satriano M, Considine M, Krigsfeld G, Ranaweera R, Tang Y, Jablonski SA, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Gaykalova DA, Weiner LM, Chung CH, Fertig EJ</p>
						                 <p>PLoS Comput Biol. 2019 Apr 19;14(4):e1006935. doi: 10.1371/journal.pcbi.1006935. eCollection 2018 Jun. PMID: 31002670</p>
											</span>
										</a>
						      </div>

						      <div id="collapse19" class="collapse" aria-labelledby="heading19" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31002670" class="altmetric-embed pull-left"> </div>
						          <p>Bioinformatics techniques to analyze time course bulk and single cell omics data are advancing. The absence of a known ground truth of the dynamics of molecular changes challenges benchmarking their performance on real data. Realistic simulated time-course datasets are essential to assess the performance of time course bioinformatics algorithms. We develop an R/Bioconductor package, CancerInSilico, to simulate bulk and single cell transcriptional data from a known ground truth obtained from mathematical models of cellular systems. This package contains a general R infrastructure for running cell-based models and simulating gene expression data based on the model states. We show how to use this package to simulate a gene expression data set and consequently benchmark analysis methods on this data set with a known ground truth. The package is freely available via Bioconductor: http://bioconductor.org/packages/CancerInSilico/.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1371/journal.pcbi.1006935 ">10.1371/journal.pcbi.1006935 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 20  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading20">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse20" aria-expanded="true" aria-controls="collapse20">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">20. Hypoxia tolerance in the Norrin-deficient retina and the chronically hypoxic brain studied at single-cell resolution. (2019)</p>
						                 <p class="font-italic font-weight-light">Heng JS, Rattner A, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Winer BL, Jones BW, Vernon HJ, Goff LA, Nathans J</p>
						                 <p>Proc Natl Acad Sci U S A. 2019 Apr 30;116(18):9103-9114. doi: 10.1073/pnas.1821122116. Epub 2019 Apr 15. PMID: 30988181</p>
											</span>
										</a>
						      </div>

						      <div id="collapse20" class="collapse" aria-labelledby="heading20" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30988181" class="altmetric-embed pull-left"> </div>
						          <p>The mammalian CNS is capable of tolerating chronic hypoxia, but cell type-specific responses to this stress have not been systematically characterized. In the Norrin KO (Ndp (KO) ) mouse, a model of familial exudative vitreoretinopathy (FEVR), developmental hypovascularization of the retina produces chronic hypoxia of inner nuclear-layer (INL) neurons and Muller glia. We used single-cell RNA sequencing, untargeted metabolomics, and metabolite labeling from (13)C-glucose to compare WT and Ndp (KO) retinas. In Ndp (KO) retinas, we observe gene expression responses consistent with hypoxia in Muller glia and retinal neurons, and we find a metabolic shift that combines reduced flux through the TCA cycle with increased synthesis of serine, glycine, and glutathione. We also used single-cell RNA sequencing to compare the responses of individual cell types in Ndp (KO) retinas with those in the hypoxic cerebral cortex of mice that were housed for 1 week in a reduced oxygen environment (7.5% oxygen). In the hypoxic cerebral cortex, glial transcriptome responses most closely resemble the response of Muller glia in the Ndp (KO) retina. In both retina and brain, vascular endothelial cells activate a previously dormant tip cell gene expression program, which likely underlies the adaptive neoangiogenic response to chronic hypoxia. These analyses of retina and brain transcriptomes at single-cell resolution reveal both shared and cell type-specific changes in gene expression in response to chronic hypoxia, implying both shared and distinct cell type-specific physiologic responses.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1821122116 ">10.1073/pnas.1821122116 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 21  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading21">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse21" aria-expanded="true" aria-controls="collapse21">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">21. Aging-like Spontaneous Epigenetic Silencing Facilitates Wnt Activation, Stemness, and Braf(V600E)-Induced Tumorigenesis. (2019)</p>
						                 <p class="font-italic font-weight-light">Tao Y, Kang B, Petkovich DA, Bhandari YR, In J, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kong X, Xie W, Zachos N, Maegawa S, Vaidya H, Brown S, Chiu Yen RW, Shao X, Thakor J, Lu Z, Cai Y, Zhang Y, Mallona I, Peinado MA, Zahnow CA, Ahuja N, Fertig E, Issa JP, Baylin SB, Easwaran H</p>
						                 <p>Cancer Cell. 2019 Feb 11;35(2):315-328.e6. doi: 10.1016/j.ccell.2019.01.005. PMID: 30753828</p>
											</span>
										</a>
						      </div>

						      <div id="collapse21" class="collapse" aria-labelledby="heading21" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30753828" class="altmetric-embed pull-left"> </div>
						          <p>We addressed the precursor role of aging-like spontaneous promoter DNA hypermethylation in initiating tumorigenesis. Using mouse colon-derived organoids, we show that promoter hypermethylation spontaneously arises in cells mimicking the human aging-like phenotype. The silenced genes activate the Wnt pathway, causing a stem-like state and differentiation defects. These changes render aged organoids profoundly more sensitive than young ones to transformation by Braf(V600E), producing the typical human proximal BRAF(V600E)-driven colon adenocarcinomas characterized by extensive, abnormal gene-promoter CpG-island methylation, or the methylator phenotype (CIMP). Conversely, CRISPR-mediated simultaneous inactivation of a panel of the silenced genes markedly sensitizes to Braf(V600E)-induced transformation. Our studies tightly link aging-like epigenetic abnormalities to intestinal cell fate changes and predisposition to oncogene-driven colon tumorigenesis.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.ccell.2019.01.005 ">10.1016/j.ccell.2019.01.005 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 22  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading22">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse22" aria-expanded="true" aria-controls="collapse22">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">22. Enter the Matrix: Factorization Uncovers Knowledge from Omics. (2018)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Arora R, Culhane AC, Favorov AV, Garmire LX, Greene CS, Goff LA, Li Y, Ngom A, Ochs MF, Xu Y, Fertig EJ</p>
						                 <p>Trends Genet. 2018 Oct;34(10):790-805. doi: 10.1016/j.tig.2018.07.003. Epub 2018 Aug 22. PMID: 30143323</p>
											</span>
										</a>
						      </div>

						      <div id="collapse22" class="collapse" aria-labelledby="heading22" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30143323" class="altmetric-embed pull-left"> </div>
						          <p>Omics data contain signals from the molecular, physical, and kinetic inter- and intracellular interactions that control biological systems. Matrix factorization (MF) techniques can reveal low-dimensional structure from high-dimensional data that reflect these interactions. These techniques can uncover new biological knowledge from diverse high-throughput omics data in applications ranging from pathway discovery to timecourse analysis. We review exemplary applications of MF for systems-level analyses. We discuss appropriate applications of these methods, their limitations, and focus on the analysis of results to facilitate optimal biological interpretation. The inference of biologically relevant features with MF enables discovery from high-throughput data beyond the limits of current biological knowledge - answering questions from high-dimensional data that we have not yet thought to ask.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.tig.2018.07.003 ">10.1016/j.tig.2018.07.003 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 23  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading23">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse23" aria-expanded="true" aria-controls="collapse23">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">23. Integrated time course omics analysis distinguishes immediate therapeutic response from acquired resistance. (2018)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kagohara LT, Li S, Thakar M, Ranaweera R, Ozawa H, Cheng H, Considine M, Schmitz S, Favorov AV, Danilova LV, Califano JA, Izumchenko E, Gaykalova DA, Chung CH, Fertig EJ</p>
						                 <p>Genome Med. 2018 May 23;10(1):37. doi: 10.1186/s13073-018-0545-2. PMID: 29792227</p>
											</span>
										</a>
						      </div>

						      <div id="collapse23" class="collapse" aria-labelledby="heading23" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="29792227" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: Targeted therapies specifically act by blocking the activity of proteins that are encoded by genes critical for tumorigenesis. However, most cancers acquire resistance and long-term disease remission is rarely observed. Understanding the time course of molecular changes responsible for the development of acquired resistance could enable optimization of patients' treatment options. Clinically, acquired therapeutic resistance can only be studied at a single time point in resistant tumors. METHODS: To determine the dynamics of these molecular changes, we obtained high throughput omics data (RNA-sequencing and DNA methylation) weekly during the development of cetuximab resistance in a head and neck cancer in vitro model. The CoGAPS unsupervised algorithm was used to determine the dynamics of the molecular changes associated with resistance during the time course of resistance development. RESULTS: CoGAPS was used to quantify the evolving transcriptional and epigenetic changes. Applying a PatternMarker statistic to the results from CoGAPS enabled novel heatmap-based visualization of the dynamics in these time course omics data. We demonstrate that transcriptional changes result from immediate therapeutic response or resistance, whereas epigenetic alterations only occur with resistance. Integrated analysis demonstrates delayed onset of changes in DNA methylation relative to transcription, suggesting that resistance is stabilized epigenetically. CONCLUSIONS: Genes with epigenetic alterations associated with resistance that have concordant expression changes are hypothesized to stabilize the resistant phenotype. These genes include FGFR1, which was associated with EGFR inhibitors resistance previously. Thus, integrated omics analysis distinguishes the timing of molecular drivers of resistance. This understanding of the time course progression of molecular changes in acquired resistance is important for the development of alternative treatment strategies that would introduce appropriate selection of new drugs to treat cancer before the resistant phenotype develops.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/s13073-018-0545-2 ">10.1186/s13073-018-0545-2 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 24  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading24">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse24" aria-expanded="true" aria-controls="collapse24">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">24. Splice Expression Variation Analysis (SEVA) for inter-tumor heterogeneity of gene isoform usage in cancer. (2018)</p>
						                 <p class="font-italic font-weight-light">Afsari B, Guo T, Considine M, Florea L, Kagohara LT, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kelley D, Flam E, Zambo KD, Ha PK, Geman D, Ochs MF, Califano JA, Gaykalova DA, Favorov AV, Fertig EJ</p>
						                 <p>Bioinformatics. 2018 Jun 1;34(11):1859-1867. doi: 10.1093/bioinformatics/bty004. PMID: 29342249</p>
											</span>
										</a>
						      </div>

						      <div id="collapse24" class="collapse" aria-labelledby="heading24" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="29342249" class="altmetric-embed pull-left"> </div>
						          <p>Motivation: Current bioinformatics methods to detect changes in gene isoform usage in distinct phenotypes compare the relative expected isoform usage in phenotypes. These statistics model differences in isoform usage in normal tissues, which have stable regulation of gene splicing. Pathological conditions, such as cancer, can have broken regulation of splicing that increases the heterogeneity of the expression of splice variants. Inferring events with such differential heterogeneity in gene isoform usage requires new statistical approaches. Results: We introduce Splice Expression Variability Analysis (SEVA) to model increased heterogeneity of splice variant usage between conditions (e.g. tumor and normal samples). SEVA uses a rank-based multivariate statistic that compares the variability of junction expression profiles within one condition to the variability within another. Simulated data show that SEVA is unique in modeling heterogeneity of gene isoform usage, and benchmark SEVA's performance against EBSeq, DiffSplice and rMATS that model differential isoform usage instead of heterogeneity. We confirm the accuracy of SEVA in identifying known splice variants in head and neck cancer and perform cross-study validation of novel splice variants. A novel comparison of splice variant heterogeneity between subtypes of head and neck cancer demonstrated unanticipated similarity between the heterogeneity of gene isoform usage in HPV-positive and HPV-negative subtypes and anticipated increased heterogeneity among HPV-negative samples with mutations in genes that regulate the splice variant machinery. These results show that SEVA accurately models differential heterogeneity of gene isoform usage from RNA-seq data. Availability and implementation: SEVA is implemented in the R/Bioconductor package GSReg. Contact: bahman@jhu.edu or favorov@sensi.org or ejfertig@jhmi.edu. Supplementary information: Supplementary data are available at Bioinformatics online.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bty004 ">10.1093/bioinformatics/bty004 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 25  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading25">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse25" aria-expanded="true" aria-controls="collapse25">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">25. Epigenetic regulation of gene expression in cancer: techniques, resources and analysis. (2018)</p>
						                 <p class="font-italic font-weight-light">Kagohara LT, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kelley D, Flam E, Wick HC, Danilova LV, Easwaran H, Favorov AV, Qian J, Gaykalova DA, Fertig EJ</p>
						                 <p>Brief Funct Genomics. 2018 Jan 1;17(1):49-63. doi: 10.1093/bfgp/elx018. PMID: 28968850</p>
											</span>
										</a>
						      </div>

						      <div id="collapse25" class="collapse" aria-labelledby="heading25" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="28968850" class="altmetric-embed pull-left"> </div>
						          <p>Cancer is a complex disease, driven by aberrant activity in numerous signaling pathways in even individual malignant cells. Epigenetic changes are critical mediators of these functional changes that drive and maintain the malignant phenotype. Changes in DNA methylation, histone acetylation and methylation, noncoding RNAs, posttranslational modifications are all epigenetic drivers in cancer, independent of changes in the DNA sequence. These epigenetic alterations were once thought to be crucial only for the malignant phenotype maintenance. Now, epigenetic alterations are also recognized as critical for disrupting essential pathways that protect the cells from uncontrolled growth, longer survival and establishment in distant sites from the original tissue. In this review, we focus on DNA methylation and chromatin structure in cancer. The precise functional role of these alterations is an area of active research using emerging high-throughput approaches and bioinformatics analysis tools. Therefore, this review also describes these high-throughput measurement technologies, public domain databases for high-throughput epigenetic data in tumors and model systems and bioinformatics algorithms for their analysis. Advances in bioinformatics data that combine these epigenetic data with genomics data are essential to infer the function of specific epigenetic alterations in cancer. These integrative algorithms are also a focus of this review. Future studies using these emerging technologies will elucidate how alterations in the cancer epigenome cooperate with genetic aberrations during tumor initiation and progression. This deeper understanding is essential to future studies with epigenetics biomarkers and precision medicine using emerging epigenetic therapies.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1093/bfgp/elx018 ">10.1093/bfgp/elx018 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 26  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading26">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse26" aria-expanded="true" aria-controls="collapse26">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">26. PatternMarkers & GWCoGAPS for novel data-driven biomarkers via whole transcriptome NMF. (2017)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Carey JL, Lee WS, Considine M, Favorov AV, Flam E, Guo T, Li S, Marchionni L, Sherman T, Sivy S, Gaykalova DA, McKay RD, Ochs MF, Colantuoni C, Fertig EJ</p>
						                 <p>Bioinformatics. 2017 Jun 15;33(12):1892-1894. doi: 10.1093/bioinformatics/btx058. PMID: 28174896</p>
											</span>
										</a>
						      </div>

						      <div id="collapse26" class="collapse" aria-labelledby="heading26" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="28174896" class="altmetric-embed pull-left"> </div>
						          <p>SUMMARY: Non-negative Matrix Factorization (NMF) algorithms associate gene expression with biological processes (e.g. time-course dynamics or disease subtypes). Compared with univariate associations, the relative weights of NMF solutions can obscure biomarkers. Therefore, we developed a novel patternMarkers statistic to extract genes for biological validation and enhanced visualization of NMF results. Finding novel and unbiased gene markers with patternMarkers requires whole-genome data. Therefore, we also developed Genome-Wide CoGAPS Analysis in Parallel Sets (GWCoGAPS), the first robust whole genome Bayesian NMF using the sparse, MCMC algorithm, CoGAPS. Additionally, a manual version of the GWCoGAPS algorithm contains analytic and visualization tools including patternMatcher, a Shiny web application. The decomposition in the manual pipeline can be replaced with any NMF algorithm, for further generalization of the software. Using these tools, we find granular brain-region and cell-type specific signatures with corresponding biomarkers in GTEx data, illustrating GWCoGAPS and patternMarkers ascertainment of data-driven biomarkers from whole-genome data. AVAILABILITY AND IMPLEMENTATION: PatternMarkers & GWCoGAPS are in the CoGAPS Bioconductor package (3.5) under the GPL license. CONTACT: gsteinobrien@jhmi.edu or ccolantu@jhmi.edu or ejfertig@jhmi.edu. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btx058 ">10.1093/bioinformatics/btx058 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 27  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading27">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse27" aria-expanded="true" aria-controls="collapse27">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">27. CoGAPS matrix factorization algorithm identifies transcriptional changes in AP-2alpha target genes in feedback from therapeutic inhibition of the EGFR network. (2016)</p>
						                 <p class="font-italic font-weight-light">Fertig EJ, Ozawa H, Thakar M, Howard JD, Kagohara LT, Krigsfeld G, Ranaweera RS, Hughes RM, Perez J, Jones S, Favorov AV, Carey J, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Gaykalova DA, Ochs MF, Chung CH</p>
						                 <p>Oncotarget. 2016 Nov 8;7(45):73845-73864. doi: 10.18632/oncotarget.12075. PMID: 27650546</p>
											</span>
										</a>
						      </div>

						      <div id="collapse27" class="collapse" aria-labelledby="heading27" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="27650546" class="altmetric-embed pull-left"> </div>
						          <p>Patients with oncogene driven tumors are treated with targeted therapeutics including EGFR inhibitors. Genomic data from The Cancer Genome Atlas (TCGA) demonstrates molecular alterations to EGFR, MAPK, and PI3K pathways in previously untreated tumors. Therefore, this study uses bioinformatics algorithms to delineate interactions resulting from EGFR inhibitor use in cancer cells with these genetic alterations. We modify the HaCaT keratinocyte cell line model to simulate cancer cells with constitutive activation of EGFR, HRAS, and PI3K in a controlled genetic background. We then measure gene expression after treating modified HaCaT cells with gefitinib, afatinib, and cetuximab. The CoGAPS algorithm distinguishes a gene expression signature associated with the anticipated silencing of the EGFR network. It also infers a feedback signature with EGFR gene expression itself increasing in cells that are responsive to EGFR inhibitors. This feedback signature has increased expression of several growth factor receptors regulated by the AP-2 family of transcription factors. The gene expression signatures for AP-2alpha are further correlated with sensitivity to cetuximab treatment in HNSCC cell lines and changes in EGFR expression in HNSCC tumors with low CDKN2A gene expression. In addition, the AP-2alpha gene expression signatures are also associated with inhibition of MEK, PI3K, and mTOR pathways in the Library of Integrated Network-Based Cellular Signatures (LINCS) data. These results suggest that AP-2 transcription factors are activated as feedback from EGFR network inhibition and may mediate EGFR inhibitor resistance.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.18632/oncotarget.12075 ">10.18632/oncotarget.12075 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 28  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading28">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse28" aria-expanded="true" aria-controls="collapse28">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">28. Latent tri-lineage potential of adult hippocampal neural stem cells revealed by Nf1 inactivation. (2015)</p>
						                 <p class="font-italic font-weight-light">Sun GJ, Zhou Y, Ito S, Bonaguidi MA, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Kawasaki NK, Modak N, Zhu Y, Ming GL, Song H</p>
						                 <p>Nat Neurosci. 2015 Dec;18(12):1722-4. doi: 10.1038/nn.4159. Epub 2015 Nov 2. PMID: 26523645</p>
											</span>
										</a>
						      </div>

						      <div id="collapse28" class="collapse" aria-labelledby="heading28" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="26523645" class="altmetric-embed pull-left"> </div>
						          <p>Endogenous neural stem cells (NSCs) in the adult hippocampus are considered to be bi-potent, as they only produce neurons and astrocytes in vivo. In mouse, we found that inactivation of neurofibromin 1 (Nf1), a gene mutated in neurofibromatosis type 1, unlocked a latent oligodendrocyte lineage potential to produce all three lineages from NSCs in vivo. Our results suggest an avenue for promoting stem cell plasticity by targeting barriers of latent lineage potential.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nn.4159 ">10.1038/nn.4159 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
								<!--  Card 29  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading29">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse29" aria-expanded="true" aria-controls="collapse29">
											<span class="pull-left text-left">

														 <p class="font-weight-bold">29. Pattern identification in time-course gene expression data with the CoGAPS matrix factorization. (2014)</p>
						                 <p class="font-italic font-weight-light">Fertig EJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Stein-O'Brien GL</u></span>, Jaffe A, Colantuoni C</p>
						                 <p>Methods Mol Biol. 2014;1101:87-112. doi: 10.1007/978-1-62703-721-1_6. PMID: 24233779</p>
											</span>
										</a>
						      </div>

						      <div id="collapse29" class="collapse" aria-labelledby="heading29" data-parent="#accordionpubs">
						        <div class="card-body" >
											<div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24233779" class="altmetric-embed pull-left"> </div>
						          <p>Patterns in time-course gene expression data can represent the biological processes that are active over the measured time period. However, the orthogonality constraint in standard pattern-finding algorithms, including notably principal components analysis (PCA), confounds expression changes resulting from simultaneous, non-orthogonal biological processes. Previously, we have shown that Markov chain Monte Carlo nonnegative matrix factorization algorithms are particularly adept at distinguishing such concurrent patterns. One such matrix factorization is implemented in the software package CoGAPS. We describe the application of this software and several technical considerations for identification of age-related patterns in a public, prefrontal cortex gene expression dataset.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1007/978-1-62703-721-1_6 ">10.1007/978-1-62703-721-1_6 </a>
						          
																	        </div>
						      </div>
						    </div>
						    
						  </div>
						    <!-- DA neuron preprint
						    <li>
						        <strong><em>Temporal and spatial variation among single dopaminergic neuron transcriptomes informs cellular phenotype diversity and Parkinson's Disease gene prioritization</em> (2017)</strong>
						                 <p class="note">Paul Hook, Sarah A. McClymont, Gabrielle H. Cannon, William D. Law, Loyal A. Goff<sup>*</sup>, Andrew S. McCallion<sup>*</sup><br>
						                 doi: <a href="https://doi.org/10.1101/148049" target="_blank">https://doi.org/10.1101/148049</a></br>
						          <p>Parkinson's Disease (PD) causes collapse of substantia nigra (SN) dopaminergic (DA) neurons of the midbrain (MB), while other DA populations are relatively spared. Here, we used single-cell RNA-seq (scRNA-seq) to characterize DA neuron populations in the mouse brain at embryonic and postnatal timepoints. These data allow for the discrimination between olfactory bulb (OB), forebrain (FB), and MB DA populations as well identification of subpopulations of DA neurons in each region. We observe a longitudinal axis of MB DA development, during which specialization and heterogeneity increases. We identify three distinct subpopulations of known MB DA neurons and provide evidence of a postnatal MB DA precursor, identifying novel markers for each subpopulation. Further, we discover gene regulatory networks (GRNs) that are significantly associated with neurodegenerative diseases and highly correlated with specific DA neuron subpopulations. By integrating these data with published genome-wide association studies (GWAS), we prioritize candidate genes in all 32 PD associated loci. Collectively, our data reveal genes and pathways that may begin to explain the selective vulnerability of SN DA neurons and allow for the systematic prioritization of genes in PD GWAS loci for functional evaluation.
						            <a target="_blank" href="http://biorxiv.org/content/early/2017/06/09/148049">Preprint</a><p>
								</li>
								-->
					</section>
			</div>
		</div>
</div>

		<!-- Footer -->
  <footer id="footer">
    <ul class="icons">
      <li><a href="https://twitter.com/intent/tweet?original_referer=https%3A%2F%2Fwww.sharelinkgenerator.com%2F&ref_src=twsrc%5Etfw&text=Share%20Link%20Generator%3A%20Facebook%2C%20Twitter%2C%20LinkedIn%2C%20Pinterest%2C%20and%20Email%20mailto%20forms&tw_p=tweetbutton&url=https%3A%2F%2Fwww.sharelinkgenerator.com%2F&via=twitterapi" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
      <!-- <li><a href="#" class="icon brands alt fa-facebook-f"><span class="label">Facebook</span></a></li> -->
      <!-- <li><a href="#" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li> -->
      <!-- <li><a href="#" class="icon brands alt fa-instagram"><span class="label">Instagram</span></a></li> -->
      <li><a href="http://github.com/genesofeve/" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
      <li><a href="mailto:gsteinobrien@jhmi.edu" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
    </ul>
    <ul class="copyright">
      <li>&copy; Stein-O'Brien Lab - JHU. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
    </ul>
  </footer>
		<!-- Scripts -->
<script src="assets/js/jquery.min.js"></script>
<script src="assets/js/jquery.scrolly.min.js"></script>
<script src="assets/js/jquery.dropotron.min.js"></script>
<script src="assets/js/jquery.scrollex.min.js"></script>
<script src="assets/js/browser.min.js"></script>
<script src="assets/js/breakpoints.min.js"></script>
<script src="assets/js/util.js"></script>
<script src="assets/js/main.js"></script>
<script src="assets/js/jquery-ui.min.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script>
  $( function() {
    $( "#accordion" ).accordion();
  } );
  </script>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-318359-8', 'auto');
ga('send', 'pageview');
</script>
	</body>
</html>
