{% extends "base.html" %}

{% block title %}
  Publications
{% endblock title %}

{% block content %}
<h2>Summary</h2>
<ul>
    <li><a href="http://europepmc.org/authors/0000-0003-2875-451X" target="_blank">EBI Summary page</a></li>
    <li><a href="https://scholar.google.com/citations?user=2ZVv2v8AAAAJ&hl=en" target="_blank">Google Scholar</a></li>
  </ul>
<h2>Preprints</h2>
<div class="panel-group" id="preprint-accordion">

 <!--Retina Methods preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint6">
                 <em>Comprehensive analysis of retinal development at single cell resolution identifies NFI factors as essential for mitotic exit and specification of late-born cells</em> (2018)</a>
                 <p class="note"> Genevieve L Stein-O'Brien, Brian S. Clark, Thomas Sherman, Christina Zibetti, Qiwen Hu, Rachel Sealfon, Sheng Liu, Jiang Qian, Carlo Colantuoni, Seth Blackshaw, Loyal A. Goff, Elana J. Fertig<br>
                 doi: <a href=https://doi.org/10.1101/395004" target="_blank">https://doi.org/10.1101/395004</a></p>
        </span>
      </div>
      <div id="preprint6" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="46741593" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>New approaches are urgently needed to glean biological insights from the vast amounts of single cell RNA sequencing (scRNA-Seq) data now being generated. To this end, we propose that cell identity should map to a reduced set of factors which will describe both exclusive and shared biology of individual cells, and that the dimensions which contain these factors reflect biologically meaningful relationships across different platforms, tissues and species. To find a robust set of dependent factors in large-scale scRNA-Seq data, we developed a Bayesian non-negative matrix factorization (NMF) algorithm, scCoGAPS. Application of scCoGAPS to scRNA-Seq data obtained over the course of mouse retinal development identified gene expression signatures for factors associated with specific cell types and continuous biological processes. To test whether these signatures are shared across diverse cellular contexts, we developed projectR to map biologically disparate datasets into the factors learned by scCoGAPS. Because projecting these dimensions preserve relative distances between samples, biologically meaningful relationships/factors will stratify new data consistent with their underlying processes, allowing labels or information from one dataset to be used for annotation of the other - a machine learning concept called transfer learning. Using projectR, data from multiple datasets was used to annotate latent spaces and reveal novel parallels between developmental programs in other tissues, species and cellular assays. Using this approach we are able to transfer cell type and state designations across datasets to rapidly annotate cellular features in a new dataset without a priori knowledge of their type, identify a species-specific signature of microglial cells, and identify a previously undescribed subpopulation of neurosecretory cells within the lung. Together, these algorithms define biologically meaningful dimensions of cellular identity, state, and trajectories that persist across technologies, molecular features, and species.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/08/20/395004.1">Preprint</a>
        </div>
      </div>
    </div>

 <!--Retina Bio preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint5">
                 <em>Comprehensive analysis of retinal development at single cell resolution identifies NFI factors as essential for mitotic exit and specification of late-born cells</em> (2018)</a>
                 <p class="note"> Brian Clark, Genevieve Stein-O'Brien, Fion Shiau, Gabrielle Cannon, Emily Davis, Thomas Sherman, Fatemeh Rajaii, Rebecca James-Esposito, Richard Gronostajski, Elana Fertig, Loyal A. Goff, Seth Blackshaw<br>
                 doi: <a href="https://doi.org/10.1101/378950" target="_blank">https://doi.org/10.1101/378950</a></p>
        </span>
      </div>
      <div id="preprint5" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="45656869" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Precise temporal control of gene expression in neuronal progenitors is necessary for correct regulation of neurogenesis and cell fate specification. However, the extensive cellular heterogeneity of the developing CNS has posed a major obstacle to identifying the gene regulatory networks that control these processes. To address this, we used single cell RNA-sequencing to profile ten developmental stages encompassing the full course of retinal neurogenesis. This allowed us to comprehensively characterize changes in gene expression that occur during initiation of neurogenesis, changes in developmental competence, and specification and differentiation of each of the major retinal cell types. These data identify transitions in gene expression between early and late-stage retinal progenitors, as well as a classification of neurogenic progenitors. We identify here the NFI family of transcription factors (Nfia, Nfib, and Nfix) as genes with enriched expression within late RPCs, and show they are regulators of bipolar interneuron and Muller glia specification and the control of proliferative quiescence.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/07/30/378950">Preprint</a>
        </div>
      </div>
    </div>

 <!--Matrix review preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint4">
                 <em>Enter the matrix: Interpreting unsupervised feature learning with matrix decomposition to discover hidden knowledge in high-throughput omics data.</em> (2017)</a>
                 <p class="note"> Genevieve L. Stein-O'Brien, Raman Arora, Aedin C. Culhane, Alexander Favorov, Casey Greene, Loyal A. Goff, Yifeng Li, Alioune Ngom, Michael F. Ochs, Yanxun Xu, Elana J. Fertig<br>
                 doi: <a href="https://doi.org/10.1101/196915" target="_blank">https://doi.org/10.1101/196915</a></p>
        </span>
      </div>
      <div id="preprint4" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="26912140" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>High-dimensional data is currently standard for biological inquiry. Biological systems are comprised of interrelated gene regulatory mechanisms, gene-gene interactions, and cellular interactions. These interactions induce low-dimensional structure within the high-dimensional data. Matrix factorization, also known as compressed sensing, learns low-dimensional mathematical representations from high-dimensional data. These factorization techniques can embed assumptions about pleiotropy, epistasis, inter-relationships between complex traits, and context-dependent interactions. They have been applied to uncover new biological knowledge in a breadth of topics ranging from pathway discovery to time course analysis. These techniques have been applied to data from diverse high-throughput omics technologies, including bulk and single-cell data. There are numerous computational techniques within the class of matrix factorization, each of which provides a unique interpretation of the processes in high-dimensional data. We review the visualization and applications of matrix factorization to systems-level analyses, which are diverse and require standardization to enable biological interpretation. Codifying the techniques to decipher biologically relevant features with matrix factorization enables their broad application to discovery beyond the limits of current biological knowledge; answering questions from high-dimensional data that we have not yet thought to ask.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2017/10/02/196915.1">Preprint</a>
        </div>
      </div>
    </div>

 <!--Hansen preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint3">
                 <em>Linear models enable powerful differential activity analysis in massively parallel reporter assays.</em> (2017)</a>
                 <p class="note"> Leslie Myint, Dimitrios G. Avramopoulos, Loyal A. Goff, Kasper Hansen<br>
                 doi: <a href="https://doi.org/10.1101/196394" target="_blank">https://doi.org/10.1101/196394</a></p>
        </span>
      </div>
      <div id="preprint3" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="26861189" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Massively parallel reporter assays (MPRAs) have emerged as a popular means for understanding noncoding variation in a variety of conditions. However, development of statistical analysis methods has not kept pace with the use of this assay. We present a linear model framework, mpralm, for the differential analysis of activity measures from these experiments that we show is calibrated and powerful. We show that it outperforms statistical tests that are commonly used in the literature. Further, we investigate theoretical and real-data properties of barcode summarization methods. These results inform recommendations for differential analysis, general group comparisons, and study design in terms of sample sizes and sequencing depth.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2017/09/30/196394">Preprint</a>
        </div>
      </div>
    </div>
    
    <!--CthPN preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint2">
                 <em>Variation in neuronal activity state, axonal projection target, and position principally define the transcriptional identity of individual neocortical projection neurons.</em> (2017)</a>
                 <p class="note"> Maxime Chev&eacute;e, Johanna D. Robertson, Gabrielle H. Cannon, Solange P. Brown<sup>*</sup>, Loyal A. Goff<sup>*</sup><br>
                 doi: <a href="https://doi.org/10.1101/157149" target="_blank">https://doi.org/10.1101/1571499</a></p>
        </span>
      </div>
      <div id="preprint2" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="21401403" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Single-cell RNA sequencing technologies have generated the first catalogs of transcriptionally defined neuronal subtypes of the brain. However, the biologically informative cellular processes that contribute to neuronal subtype specification and transcriptional heterogeneity remain unclear. By comparing the gene expression profiles of single layer 6 corticothalamic neurons in somatosensory cortex, we show that transcriptional subtypes primarily reflect axonal projection pattern, laminar position within the cortex, and neuronal activity state. Pseudotemporal ordering of 1023 cellular responses to manipulation of sensory input demonstrates that changes in expression of activity-induced genes both reinforced cell-type identity and contributed to increased transcriptional heterogeneity within each cell type. This is due to cell-type specific biases in the choice of transcriptional states following manipulation of neuronal activity. These results reveal that axonal projection pattern, laminar position, and activity state define significant axes of variation that contribute both to the transcriptional identity of individual neurons and to the transcriptional heterogeneity within each neuronal subtype.</p>
            <a href="https://f1000.com/prime/732442069?bd=1" target="_blank"><img src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif" alt="Access the recommendation on F1000Prime" id="bg" /></a><br>
            <a target="_blank" href="http://www.biorxiv.org/content/early/2017/06/30/157149">Preprint</a>
        </div>
      </div>
    </div>


    <!-- DA neuron preprint -->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint1">
                 <em>Temporal and spatial variation among single dopaminergic neuron transcriptomes informs cellular phenotype diversity and Parkinson's Disease gene prioritization</em> (2017)</a>
                 <p class="note">Paul Hook, Sarah A. McClymont, Gabrielle H. Cannon, William D. Law, Loyal A. Goff<sup>*</sup>, Andrew S. McCallion<sup>*</sup><br>
                 doi: <a href="https://doi.org/10.1101/148049" target="_blank">https://doi.org/10.1101/148049</a></p>
        </span>
      </div>
      <div id="preprint1" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="20930309" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Parkinson's Disease (PD) causes collapse of substantia nigra (SN) dopaminergic (DA) neurons of the midbrain (MB), while other DA populations are relatively spared. Here, we used single-cell RNA-seq (scRNA-seq) to characterize DA neuron populations in the mouse brain at embryonic and postnatal timepoints. These data allow for the discrimination between olfactory bulb (OB), forebrain (FB), and MB DA populations as well identification of subpopulations of DA neurons in each region. We observe a longitudinal axis of MB DA development, during which specialization and heterogeneity increases. We identify three distinct subpopulations of known MB DA neurons and provide evidence of a postnatal MB DA precursor, identifying novel markers for each subpopulation. Further, we discover gene regulatory networks (GRNs) that are significantly associated with neurodegenerative diseases and highly correlated with specific DA neuron subpopulations. By integrating these data with published genome-wide association studies (GWAS), we prioritize candidate genes in all 32 PD associated loci. Collectively, our data reveal genes and pathways that may begin to explain the selective vulnerability of SN DA neurons and allow for the systematic prioritization of genes in PD GWAS loci for functional evaluation.</p>
            <a target="_blank" href="http://biorxiv.org/content/early/2017/06/09/148049">Preprint</a>
        </div>
      </div>
    </div>
  </div>

<h2>Publications</h2>
  
  <div class="panel-group" id="accordion">
    {% for record in records %}
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse{{ loop.index }}">
                 <em>{{ record['TI'] }}</em> ({{ record['DP'][:4] }})</a>
                 <p class="note">{{ ", ".join(record['AU'])}}<br>
                 {{ record['SO'] }}</p>
        </span>
      </div>
      <div id="collapse{{ loop.index }}" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="{{record['PMID']}}" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>{{ record['AB'] }}</p>
          {% if record['LID'] is defined%}
              <a target="_blank" href="http://dx.doi.org/{{ record['LID'].split('[doi]')[0] }}">{{ record['LID'].split('[doi]')[0] }}</a>
          {% else %}
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/{{ record['PMID'] }}">Article</a>
          {% endif %}
        </div>
      </div>
    </div>
    {% endfor %}
  </div>

{% endblock content %}
