{% extends "base.html" %}

{% block title %}
  Publications
{% endblock title %}

{% block content %}

<h2>Summary</h2>
<ul>
    <li><a href="http://europepmc.org/authors/0000-0003-2875-451X" target="_blank">EBI Summary page</a></li>
    <li><a href="https://scholar.google.com/citations?user=2ZVv2v8AAAAJ&hl=en" target="_blank">Google Scholar</a></li>
  </ul>
<h2>Preprints</h2>
<div class="panel-group" id="preprint-accordion">

<!--ProjectR Appnote preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint10">
                 <em>projectR: An R/Bioconductor package for transfer learning via PCA, NMF, correlation, and clustering</em> (2019)</a>
                 <p class="note">Gaurav Sharma, Carlo Colantuoni, Loyal A Goff, Genevieve L Stein-O'Brien, Elana Fertig<br>
                 doi: <a href=https://doi.org/10.1101/726547" target="_blank">https://doi.org/10.1101/726547</a></p>
        </span>
      </div>
      <div id="preprint10" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="64639523" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Motivation: Dimension reduction techniques are widely used to interpret high-dimensional biological data. Features learned from these methods are used to discover both technical artifacts and novel biological phenomena. Such feature discovery is critically import to large single-cell datasets, where lack of a ground truth limits validation and interpretation. Transfer learning (TL) can be used to relate the features learned from one source dataset to a new target dataset to perform biologically-driven validation by evaluating their use in or association with additional sample annotations in that independent target dataset. Results: We developed an R/Bioconductor package, projectR, to perform TL for analyses of genomics data via TL of clustering, correlation, and factorization methods. We then demonstrate the utility TL for integrated data analysis with an example for spatial single-cell analysis. Availability: projectR is available on Bioconductor and at https://github.com/genesofeve/projectR.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/10.1101/726547v1">Preprint</a>
        </div>
      </div>
    </div>

<!--KS1 preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint9">
                 <em>Transcriptional suppression from KMT2D loss disrupts cell cycle and hypoxic responses in neurodevelopmental models of Kabuki syndrome</em> (2019)</a>
                 <p class="note"> Giovanni A Carosso, Leandros Boukas, Jonathan J Augustin, Ha Nam Nguyen, Briana L Winer, Gabrielle H Cannon, Johanna D Robertson, Li Zhang, Kasper D Hansen, Loyal A Goff, Hans T Bjornsson<br>
                 doi: <a href=https://doi.org/10.1101/484410" target="_blank">https://doi.org/10.1101/484410</a></p>
        </span>
      </div>
      <div id="preprint9" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="52114227" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Chromatin modifiers act to coordinate gene expression changes critical to neuronal differentiation from neural progenitor cells (NPCs). Histone methyltransferase KMT2D promotes transcriptional activation and its loss causes the intellectual disability disorder Kabuki syndrome 1 (KS1) in ways that are not well understood. We conducted parallel studies of proliferation, differentiation, and transcription and chromatin profiling in KMT2D-deficient human and mouse models of brain development to define KMT2D-dependent functions in neurodevelopmental contexts, including adult-born hippocampal NPCs in vivo. We report cell-autonomous proliferation, cell cycle progression, and survival deficits accompanied by transcriptional perturbance of oxygen-sensing, central carbon metabolism, and mRNA/protein metabolism pathways, with functional experiments demonstrating disruption of KMT2D-dependent hypoxia responses and proteasome functions. Importantly, signatures of precocious neuronal differentiation accompanied defects in NPC maintenance. Together, our findings support a model in which KMT2D loss causes transcriptional suppression of metabolic programs for NPC maintenance, resulting in exhaustion of precursors required for adult hippocampal neurogenesis.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/12/02/484410">Preprint</a>
        </div>
      </div>
    </div>

<!--EVAsc preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint8">
                 <em>Expression variation analysis for tumor heterogeneity in single-cell RNA-sequencing data</em> (2019)</a>
                 <p class="note"> Emily F. Davis-Marcisak, Pranay Orugunta, Genevieve Stein-O'Brien, Sidharth V. Puram, Evanthia Roussos Torres, Alexander Hopkins, Elizabeth M. Jaffee, Alexander V. Favorov, Bahman Afsari, Loyal A. Goff, Elana J. Fertig<br>
                 doi: <a href=https://doi.org/10.1101/479287" target="_blank">https://doi.org/10.1101/479287</a></p>
        </span>
      </div>
      <div id="preprint8" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="51832704" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Tumor heterogeneity provides a complex challenge to cancer treatment and is a critical component of therapeutic response, disease recurrence, and patient survival. Single-cell RNA-sequencing (scRNA-seq) technologies reveal the prevalence of intra- and inter-tumor heterogeneity. Computational techniques are essential to quantify the differences in variation of these profiles between distinct cell types, tumor subtypes, and patients to fully characterize intra- and inter-tumor molecular heterogeneity. We devised a new algorithm, Expression Variation Analysis in Single Cells (EVAsc), to perform multivariate statistical analyses of differential variation of expression in gene sets for scRNA-seq. EVAsc has high sensitivity and specificity to detect pathways with true differential heterogeneity in simulated data. We then apply EVAsc to several public domain scRNA-seq tumor datasets to quantify the landscape of tumor heterogeneity in several key applications in cancer genomics, i.e. immunogenicity, cancer subtypes, and metastasis. Immune pathway heterogeneity in hematopoietic cell populations in breast tumors corresponded to the amount diversity present in the T-cell repertoire of each individual. In head and neck squamous cell carcinoma (HNSCC) patients, we found dramatic differences in pathway dysregulation across basal primary tumors. Within the basal primary tumors we also identified increased immune dysregulation in individuals with a high proportion of fibroblasts present in the tumor microenvironment. Moreover, cells in HNSCC primary tumors had significantly more heterogeneity across pathways than cells in metastases, consistent with a model of clonal outgrowth. These results demonstrate the broad utility of EVAsc to quantify inter- and intra-tumor heterogeneity from scRNA-seq data without reliance on low dimensional visualization.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/11/27/479287">Preprint</a>
        </div>
      </div>
    </div>

<!--Schiz MPRA preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint7">
                 <em>Testing the Regulatory Consequences of 1,049 Schizophrenia Associated Variants With a Massively Parallel Reporter Assay</em> (2018)</a>
                 <p class="note"> Leslie Myint, Ruihua Wang, Leandros Boukas, Kasper D Hansen, Loyal A Goff, Dimitrios Avramopoulos<br>
                 doi: <a href=https://doi.org/10.1101/447557" target="_blank">https://doi.org/10.1101/447557</a></p>
        </span>
      </div>
      <div id="preprint7" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="49938395" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Recent genome-wide association studies (GWAS) have identified numerous schizophrenia (SZ) associated loci. As is common across disorders, many of the SZ associated variants are located outside protein-coding regions and are hypothesized to differentially affect the transcription of nearby genes. To systematically identify which variants affect potential regulatory activity, we assayed 1,049 variants in high Linkage Disequilibrium (LD) within 64 reported SZ-associated loci and an additional 30 SNPs from 9 loci associated with Alzheimer's disease (AD) using a massively parallel reporter assay (MPRA). Each variant was assayed on the center of a 95bp synthetic oligonucleotide. The resulting library was transfected 3 independent times into K562 chronic myelogenous leukemia lymphoblasts and another 6 times into SK-SY5Y human neuroblastoma cells. We identified 148 SNPs with significant allelic differences in their effect on expression of the reporter gene in the K562 cells and 53 in the SK-SY5Y cells with an average of 2.6 such SNPs per locus and a median of 1. The overlap between cell lines was modest with 9 SNPs having significant allelic differences in both lines, 8 of these 9 in the same direction. We do not observe a direction preference (increased or decreased enhancer activity) for risk vs. non risk alleles. We find that large LD blocks have a greater density of functional SNPs supporting instances of combinatorial SNP effects that may lead to selection at the haplotype level. Our results help determine driver GWAS variant(s), guide the functional follow up of disease associated loci and enhance our understanding of the genomic dynamics of gene regulation.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/10/30/447557">Preprint</a>
        </div>
      </div>
    </div>

 <!--Retina Methods preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint6">
                 <em>Decomposing cell identity for transfer learning across cellular measurements, platforms, tissues, and species</em> (2018)</a>
                 <p class="note"> Genevieve L Stein-O'Brien, Brian S. Clark, Thomas Sherman, Christina Zibetti, Qiwen Hu, Rachel Sealfon, Sheng Liu, Jiang Qian, Carlo Colantuoni, Seth Blackshaw, Loyal A. Goff, Elana J. Fertig<br>
                 doi: <a href=https://doi.org/10.1101/395004" target="_blank">https://doi.org/10.1101/395004</a></p>
        </span>
      </div>
      <div id="preprint6" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="46741593" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>New approaches are urgently needed to glean biological insights from the vast amounts of single cell RNA sequencing (scRNA-Seq) data now being generated. To this end, we propose that cell identity should map to a reduced set of factors which will describe both exclusive and shared biology of individual cells, and that the dimensions which contain these factors reflect biologically meaningful relationships across different platforms, tissues and species. To find a robust set of dependent factors in large-scale scRNA-Seq data, we developed a Bayesian non-negative matrix factorization (NMF) algorithm, scCoGAPS. Application of scCoGAPS to scRNA-Seq data obtained over the course of mouse retinal development identified gene expression signatures for factors associated with specific cell types and continuous biological processes. To test whether these signatures are shared across diverse cellular contexts, we developed projectR to map biologically disparate datasets into the factors learned by scCoGAPS. Because projecting these dimensions preserve relative distances between samples, biologically meaningful relationships/factors will stratify new data consistent with their underlying processes, allowing labels or information from one dataset to be used for annotation of the other - a machine learning concept called transfer learning. Using projectR, data from multiple datasets was used to annotate latent spaces and reveal novel parallels between developmental programs in other tissues, species and cellular assays. Using this approach we are able to transfer cell type and state designations across datasets to rapidly annotate cellular features in a new dataset without a priori knowledge of their type, identify a species-specific signature of microglial cells, and identify a previously undescribed subpopulation of neurosecretory cells within the lung. Together, these algorithms define biologically meaningful dimensions of cellular identity, state, and trajectories that persist across technologies, molecular features, and species.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/08/20/395004.1">Preprint</a>
        </div>
      </div>
    </div>

 <!--Retina Bio preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint5">
                 <em>Comprehensive analysis of retinal development at single cell resolution identifies NFI factors as essential for mitotic exit and specification of late-born cells</em> (2018)</a>
                 <p class="note"> Brian Clark, Genevieve Stein-O'Brien, Fion Shiau, Gabrielle Cannon, Emily Davis, Thomas Sherman, Fatemeh Rajaii, Rebecca James-Esposito, Richard Gronostajski, Elana Fertig, Loyal A. Goff, Seth Blackshaw<br>
                 doi: <a href="https://doi.org/10.1101/378950" target="_blank">https://doi.org/10.1101/378950</a></p>
        </span>
      </div>
      <div id="preprint5" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="45656869" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Precise temporal control of gene expression in neuronal progenitors is necessary for correct regulation of neurogenesis and cell fate specification. However, the extensive cellular heterogeneity of the developing CNS has posed a major obstacle to identifying the gene regulatory networks that control these processes. To address this, we used single cell RNA-sequencing to profile ten developmental stages encompassing the full course of retinal neurogenesis. This allowed us to comprehensively characterize changes in gene expression that occur during initiation of neurogenesis, changes in developmental competence, and specification and differentiation of each of the major retinal cell types. These data identify transitions in gene expression between early and late-stage retinal progenitors, as well as a classification of neurogenic progenitors. We identify here the NFI family of transcription factors (Nfia, Nfib, and Nfix) as genes with enriched expression within late RPCs, and show they are regulators of bipolar interneuron and Muller glia specification and the control of proliferative quiescence.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2018/07/30/378950">Preprint</a>
        </div>
      </div>
    </div>

 <!--Matrix review preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint4">
                 <em>Enter the matrix: Interpreting unsupervised feature learning with matrix decomposition to discover hidden knowledge in high-throughput omics data.</em> (2017)</a>
                 <p class="note"> Genevieve L. Stein-O'Brien, Raman Arora, Aedin C. Culhane, Alexander Favorov, Casey Greene, Loyal A. Goff, Yifeng Li, Alioune Ngom, Michael F. Ochs, Yanxun Xu, Elana J. Fertig<br>
                 doi: <a href="https://doi.org/10.1101/196915" target="_blank">https://doi.org/10.1101/196915</a></p>
        </span>
      </div>
      <div id="preprint4" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="26912140" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>High-dimensional data is currently standard for biological inquiry. Biological systems are comprised of interrelated gene regulatory mechanisms, gene-gene interactions, and cellular interactions. These interactions induce low-dimensional structure within the high-dimensional data. Matrix factorization, also known as compressed sensing, learns low-dimensional mathematical representations from high-dimensional data. These factorization techniques can embed assumptions about pleiotropy, epistasis, inter-relationships between complex traits, and context-dependent interactions. They have been applied to uncover new biological knowledge in a breadth of topics ranging from pathway discovery to time course analysis. These techniques have been applied to data from diverse high-throughput omics technologies, including bulk and single-cell data. There are numerous computational techniques within the class of matrix factorization, each of which provides a unique interpretation of the processes in high-dimensional data. We review the visualization and applications of matrix factorization to systems-level analyses, which are diverse and require standardization to enable biological interpretation. Codifying the techniques to decipher biologically relevant features with matrix factorization enables their broad application to discovery beyond the limits of current biological knowledge; answering questions from high-dimensional data that we have not yet thought to ask.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2017/10/02/196915.1">Preprint</a>
        </div>
      </div>
    </div>

 <!--Hansen preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint3">
                 <em>Linear models enable powerful differential activity analysis in massively parallel reporter assays.</em> (2017)</a>
                 <p class="note"> Leslie Myint, Dimitrios G. Avramopoulos, Loyal A. Goff, Kasper Hansen<br>
                 doi: <a href="https://doi.org/10.1101/196394" target="_blank">https://doi.org/10.1101/196394</a></p>
        </span>
      </div>
      <div id="preprint3" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="26861189" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Massively parallel reporter assays (MPRAs) have emerged as a popular means for understanding noncoding variation in a variety of conditions. However, development of statistical analysis methods has not kept pace with the use of this assay. We present a linear model framework, mpralm, for the differential analysis of activity measures from these experiments that we show is calibrated and powerful. We show that it outperforms statistical tests that are commonly used in the literature. Further, we investigate theoretical and real-data properties of barcode summarization methods. These results inform recommendations for differential analysis, general group comparisons, and study design in terms of sample sizes and sequencing depth.</p>
            <a target="_blank" href="https://www.biorxiv.org/content/early/2017/09/30/196394">Preprint</a>
        </div>
      </div>
    </div>

    <!--CthPN preprint-->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint2">
                 <em>Variation in neuronal activity state, axonal projection target, and position principally define the transcriptional identity of individual neocortical projection neurons.</em> (2017)</a>
                 <p class="note"> Maxime Chev&eacute;e, Johanna D. Robertson, Gabrielle H. Cannon, Solange P. Brown<sup>*</sup>, Loyal A. Goff<sup>*</sup><br>
                 doi: <a href="https://doi.org/10.1101/157149" target="_blank">https://doi.org/10.1101/1571499</a></p>
        </span>
      </div>
      <div id="preprint2" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="21401403" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Single-cell RNA sequencing technologies have generated the first catalogs of transcriptionally defined neuronal subtypes of the brain. However, the biologically informative cellular processes that contribute to neuronal subtype specification and transcriptional heterogeneity remain unclear. By comparing the gene expression profiles of single layer 6 corticothalamic neurons in somatosensory cortex, we show that transcriptional subtypes primarily reflect axonal projection pattern, laminar position within the cortex, and neuronal activity state. Pseudotemporal ordering of 1023 cellular responses to manipulation of sensory input demonstrates that changes in expression of activity-induced genes both reinforced cell-type identity and contributed to increased transcriptional heterogeneity within each cell type. This is due to cell-type specific biases in the choice of transcriptional states following manipulation of neuronal activity. These results reveal that axonal projection pattern, laminar position, and activity state define significant axes of variation that contribute both to the transcriptional identity of individual neurons and to the transcriptional heterogeneity within each neuronal subtype.</p>
            <a href="https://f1000.com/prime/732442069?bd=1" target="_blank"><img src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif" alt="Access the recommendation on F1000Prime" id="bg" /></a><br>
            <a target="_blank" href="http://www.biorxiv.org/content/early/2017/06/30/157149">Preprint</a>
        </div>
      </div>
    </div>


    <!-- DA neuron preprint -->
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#preprint1">
                 <em>Temporal and spatial variation among single dopaminergic neuron transcriptomes informs cellular phenotype diversity and Parkinson's Disease gene prioritization</em> (2017)</a>
                 <p class="note">Paul Hook, Sarah A. McClymont, Gabrielle H. Cannon, William D. Law, Loyal A. Goff<sup>*</sup>, Andrew S. McCallion<sup>*</sup><br>
                 doi: <a href="https://doi.org/10.1101/148049" target="_blank">https://doi.org/10.1101/148049</a></p>
        </span>
      </div>
      <div id="preprint1" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-altmetric-id="20930309" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>Parkinson's Disease (PD) causes collapse of substantia nigra (SN) dopaminergic (DA) neurons of the midbrain (MB), while other DA populations are relatively spared. Here, we used single-cell RNA-seq (scRNA-seq) to characterize DA neuron populations in the mouse brain at embryonic and postnatal timepoints. These data allow for the discrimination between olfactory bulb (OB), forebrain (FB), and MB DA populations as well identification of subpopulations of DA neurons in each region. We observe a longitudinal axis of MB DA development, during which specialization and heterogeneity increases. We identify three distinct subpopulations of known MB DA neurons and provide evidence of a postnatal MB DA precursor, identifying novel markers for each subpopulation. Further, we discover gene regulatory networks (GRNs) that are significantly associated with neurodegenerative diseases and highly correlated with specific DA neuron subpopulations. By integrating these data with published genome-wide association studies (GWAS), we prioritize candidate genes in all 32 PD associated loci. Collectively, our data reveal genes and pathways that may begin to explain the selective vulnerability of SN DA neurons and allow for the systematic prioritization of genes in PD GWAS loci for functional evaluation.</p>
            <a target="_blank" href="http://biorxiv.org/content/early/2017/06/09/148049">Preprint</a>
        </div>
      </div>
    </div>
  </div>

<h2>Publications</h2>

  <div class="panel-group" id="accordion">
    {% for record in records %}
    <div class="panel panel-default">
      <div class="panel-heading">
        <span class="panel-title">
        <a data-toggle="collapse" class="panel-article-title" data-parent="#accordionpubs" href="#collapse{{ loop.index }}">
                 <em>{{ record['TI'] }}</em> ({{ record['DP'][:4] }})</a>
                 <p class="note">{{ ", ".join(record['AU'])}}<br>
                 {{ record['SO'] }}</p>
        </span>
      </div>
      <div id="collapse{{ loop.index }}" class="panel-collapse collapse">
        <div class="panel-body" style="background-color:#fffff5;">
          <div data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" data-pmid="{{record['PMID']}}" class="altmetric-embed pull-left" style="padding-right: 0.25cm;"></div>
          <p>{{ record['AB'] }}</p>
          {% if record['LID'] is defined%}
              <a target="_blank" href="http://dx.doi.org/{{ record['LID'].split('[doi]')[0] }}">{{ record['LID'].split('[doi]')[0] }}</a>
          {% else %}
              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/{{ record['PMID'] }}">Article</a>
          {% endif %}
        </div>
      </div>
    </div>
    {% endfor %}
  </div>

{% endblock content %}
