{% extends "base.html" %}

{% block title %}
	Software Products
{% endblock title %}

{% block content %}
<body class="is-preload">
<div id="page-wrapper">
	{% include 'header.html' %}
	<!-- Main -->
		<div id="main" class="wrapper style1">
			<div class="container">
				<header class="major">
					<h2>Software</h2>
				</header>

				<!-- Content -->
					<section id="content">

						<!-- pySeq 2501 -->
						<div class="box alt">
						     <a href="https://github.com/chaichontat/pyseq2501">
						       <img class="image software left" src="images/pyseq.png" width="200px"/>
						     </a>
						       <h4 class="media-heading"><a href="https://github.com/chaichontat/pyseq2501">pySeq 2501</a></h4>
									 <h6>Author: ChaichontatÂ Sriworarat</h6>
						       <p><em><b>Control your HiSeq 2000/2500 with ease.</b></em> pyseq2501 is an adaptation and refactoring of the <a href="https://github.com/nygctech/PySeq2500" target="_blank">pyseq 2500</a> controller software to repurpose your aging, unsupported HiSeq 2500 machine into a useful fluidics + microscopy platform for custom workflows.</p>
						</div>
						<!-- TRICycle -->
						<div class="box alt">
						     <a href="https://github.com/hansenlab/tricycle">
						       <img class="image software left" src="images/tricycle_sticker.png" width="200px"/>
						     </a>
						       <h4 class="media-heading"><a href="https://github.com/hansenlab/tricycle">TRICycle</a></h4><br>
						       <p>TRICycle (Transferable Representation and Inference of Cell Cycle) is an R package for rapid and universal annotation of cell cycle state for large-scale single cell assays. Building off of the ProjectR package for transfer learning, tricycle embeds new single cell RNA-Seq, ATAC-Seq, or other single cell assays into a reference latent space to enable the rapid estimation of continuous position along a cell cycle trajectory. This projection is fast, memory-efficient, requires only minimal data preprocessing, and requires no <em>a priori</em> knowledge of cellular annotations. The speed, efficiency, and accuracy of tricycle enables annotation of atlas-level single cell datasets for exploration of cell cycle dynamics at scale.</p>
						</div>

						<!-- ProjectR -->
						<div class="box alt">
						     <a href="https://github.com/genesofeve/projectR">
						       <img class="image software left" src="images/projectR.png" width="200px"/>
						     </a>
						       <h4 class="media-heading"><a href="https://github.com/genesofeve/projectR">projectR</a></h4><br>
						       <p>ProjectR is a regression-based trasfer learning tool for single cell analysis. ProjectR can embed any type of high-throughput molecular biology assay into the latent spaces learned from a source dataset to facilitate rapid annotation of cell types or biological features. Because projecting these dimensions preserves relative distances between samples, biologically meaningful relationships/factors will stratify new data consistent with their underlying processes, allowing labels or information from one dataset to be used for annotation of the other. ProjectR can reveal novel parallels between gene expression programs in other tissues, species and cellular assays. We have used this approach to transfer cell type and state designations across datasets to rapidly annotate cellular features in a new dataset without a priori knowledge of their type, identify a species-specific signature of microglial cells, and identify a previously undescribed subpopulation of neurosecretory cells within the lung.</p>
						</div>


					</section>
			</div>
		</div>
</div>
{% endblock content %}
