{% extends "base.html" %}

{% block title %}
	Software Products
{% endblock title %}

{% block content %}
<body class="is-preload">
<div id="page-wrapper">
	{% include 'header.html' %}
	<!-- Main -->
		<div id="main" class="wrapper style1">
			<div class="container">
				<header class="major">
					<h2>Software</h2>
				</header>

				<!-- Content -->
					<section id="content">
						<!-- TRICycle -->
						<div class="box alt">
						     <a href="https://github.com/hansenlab/tricycle">
						       <img class="image software left" src="images/tricycle_sticker.png" width="200px"/>
						     </a>
						       <h4 class="media-heading"><a href="https://github.com/hansenlab/tricycle">TRICycle</a></h4><br>
						       <p>TRICycle (Transferable Representation and Inference of Cell Cycle) is an R package for rapid and universal annotation of cell cycle state for large-scale single cell assays. Building off of the ProjectR package for transfer learning, tricycle embeds new single cell RNA-Seq, ATAC-Seq, or other single cell assays into a reference latent space to enable the rapid estimation of continuous position along a cell cycle trajectory. This projection is fast, memory-efficient, requires only minimal data preprocessing, and requires no <em>a priori</em> knowledge of cellular annotations. The speed, efficiency, and accuracy of tricycle enables annotation of atlas-level single cell datasets for exploration of cell cycle dynamics at scale.</p>
						</div>

						<!-- ProjectR -->
						<div class="box alt">
						     <a href="https://github.com/genesofeve/projectR">
						       <img class="image software left" src="images/projectR.png" width="200px"/>
						     </a>
						       <h4 class="media-heading"><a href="https://github.com/genesofeve/projectR">projectR</a></h4><br>
						       <p>ProjectR is a regression-based trasfer learning tool for single cell analysis. ProjectR can embed any type of high-throughput molecular biology assay into the latent spaces learned from a source dataset to facilitate rapid annotation of cell types or biological features. Because projecting these dimensions preserves relative distances between samples, biologically meaningful relationships/factors will stratify new data consistent with their underlying processes, allowing labels or information from one dataset to be used for annotation of the other. ProjectR can reveal novel parallels between gene expression programs in other tissues, species and cellular assays. We have used this approach to transfer cell type and state designations across datasets to rapidly annotate cellular features in a new dataset without a priori knowledge of their type, identify a species-specific signature of microglial cells, and identify a previously undescribed subpopulation of neurosecretory cells within the lung.</p>
						</div>

						<!-- cummeRbund -->
						<div class="box alt">
						     <a href="http://compbio.mit.edu/cummeRbund">
						       <img class="image software left" src="images/cummeRbund_gold.png" width="100px"/>
						     </a>
						       <h4 class="media-heading"><a class="pull-left" href="http://compbio.mit.edu/cummeRbund">CummeRbund</a></h4><br>
						       <p>An R/Bioconductor package for the persitent storage, manipulation, analysis, and visualization of Cufflinks RNA-Seq data. High-throughput sequencing of RNA-fragments is a powerful technique with many applications. The results of these analyses is often very large data sets with a high degree of relations between various data types and can be somewhat overwhelming. CummeRbund was designed to help simplify the analysis and exploration portion of RNA-Seq data derived from the output of a differential expression analysis using cuffdiff, with the goal of providing fast and intuitive access to your results. CummeRbund takes the various output files from a cuffdiff run and creates a SQLite database of the results describing appropriate relationships betweeen genes, transcripts, transcription start sites, and CDS regions. Once stored and indexed, data for these features, even across multiple samples or conditions, can be retrieved very efficiently and allows the user to explore subfeatures of individual genes, or genesets as the analysis requires.</p>
						</div>

						<!-- DeCoN -->
						<div class="box alt">
						     <a href="http://decon.fas.harvard.edu/">
						       <img class="image software left" src="images/decon.png" width="200px"/>
						     </a>
						       <h4 class="media-heading"><a href="http://decon.fas.harvard.edu">DeCoN</a></h4><br>
						       <p>The Developing Cortical Neuron Transcriptional Resource. DeCoN is a web utility for the exploration of deep transcriptional profiling (RNA-Seq) data derived from sorted populations of mouse cortical projection neurons. This utility presents comprehensive isoform, gene, and geneset analyses of expression levels for Callosal, Subcerebral, and Corticothalamic projection neurons during corticogenesis.</p>
						</div>

					</section>
			</div>
		</div>
</div>
{% endblock content %}
